<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Symmetry and Asymmetry | Toward an analysis of design intent and morphological standardization in Gahagan bifaces</title>
  <meta name="description" content="Gahagan biface shape change." />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Symmetry and Asymmetry | Toward an analysis of design intent and morphological standardization in Gahagan bifaces" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Gahagan biface shape change." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Symmetry and Asymmetry | Toward an analysis of design intent and morphological standardization in Gahagan bifaces" />
  
  <meta name="twitter:description" content="Gahagan biface shape change." />
  

<meta name="author" content="Robert Z. Selden, Jr." />


<meta name="date" content="2020-10-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="geometric-morphometrics.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">gahaganmorph3</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminaries</a><ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.1</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.2</b> Funding</a></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.3</b> Data management</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html"><i class="fa fa-check"></i><b>1</b> Temporal Dynamics</a><ul>
<li class="chapter" data-level="1.1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-5-at-the-mounds-plantation-site"><i class="fa fa-check"></i><b>1.1</b> Calibration and plots for dates from Burial Pit 5 at the Mounds Plantation site</a></li>
<li class="chapter" data-level="1.2" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-2-at-the-gahagan-mound-site"><i class="fa fa-check"></i><b>1.2</b> Calibration and plots for dates from Burial Pit 2 at the Gahagan Mound site</a></li>
<li class="chapter" data-level="1.3" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-f119-and-f134-at-the-george-c.-davis-site"><i class="fa fa-check"></i><b>1.3</b> Calibration and plots for the dates from F119 and F134 at the George C. Davis site</a></li>
<li class="chapter" data-level="1.4" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-au2-at-the-j.-b.-white-site"><i class="fa fa-check"></i><b>1.4</b> Calibration and plots for dates from AU2 at the J. B. White site</a></li>
<li class="chapter" data-level="1.5" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-the-baker-site"><i class="fa fa-check"></i><b>1.5</b> Calibration and plots for the dates from the Baker site</a></li>
<li class="chapter" data-level="1.6" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#summed-probability-distributions-for-contexts-yielding-gahagan-bifaces"><i class="fa fa-check"></i><b>1.6</b> Summed probability distributions for contexts yielding Gahagan bifaces</a></li>
<li class="chapter" data-level="1.7" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#chronological-model-for-sca-north-sca-south-and-central-texas-contexts-yielding-gahagan-bifaces-with-baker-site-dates"><i class="fa fa-check"></i><b>1.7</b> Chronological model for SCA-north, SCA-south, and central Texas contexts yielding Gahagan bifaces with Baker site dates</a></li>
<li class="chapter" data-level="1.8" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>1.8</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="modeling-geometry.html"><a href="modeling-geometry.html"><i class="fa fa-check"></i><b>2</b> Modeling Geometry</a><ul>
<li class="chapter" data-level="2.1" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section"><i class="fa fa-check"></i><b>2.1</b> 4078-8</a></li>
<li class="chapter" data-level="2.2" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section-1"><i class="fa fa-check"></i><b>2.2</b> 4078-72</a></li>
<li class="chapter" data-level="2.3" data-path="modeling-geometry.html"><a href="modeling-geometry.html#references-cited"><i class="fa fa-check"></i><b>2.3</b> References cited</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html"><i class="fa fa-check"></i><b>3</b> Elliptical Fourier Analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#load-packages-data"><i class="fa fa-check"></i><b>3.1</b> Load packages + data</a></li>
<li class="chapter" data-level="3.2" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#generate-outlines"><i class="fa fa-check"></i><b>3.2</b> Generate outlines</a></li>
<li class="chapter" data-level="3.3" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#pile-and-mosaics"><i class="fa fa-check"></i><b>3.3</b> Pile and mosaics</a></li>
<li class="chapter" data-level="3.4" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#calibrate-harmonic-efa"><i class="fa fa-check"></i><b>3.4</b> Calibrate harmonic + EFA</a></li>
<li class="chapter" data-level="3.5" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#pca"><i class="fa fa-check"></i><b>3.5</b> PCA</a></li>
<li class="chapter" data-level="3.6" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#manova-manova_pw"><i class="fa fa-check"></i><b>3.6</b> MANOVA + MANOVA_PW</a></li>
<li class="chapter" data-level="3.7" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#mean-shapes"><i class="fa fa-check"></i><b>3.7</b> Mean shapes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html"><i class="fa fa-check"></i><b>4</b> Landmarking Protocol 3d2</a><ul>
<li class="chapter" data-level="4.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#foundations"><i class="fa fa-check"></i><b>4.1</b> Foundations</a><ul>
<li class="chapter" data-level="4.1.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#learning-from-efa"><i class="fa fa-check"></i><b>4.1.1</b> Learning from EFA</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#generating-the-peripheral-plan-view-spline"><i class="fa fa-check"></i><b>4.2</b> Generating the peripheral (plan view) spline</a><ul>
<li class="chapter" data-level="4.2.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-spline-1"><i class="fa fa-check"></i><b>4.2.1</b> Splitting Spline 1</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-geometry-build-1"><i class="fa fa-check"></i><b>4.3</b> Reference geometry, Build 1</a><ul>
<li class="chapter" data-level="4.3.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-vector-1-point-2-and-plane-2"><i class="fa fa-check"></i><b>4.3.1</b> Reference Vector 1, Point 2, and Plane 2</a></li>
<li class="chapter" data-level="4.3.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-planes-3-and-4"><i class="fa fa-check"></i><b>4.3.2</b> Reference Planes 3 and 4</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#cross-sections"><i class="fa fa-check"></i><b>4.4</b> Cross-sections</a><ul>
<li class="chapter" data-level="4.4.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-1"><i class="fa fa-check"></i><b>4.4.1</b> Splitting the curves (Step 1)</a></li>
<li class="chapter" data-level="4.4.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-2"><i class="fa fa-check"></i><b>4.4.2</b> Splitting the curves (Step 2)</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#landmark-and-semilandmark-placement"><i class="fa fa-check"></i><b>4.5</b> Landmark and semilandmark placement</a></li>
<li class="chapter" data-level="4.6" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>4.6</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html"><i class="fa fa-check"></i><b>5</b> Landmark Sampling Evaluation Curve (LaSEC)</a><ul>
<li class="chapter" data-level="5.1" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#initial-2d-iteration"><i class="fa fa-check"></i><b>5.1</b> Initial 2D iteration</a></li>
<li class="chapter" data-level="5.2" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#first-3d-iteration"><i class="fa fa-check"></i><b>5.2</b> First 3D iteration</a></li>
<li class="chapter" data-level="5.3" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#current-3d-iteration"><i class="fa fa-check"></i><b>5.3</b> Current 3D iteration</a></li>
<li class="chapter" data-level="5.4" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>5.4</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html"><i class="fa fa-check"></i><b>6</b> Geometric morphometrics</a><ul>
<li class="chapter" data-level="6.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#load-geomorph-and-data"><i class="fa fa-check"></i><b>6.1</b> Load geomorph and data</a></li>
<li class="chapter" data-level="6.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#generalized-procrustes-analysis"><i class="fa fa-check"></i><b>6.2</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="6.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#hypothesis-1"><i class="fa fa-check"></i><b>6.3</b> Hypothesis 1</a><ul>
<li class="chapter" data-level="6.3.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#boxplot-h1"><i class="fa fa-check"></i><b>6.3.1</b> Boxplot (H1)</a></li>
<li class="chapter" data-level="6.3.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#principal-components-analysis-h1"><i class="fa fa-check"></i><b>6.3.2</b> Principal Components Analysis (H1)</a></li>
<li class="chapter" data-level="6.3.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#define-models-h1"><i class="fa fa-check"></i><b>6.3.3</b> Define models (H1)</a></li>
<li class="chapter" data-level="6.3.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#allometry-h1"><i class="fa fa-check"></i><b>6.3.4</b> Allometry (H1)</a></li>
<li class="chapter" data-level="6.3.5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#procrustes-anova-h1"><i class="fa fa-check"></i><b>6.3.5</b> Procrustes ANOVA (H1)</a></li>
<li class="chapter" data-level="6.3.6" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity-h1"><i class="fa fa-check"></i><b>6.3.6</b> Morphological disparity (H1)</a></li>
<li class="chapter" data-level="6.3.7" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#mean-shapes-h1"><i class="fa fa-check"></i><b>6.3.7</b> Mean shapes (H1)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#hypothesis-2"><i class="fa fa-check"></i><b>6.4</b> Hypothesis 2</a><ul>
<li class="chapter" data-level="6.4.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#boxplot-h2"><i class="fa fa-check"></i><b>6.4.1</b> Boxplot (H2)</a></li>
<li class="chapter" data-level="6.4.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#principal-components-analysis-h2"><i class="fa fa-check"></i><b>6.4.2</b> Principal Components Analysis (H2)</a></li>
<li class="chapter" data-level="6.4.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#define-models-h2"><i class="fa fa-check"></i><b>6.4.3</b> Define models (H2)</a></li>
<li class="chapter" data-level="6.4.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#allometry-h2"><i class="fa fa-check"></i><b>6.4.4</b> Allometry (H2)</a></li>
<li class="chapter" data-level="6.4.5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#procrustes-anova-h2"><i class="fa fa-check"></i><b>6.4.5</b> Procrustes ANOVA (H2)</a></li>
<li class="chapter" data-level="6.4.6" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity-h2"><i class="fa fa-check"></i><b>6.4.6</b> Morphological disparity (H2)</a></li>
<li class="chapter" data-level="6.4.7" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#mean-shapes-h2"><i class="fa fa-check"></i><b>6.4.7</b> Mean shapes (H2)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html"><i class="fa fa-check"></i><b>7</b> Symmetry and Asymmetry</a><ul>
<li class="chapter" data-level="7.1" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#gahagan-bifaces"><i class="fa fa-check"></i><b>7.1</b> Gahagan bifaces</a></li>
<li class="chapter" data-level="7.2" data-path="elliptical-fourier-analysis.html"><a href="elliptical-fourier-analysis.html#load-packages-data"><i class="fa fa-check"></i><b>7.2</b> Load packages + data</a></li>
<li class="chapter" data-level="7.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#generalized-procrustes-analysis"><i class="fa fa-check"></i><b>7.3</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="7.4" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#principal-components-analysis"><i class="fa fa-check"></i><b>7.4</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="7.5" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#define-models"><i class="fa fa-check"></i><b>7.5</b> Define models</a><ul>
<li class="chapter" data-level="7.5.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#hypothesis-1"><i class="fa fa-check"></i><b>7.5.1</b> Hypothesis 1</a></li>
<li class="chapter" data-level="7.5.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#hypothesis-2"><i class="fa fa-check"></i><b>7.5.2</b> Hypothesis 2</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#hypothesis-1-1"><i class="fa fa-check"></i><b>7.6</b> Hypothesis 1</a><ul>
<li class="chapter" data-level="7.6.1" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#pca-h1"><i class="fa fa-check"></i><b>7.6.1</b> PCA (H1)</a></li>
<li class="chapter" data-level="7.6.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#procrustes-anova-h1"><i class="fa fa-check"></i><b>7.6.2</b> Procrustes ANOVA (H1)</a></li>
<li class="chapter" data-level="7.6.3" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#mean-symmetric-shapes-h1"><i class="fa fa-check"></i><b>7.6.3</b> Mean Symmetric Shapes (H1)</a></li>
<li class="chapter" data-level="7.6.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity-h1"><i class="fa fa-check"></i><b>7.6.4</b> Morphological disparity (H1)</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#hypothesis-2-1"><i class="fa fa-check"></i><b>7.7</b> Hypothesis 2</a><ul>
<li class="chapter" data-level="7.7.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#principal-components-analysis-h2"><i class="fa fa-check"></i><b>7.7.1</b> Principal Components Analysis (H2)</a></li>
<li class="chapter" data-level="7.7.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#procrustes-anova-h2"><i class="fa fa-check"></i><b>7.7.2</b> Procrustes ANOVA (H2)</a></li>
<li class="chapter" data-level="7.7.3" data-path="symmetry-and-asymmetry.html"><a href="symmetry-and-asymmetry.html#mean-asymmetric-shapes-h2"><i class="fa fa-check"></i><b>7.7.3</b> Mean Asymmetric Shapes (H2)</a></li>
<li class="chapter" data-level="7.7.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity-h2"><i class="fa fa-check"></i><b>7.7.4</b> Morphological disparity (H2)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Toward an analysis of <em>design intent</em> and <em>morphological standardization</em> in Gahagan bifaces</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="symmetry-and-asymmetry" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Symmetry and Asymmetry</h1>
<p>Within studies of biface morphology, an analysis of bilateral/object symmetry can yield useful insights related to <em>design intent</em>, given the assumption that the knapper’s goal was to achieve a symmetrical shape. Within that framework, deviation from bilateral symmetry (fluctuating asymmetry) can serve as a useful measure of <em>morphological standardization</em> in an analysis of morphological disparity. Both measures have utility in comparisons of contexts and sites, to include wide-ranging spatial and temporal applications.</p>
<div id="gahagan-bifaces" class="section level2">
<h2><span class="header-section-number">7.1</span> Gahagan bifaces</h2>
<p>Per Klingenberg <span class="citation">(<a href="#ref-RN8579" role="doc-biblioref">2002</a>, 1912 and Figure 2)</span>:</p>
<ul>
<li>Symmetric variation includes movements of the unpaired landmarks along the median axis (or within the median plane) and joint mirror-image shifts of pairs of corresponding landmarks in any direction.</li>
<li>Asymmetry is quantified through the differences between the original and mirrored configuration from the symmetric consensus of the original and mirror image.</li>
</ul>
</div>
<div id="load-packages-data" class="section level2">
<h2><span class="header-section-number">7.2</span> Load packages + data</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="symmetry-and-asymmetry.html#cb1-1"></a><span class="co"># load packages</span></span>
<span id="cb1-2"><a href="symmetry-and-asymmetry.html#cb1-2"></a></span>
<span id="cb1-3"><a href="symmetry-and-asymmetry.html#cb1-3"></a><span class="co"># devtools::install_github(&quot;geomorphR/geomorph&quot;, ref = &quot;Stable&quot;, build_vignettes = TRUE)</span></span>
<span id="cb1-4"><a href="symmetry-and-asymmetry.html#cb1-4"></a><span class="kw">library</span>(geomorph)</span></code></pre></div>
<pre><code>## Loading required package: RRPP</code></pre>
<pre><code>## Loading required package: rgl</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="symmetry-and-asymmetry.html#cb4-1"></a><span class="kw">library</span>(tidyverse)</span></code></pre></div>
<pre><code>## -- Attaching packages ------------------------------------------ tidyverse 1.3.0.9000 --</code></pre>
<pre><code>## v ggplot2 3.3.2     v purrr   0.3.4
## v tibble  3.0.3     v dplyr   1.0.2
## v tidyr   1.1.2     v stringr 1.4.0
## v readr   1.3.1     v forcats 0.5.0</code></pre>
<pre><code>## -- Conflicts -------------------------------------------------- tidyverse_conflicts() --
## x dplyr::filter() masks stats::filter()
## x dplyr::lag()    masks stats::lag()</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="symmetry-and-asymmetry.html#cb8-1"></a><span class="kw">library</span>(wesanderson)</span>
<span id="cb8-2"><a href="symmetry-and-asymmetry.html#cb8-2"></a></span>
<span id="cb8-3"><a href="symmetry-and-asymmetry.html#cb8-3"></a><span class="co"># setwd</span></span>
<span id="cb8-4"><a href="symmetry-and-asymmetry.html#cb8-4"></a><span class="kw">setwd</span>(<span class="kw">getwd</span>())</span>
<span id="cb8-5"><a href="symmetry-and-asymmetry.html#cb8-5"></a></span>
<span id="cb8-6"><a href="symmetry-and-asymmetry.html#cb8-6"></a><span class="co"># read GM data</span></span>
<span id="cb8-7"><a href="symmetry-and-asymmetry.html#cb8-7"></a><span class="kw">source</span>(<span class="st">&#39;readmulti.csv.R&#39;</span>)</span>
<span id="cb8-8"><a href="symmetry-and-asymmetry.html#cb8-8"></a></span>
<span id="cb8-9"><a href="symmetry-and-asymmetry.html#cb8-9"></a><span class="co"># read .csv files</span></span>
<span id="cb8-10"><a href="symmetry-and-asymmetry.html#cb8-10"></a><span class="kw">setwd</span>(<span class="st">&quot;./data&quot;</span>)</span>
<span id="cb8-11"><a href="symmetry-and-asymmetry.html#cb8-11"></a>filelist &lt;-<span class="st"> </span><span class="kw">list.files</span>(<span class="dt">pattern =</span> <span class="st">&quot;.csv&quot;</span>)</span>
<span id="cb8-12"><a href="symmetry-and-asymmetry.html#cb8-12"></a>coords &lt;-<span class="st"> </span><span class="kw">readmulti.csv</span>(filelist)</span>
<span id="cb8-13"><a href="symmetry-and-asymmetry.html#cb8-13"></a><span class="kw">setwd</span>(<span class="st">&quot;../&quot;</span>)</span>
<span id="cb8-14"><a href="symmetry-and-asymmetry.html#cb8-14"></a></span>
<span id="cb8-15"><a href="symmetry-and-asymmetry.html#cb8-15"></a><span class="co"># read qualitative data</span></span>
<span id="cb8-16"><a href="symmetry-and-asymmetry.html#cb8-16"></a>qdata &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;qdata.csv&quot;</span>, <span class="dt">header =</span> <span class="ot">TRUE</span>, <span class="dt">row.names =</span> <span class="dv">1</span>)</span>
<span id="cb8-17"><a href="symmetry-and-asymmetry.html#cb8-17"></a>qdata &lt;-<span class="st"> </span>qdata[<span class="kw">match</span>(<span class="kw">dimnames</span>(coords)[[<span class="dv">3</span>]], <span class="kw">rownames</span>(qdata)),]</span>
<span id="cb8-18"><a href="symmetry-and-asymmetry.html#cb8-18"></a>land.pairs &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;asymmetry.csv&quot;</span>, <span class="dt">header =</span> <span class="ot">TRUE</span>)</span>
<span id="cb8-19"><a href="symmetry-and-asymmetry.html#cb8-19"></a>ind &lt;-<span class="st"> </span><span class="kw">rep</span>(<span class="dv">1</span><span class="op">:</span><span class="kw">length</span>(filelist))</span></code></pre></div>
</div>
<div id="generalized-procrustes-analysis" class="section level2">
<h2><span class="header-section-number">7.3</span> Generalized Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(Kendall <a href="#ref-RN11622" role="doc-biblioref">1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span> performed in R 4.0.2 <span class="citation">(R Core Development Team, <a href="#ref-R" role="doc-biblioref">2020</a>)</span> using the <code>geomorph</code> library v. 3.3.1 <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>; Adams and Otárola-Castillo <a href="#ref-RN1774" role="doc-biblioref">2013</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(Gower <a href="#ref-RN11564" role="doc-biblioref">1975</a>; Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(Rohlf <a href="#ref-RN1646" role="doc-biblioref">1999</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="symmetry-and-asymmetry.html#cb9-1"></a><span class="co"># gpa - bilateral asymmetry</span></span>
<span id="cb9-2"><a href="symmetry-and-asymmetry.html#cb9-2"></a>res.bilat &lt;-<span class="st"> </span><span class="kw">bilat.symmetry</span>(coords,</span>
<span id="cb9-3"><a href="symmetry-and-asymmetry.html#cb9-3"></a>                          <span class="dt">ind =</span> ind, </span>
<span id="cb9-4"><a href="symmetry-and-asymmetry.html#cb9-4"></a>                          <span class="dt">object.sym =</span> <span class="ot">TRUE</span>,</span>
<span id="cb9-5"><a href="symmetry-and-asymmetry.html#cb9-5"></a>                          <span class="dt">land.pairs =</span> land.pairs,</span>
<span id="cb9-6"><a href="symmetry-and-asymmetry.html#cb9-6"></a>                          <span class="dt">RRPP =</span> <span class="ot">TRUE</span>,</span>
<span id="cb9-7"><a href="symmetry-and-asymmetry.html#cb9-7"></a>                          <span class="dt">iter =</span> <span class="dv">9999</span>,</span>
<span id="cb9-8"><a href="symmetry-and-asymmetry.html#cb9-8"></a>                          <span class="dt">print.progress =</span> <span class="ot">FALSE</span>)</span>
<span id="cb9-9"><a href="symmetry-and-asymmetry.html#cb9-9"></a><span class="kw">summary</span>(res.bilat)</span></code></pre></div>
<pre><code>## 
## Call:
## bilat.symmetry(A = coords, ind = ind, object.sym = TRUE, land.pairs = land.pairs,  
##     iter = 9999, RRPP = TRUE, print.progress = FALSE) 
## 
## 
## Symmetry (data) type: Object 
## 
## Type I (Sequential) Sums of Squares and Cross-products
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Shape ANOVA
##           Df      SS        MS     Rsq      F       Z Pr(&gt;F)    
## ind      101 0.97518 0.0096552 0.87450 7.4576 14.1594  1e-04 ***
## side       1 0.00919 0.0091887 0.00824 7.0973  3.3693  2e-04 ***
## ind:side 101 0.13076 0.0012947 0.11726                          
## Total    203 1.11513                                            
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="symmetry-and-asymmetry.html#cb11-1"></a><span class="co"># symmetric shape/design intent</span></span>
<span id="cb11-2"><a href="symmetry-and-asymmetry.html#cb11-2"></a>symm.shape &lt;-<span class="st"> </span>res.bilat<span class="op">$</span>symm.shape[,<span class="dv">1</span><span class="op">:</span><span class="dv">2</span>,]</span>
<span id="cb11-3"><a href="symmetry-and-asymmetry.html#cb11-3"></a>symm.shape2 &lt;-<span class="st"> </span>res.bilat<span class="op">$</span>symm.shape[,<span class="dv">2</span><span class="op">:</span><span class="dv">3</span>,]</span>
<span id="cb11-4"><a href="symmetry-and-asymmetry.html#cb11-4"></a></span>
<span id="cb11-5"><a href="symmetry-and-asymmetry.html#cb11-5"></a><span class="co"># plot all specimens/2D (symmetric component)</span></span>
<span id="cb11-6"><a href="symmetry-and-asymmetry.html#cb11-6"></a><span class="kw">plotAllSpecimens</span>(symm.shape)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/symm.shape-1.png" alt="_**Figure 3a-b. Mean shape (black) and specimens (gray) for the symmetric shape component plotted in 2D.**_" width="100%" />
<p class="caption">
(#fig:symm.shape-1)<em><strong>Figure 3a-b. Mean shape (black) and specimens (gray) for the symmetric shape component plotted in 2D.</strong></em>
</p>
</div>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="symmetry-and-asymmetry.html#cb12-1"></a><span class="kw">plotAllSpecimens</span>(symm.shape2)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/symm.shape-2.png" alt="_**Figure 3a-b. Mean shape (black) and specimens (gray) for the symmetric shape component plotted in 2D.**_" width="100%" />
<p class="caption">
(#fig:symm.shape-2)<em><strong>Figure 3a-b. Mean shape (black) and specimens (gray) for the symmetric shape component plotted in 2D.</strong></em>
</p>
</div>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="symmetry-and-asymmetry.html#cb13-1"></a><span class="co"># asymmetric shape/standardization?</span></span>
<span id="cb13-2"><a href="symmetry-and-asymmetry.html#cb13-2"></a>asymm.shape &lt;-<span class="st"> </span>res.bilat<span class="op">$</span>FA.component[,<span class="dv">1</span><span class="op">:</span><span class="dv">2</span>,]</span>
<span id="cb13-3"><a href="symmetry-and-asymmetry.html#cb13-3"></a>asymm.shape2 &lt;-<span class="st"> </span>res.bilat<span class="op">$</span>FA.component[,<span class="dv">2</span><span class="op">:</span><span class="dv">3</span>,]</span>
<span id="cb13-4"><a href="symmetry-and-asymmetry.html#cb13-4"></a></span>
<span id="cb13-5"><a href="symmetry-and-asymmetry.html#cb13-5"></a><span class="co"># plot all specimens/2D (asymmetric component)</span></span>
<span id="cb13-6"><a href="symmetry-and-asymmetry.html#cb13-6"></a><span class="kw">plotAllSpecimens</span>(asymm.shape)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/asymm.shape-1.png" alt="_**Figure 4a-b. Mean shape (black) and specimens (gray) for the asymmetric shape component plotted in 2D.**_" width="100%" />
<p class="caption">
(#fig:asymm.shape-1)<em><strong>Figure 4a-b. Mean shape (black) and specimens (gray) for the asymmetric shape component plotted in 2D.</strong></em>
</p>
</div>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="symmetry-and-asymmetry.html#cb14-1"></a><span class="kw">plotAllSpecimens</span>(asymm.shape2)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/asymm.shape-2.png" alt="_**Figure 4a-b. Mean shape (black) and specimens (gray) for the asymmetric shape component plotted in 2D.**_" width="100%" />
<p class="caption">
(#fig:asymm.shape-2)<em><strong>Figure 4a-b. Mean shape (black) and specimens (gray) for the asymmetric shape component plotted in 2D.</strong></em>
</p>
</div>
</div>
<div id="principal-components-analysis" class="section level2">
<h2><span class="header-section-number">7.4</span> Principal Components Analysis</h2>
<p>Principal components analysis <span class="citation">(Jolliffe <a href="#ref-RN1746" role="doc-biblioref">2002</a>)</span> was used to visualize shape variation among the bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(Klingenberg <a href="#ref-RN1731" role="doc-biblioref">2013</a>; Sherratt et al. <a href="#ref-RN479" role="doc-biblioref">2014</a>)</span>.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="symmetry-and-asymmetry.html#cb15-1"></a><span class="co"># geomorph data frame</span></span>
<span id="cb15-2"><a href="symmetry-and-asymmetry.html#cb15-2"></a>gdf &lt;-<span class="st"> </span><span class="kw">geomorph.data.frame</span>(<span class="dt">shape =</span> res.bilat<span class="op">$</span>symm.shape,</span>
<span id="cb15-3"><a href="symmetry-and-asymmetry.html#cb15-3"></a>                           <span class="dt">ashape =</span> res.bilat<span class="op">$</span>FA.component,</span>
<span id="cb15-4"><a href="symmetry-and-asymmetry.html#cb15-4"></a>                           <span class="dt">reg =</span> qdata<span class="op">$</span>region2,</span>
<span id="cb15-5"><a href="symmetry-and-asymmetry.html#cb15-5"></a>                           <span class="dt">mort =</span> qdata<span class="op">$</span>mortuary) </span>
<span id="cb15-6"><a href="symmetry-and-asymmetry.html#cb15-6"></a></span>
<span id="cb15-7"><a href="symmetry-and-asymmetry.html#cb15-7"></a><span class="co"># attributes</span></span>
<span id="cb15-8"><a href="symmetry-and-asymmetry.html#cb15-8"></a>reg &lt;-<span class="st"> </span>qdata<span class="op">$</span>region2</span>
<span id="cb15-9"><a href="symmetry-and-asymmetry.html#cb15-9"></a>mort &lt;-<span class="st"> </span>qdata<span class="op">$</span>mortuary</span>
<span id="cb15-10"><a href="symmetry-and-asymmetry.html#cb15-10"></a></span>
<span id="cb15-11"><a href="symmetry-and-asymmetry.html#cb15-11"></a><span class="co"># principal components analysis (symmetric)</span></span>
<span id="cb15-12"><a href="symmetry-and-asymmetry.html#cb15-12"></a>pca &lt;-<span class="st"> </span><span class="kw">gm.prcomp</span>(res.bilat<span class="op">$</span>symm.shape)</span>
<span id="cb15-13"><a href="symmetry-and-asymmetry.html#cb15-13"></a><span class="kw">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 102 
## Number of vectors 102 
## 
## Importance of Components:
##                              Comp1        Comp2        Comp3        Comp4
## Eigenvalues            0.003404138 0.0007768963 0.0003500284 0.0000986722
## Proportion of Variance 0.705138813 0.1609275863 0.0725054583 0.0204391226
## Cumulative Proportion  0.705138813 0.8660663993 0.9385718576 0.9590109801
##                               Comp5        Comp6        Comp7        Comp8
## Eigenvalues            0.0000606932 4.555788e-05 1.697868e-05 1.226922e-05
## Proportion of Variance 0.0125720895 9.436934e-03 3.516992e-03 2.541467e-03
## Cumulative Proportion  0.9715830696 9.810200e-01 9.845370e-01 9.870785e-01
##                               Comp9       Comp10       Comp11       Comp12
## Eigenvalues            9.280989e-06 6.433906e-06 5.793873e-06 5.109631e-06
## Proportion of Variance 1.922479e-03 1.332730e-03 1.200152e-03 1.058417e-03
## Cumulative Proportion  9.890009e-01 9.903337e-01 9.915338e-01 9.925922e-01
##                              Comp13       Comp14       Comp15       Comp16
## Eigenvalues            3.346469e-06 2.765029e-06 2.693003e-06 2.391320e-06
## Proportion of Variance 6.931932e-04 5.727527e-04 5.578330e-04 4.953420e-04
## Cumulative Proportion  9.932854e-01 9.938582e-01 9.944160e-01 9.949114e-01
##                              Comp17       Comp18       Comp19       Comp20
## Eigenvalues            2.132892e-06 1.867393e-06 1.695757e-06 1.567577e-06
## Proportion of Variance 4.418107e-04 3.868148e-04 3.512620e-04 3.247105e-04
## Cumulative Proportion  9.953532e-01 9.957400e-01 9.960912e-01 9.964160e-01
##                              Comp21       Comp22       Comp23       Comp24
## Eigenvalues            1.446555e-06 1.265297e-06 1.174284e-06 1.045820e-06
## Proportion of Variance 2.996417e-04 2.620957e-04 2.432431e-04 2.166330e-04
## Cumulative Proportion  9.967156e-01 9.969777e-01 9.972209e-01 9.974376e-01
##                              Comp25       Comp26       Comp27       Comp28
## Eigenvalues            1.010178e-06 9.557273e-07 8.419270e-07 7.866692e-07
## Proportion of Variance 2.092500e-04 1.979709e-04 1.743981e-04 1.629520e-04
## Cumulative Proportion  9.976468e-01 9.978448e-01 9.980192e-01 9.981821e-01
##                              Comp29       Comp30       Comp31       Comp32
## Eigenvalues            7.686443e-07 6.716027e-07 6.089177e-07 5.852480e-07
## Proportion of Variance 1.592183e-04 1.391169e-04 1.261322e-04 1.212292e-04
## Cumulative Proportion  9.983414e-01 9.984805e-01 9.986066e-01 9.987278e-01
##                              Comp33       Comp34       Comp35       Comp36
## Eigenvalues            5.588653e-07 5.166258e-07 4.603312e-07 4.363582e-07
## Proportion of Variance 1.157643e-04 1.070147e-04 9.535376e-05 9.038796e-05
## Cumulative Proportion  9.988436e-01 9.989506e-01 9.990460e-01 9.991364e-01
##                              Comp37       Comp38       Comp39       Comp40
## Eigenvalues            4.171225e-07 3.499439e-07 3.277035e-07 3.004796e-07
## Proportion of Variance 8.640343e-05 7.248795e-05 6.788103e-05 6.224184e-05
## Cumulative Proportion  9.992228e-01 9.992953e-01 9.993631e-01 9.994254e-01
##                              Comp41       Comp42       Comp43       Comp44
## Eigenvalues            2.987827e-07 2.603766e-07 2.109061e-07 1.978412e-07
## Proportion of Variance 6.189035e-05 5.393483e-05 4.368745e-05 4.098115e-05
## Cumulative Proportion  9.994873e-01 9.995412e-01 9.995849e-01 9.996259e-01
##                              Comp45       Comp46       Comp47       Comp48
## Eigenvalues            1.953721e-07 1.775082e-07 1.626235e-07 1.456227e-07
## Proportion of Variance 4.046969e-05 3.676933e-05 3.368610e-05 3.016452e-05
## Cumulative Proportion  9.996663e-01 9.997031e-01 9.997368e-01 9.997670e-01
##                              Comp49       Comp50       Comp51       Comp52
## Eigenvalues            1.306448e-07 1.143209e-07 1.050670e-07 9.213378e-08
## Proportion of Variance 2.706197e-05 2.368062e-05 2.176375e-05 1.908474e-05
## Cumulative Proportion  9.997940e-01 9.998177e-01 9.998395e-01 9.998586e-01
##                              Comp53       Comp54       Comp55       Comp56
## Eigenvalues            9.082021e-08 7.848003e-08 7.442553e-08 6.001279e-08
## Proportion of Variance 1.881265e-05 1.625648e-05 1.541663e-05 1.243115e-05
## Cumulative Proportion  9.998774e-01 9.998936e-01 9.999090e-01 9.999215e-01
##                              Comp57       Comp58       Comp59       Comp60
## Eigenvalues            5.397617e-08 4.571620e-08 3.921471e-08 3.621184e-08
## Proportion of Variance 1.118071e-05 9.469730e-06 8.122999e-06 7.500980e-06
## Cumulative Proportion  9.999326e-01 9.999421e-01 9.999502e-01 9.999577e-01
##                              Comp61       Comp62       Comp63       Comp64
## Eigenvalues            3.008465e-08 2.753055e-08 2.490676e-08 1.967315e-08
## Proportion of Variance 6.231783e-06 5.702724e-06 5.159227e-06 4.075129e-06
## Cumulative Proportion  9.999640e-01 9.999697e-01 9.999748e-01 9.999789e-01
##                              Comp65       Comp66       Comp67       Comp68
## Eigenvalues            1.852310e-08 1.635542e-08 1.277850e-08 1.091701e-08
## Proportion of Variance 3.836905e-06 3.387889e-06 2.646960e-06 2.261366e-06
## Cumulative Proportion  9.999827e-01 9.999861e-01 9.999888e-01 9.999910e-01
##                              Comp69       Comp70       Comp71       Comp72
## Eigenvalues            8.754168e-09 6.950731e-09 6.117323e-09 5.097733e-09
## Proportion of Variance 1.813353e-06 1.439786e-06 1.267152e-06 1.055953e-06
## Cumulative Proportion  9.999929e-01 9.999943e-01 9.999956e-01 9.999966e-01
##                              Comp73       Comp74       Comp75       Comp76
## Eigenvalues            4.381224e-09 3.414802e-09 3.188546e-09 2.360309e-09
## Proportion of Variance 9.075341e-07 7.073477e-07 6.604806e-07 4.889183e-07
## Cumulative Proportion  9.999975e-01 9.999982e-01 9.999989e-01 9.999994e-01
##                              Comp77       Comp78       Comp79       Comp80
## Eigenvalues            1.709143e-09 1.261802e-09 3.191757e-17 5.194158e-18
## Proportion of Variance 3.540347e-07 2.613717e-07 6.611457e-15 1.075926e-15
## Cumulative Proportion  9.999997e-01 1.000000e+00 1.000000e+00 1.000000e+00
##                              Comp81       Comp82       Comp83       Comp84
## Eigenvalues            2.429745e-18 8.738194e-19 8.014232e-19 4.431625e-19
## Proportion of Variance 5.033015e-16 1.810044e-16 1.660081e-16 9.179741e-17
## Cumulative Proportion  1.000000e+00 1.000000e+00 1.000000e+00 1.000000e+00
##                              Comp85       Comp86       Comp87       Comp88
## Eigenvalues            2.410115e-19 1.671418e-19 1.645101e-19 1.103698e-19
## Proportion of Variance 4.992352e-17 3.462203e-17 3.407688e-17 2.286218e-17
## Cumulative Proportion  1.000000e+00 1.000000e+00 1.000000e+00 1.000000e+00
##                              Comp89       Comp90       Comp91       Comp92
## Eigenvalues            9.304998e-20 6.910188e-20 6.184955e-20 5.256365e-20
## Proportion of Variance 1.927453e-17 1.431388e-17 1.281162e-17 1.088812e-17
## Cumulative Proportion  1.000000e+00 1.000000e+00 1.000000e+00 1.000000e+00
##                              Comp93       Comp94       Comp95       Comp96
## Eigenvalues            4.831758e-20 3.043512e-20 2.047550e-20 1.445427e-20
## Proportion of Variance 1.000858e-17 6.304381e-18 4.241328e-18 2.994082e-18
## Cumulative Proportion  1.000000e+00 1.000000e+00 1.000000e+00 1.000000e+00
##                              Comp97       Comp98       Comp99      Comp100
## Eigenvalues            1.354425e-20 9.436147e-21 7.354838e-21 6.099898e-21
## Proportion of Variance 2.805578e-18 1.954619e-18 1.523493e-18 1.263543e-18
## Cumulative Proportion  1.000000e+00 1.000000e+00 1.000000e+00 1.000000e+00
##                             Comp101      Comp102
## Eigenvalues            4.238130e-21 1.295152e-33
## Proportion of Variance 8.778932e-19 2.682798e-31
## Cumulative Proportion  1.000000e+00 1.000000e+00</code></pre>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="symmetry-and-asymmetry.html#cb17-1"></a><span class="co"># set plot parameters to plot by region</span></span>
<span id="cb17-2"><a href="symmetry-and-asymmetry.html#cb17-2"></a>pch.gps.reg &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">15</span>,<span class="dv">17</span>,<span class="dv">18</span>)[<span class="kw">as.factor</span>(reg)]</span>
<span id="cb17-3"><a href="symmetry-and-asymmetry.html#cb17-3"></a>col.gps.reg &lt;-<span class="st"> </span><span class="kw">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)[<span class="kw">as.factor</span>(reg)]</span>
<span id="cb17-4"><a href="symmetry-and-asymmetry.html#cb17-4"></a>col.hull<span class="fl">.2</span> &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;#798E87&quot;</span>,<span class="st">&quot;#C27D38&quot;</span>,<span class="st">&quot;#CCC591&quot;</span>)</span>
<span id="cb17-5"><a href="symmetry-and-asymmetry.html#cb17-5"></a></span>
<span id="cb17-6"><a href="symmetry-and-asymmetry.html#cb17-6"></a><span class="co"># set plot parameters to plot by mortuary context</span></span>
<span id="cb17-7"><a href="symmetry-and-asymmetry.html#cb17-7"></a>pch.gps.mort &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">15</span>,<span class="dv">17</span>)[<span class="kw">as.factor</span>(mort)]</span>
<span id="cb17-8"><a href="symmetry-and-asymmetry.html#cb17-8"></a>col.gps.mort &lt;-<span class="st"> </span><span class="kw">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)[<span class="kw">as.factor</span>(mort)]</span>
<span id="cb17-9"><a href="symmetry-and-asymmetry.html#cb17-9"></a>col.hull.mort &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;#798E87&quot;</span>,<span class="st">&quot;#C27D38&quot;</span>)</span></code></pre></div>
</div>
<div id="define-models" class="section level2">
<h2><span class="header-section-number">7.5</span> Define models</h2>
<div id="hypothesis-1" class="section level3">
<h3><span class="header-section-number">7.5.1</span> Hypothesis 1</h3>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="symmetry-and-asymmetry.html#cb18-1"></a><span class="co"># symmetric shape as a function of region</span></span>
<span id="cb18-2"><a href="symmetry-and-asymmetry.html#cb18-2"></a><span class="co">## (differences in design intent?)</span></span>
<span id="cb18-3"><a href="symmetry-and-asymmetry.html#cb18-3"></a>reg.symm &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>reg, </span>
<span id="cb18-4"><a href="symmetry-and-asymmetry.html#cb18-4"></a>                     <span class="dt">data =</span> gdf, </span>
<span id="cb18-5"><a href="symmetry-and-asymmetry.html#cb18-5"></a>                     <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb18-6"><a href="symmetry-and-asymmetry.html#cb18-6"></a>                     <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb18-7"><a href="symmetry-and-asymmetry.html#cb18-7"></a></span>
<span id="cb18-8"><a href="symmetry-and-asymmetry.html#cb18-8"></a><span class="co"># asymmetric shape as a function of region</span></span>
<span id="cb18-9"><a href="symmetry-and-asymmetry.html#cb18-9"></a><span class="co">## (standardization?)</span></span>
<span id="cb18-10"><a href="symmetry-and-asymmetry.html#cb18-10"></a>reg.asymm &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(ashape <span class="op">~</span><span class="st"> </span>reg, </span>
<span id="cb18-11"><a href="symmetry-and-asymmetry.html#cb18-11"></a>                      <span class="dt">data =</span> gdf, </span>
<span id="cb18-12"><a href="symmetry-and-asymmetry.html#cb18-12"></a>                      <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb18-13"><a href="symmetry-and-asymmetry.html#cb18-13"></a>                      <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
</div>
<div id="hypothesis-2" class="section level3">
<h3><span class="header-section-number">7.5.2</span> Hypothesis 2</h3>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="symmetry-and-asymmetry.html#cb19-1"></a><span class="co"># symmetric shape as a function of mortuary context</span></span>
<span id="cb19-2"><a href="symmetry-and-asymmetry.html#cb19-2"></a><span class="co">## (differences in design intent?)</span></span>
<span id="cb19-3"><a href="symmetry-and-asymmetry.html#cb19-3"></a>mort.symm &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>mort, </span>
<span id="cb19-4"><a href="symmetry-and-asymmetry.html#cb19-4"></a>                     <span class="dt">data =</span> gdf, </span>
<span id="cb19-5"><a href="symmetry-and-asymmetry.html#cb19-5"></a>                     <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb19-6"><a href="symmetry-and-asymmetry.html#cb19-6"></a>                     <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb19-7"><a href="symmetry-and-asymmetry.html#cb19-7"></a></span>
<span id="cb19-8"><a href="symmetry-and-asymmetry.html#cb19-8"></a><span class="co"># asymmetric shape as a function of mortuary context</span></span>
<span id="cb19-9"><a href="symmetry-and-asymmetry.html#cb19-9"></a><span class="co">## (standardization?)</span></span>
<span id="cb19-10"><a href="symmetry-and-asymmetry.html#cb19-10"></a>mort.asymm &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(ashape <span class="op">~</span><span class="st"> </span>mort, </span>
<span id="cb19-11"><a href="symmetry-and-asymmetry.html#cb19-11"></a>                      <span class="dt">data =</span> gdf, </span>
<span id="cb19-12"><a href="symmetry-and-asymmetry.html#cb19-12"></a>                      <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb19-13"><a href="symmetry-and-asymmetry.html#cb19-13"></a>                      <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
</div>
</div>
<div id="hypothesis-1-1" class="section level2">
<h2><span class="header-section-number">7.6</span> Hypothesis 1</h2>
<p>Assessment of <em>design intent</em> and <em>standardization</em> by region.</p>
<div id="pca-h1" class="section level3">
<h3><span class="header-section-number">7.6.1</span> PCA (H1)</h3>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="symmetry-and-asymmetry.html#cb20-1"></a><span class="co"># plot pca (symmetric) by region</span></span>
<span id="cb20-2"><a href="symmetry-and-asymmetry.html#cb20-2"></a>pc.plot &lt;-<span class="st"> </span><span class="kw">plot</span>(pca, </span>
<span id="cb20-3"><a href="symmetry-and-asymmetry.html#cb20-3"></a>                 <span class="dt">asp =</span> <span class="dv">1</span>,</span>
<span id="cb20-4"><a href="symmetry-and-asymmetry.html#cb20-4"></a>                 <span class="dt">pch =</span> pch.gps.reg,</span>
<span id="cb20-5"><a href="symmetry-and-asymmetry.html#cb20-5"></a>                 <span class="dt">col =</span> col.gps.reg)</span>
<span id="cb20-6"><a href="symmetry-and-asymmetry.html#cb20-6"></a>                    <span class="kw">shapeHulls</span>(pc.plot, </span>
<span id="cb20-7"><a href="symmetry-and-asymmetry.html#cb20-7"></a>                               <span class="dt">groups =</span> reg,</span>
<span id="cb20-8"><a href="symmetry-and-asymmetry.html#cb20-8"></a>                               <span class="dt">group.cols =</span> col.hull<span class="fl">.2</span>)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/symm.pca-1.png" alt="_**Figure 5. PCA (symmetric) for SCA-north (orange triangles), SCA-south (tan diamonds), and central Texas (gray squares).**_" width="100%" />
<p class="caption">
(#fig:symm.pca)<em><strong>Figure 5. PCA (symmetric) for SCA-north (orange triangles), SCA-south (tan diamonds), and central Texas (gray squares).</strong></em>
</p>
</div>
</div>
<div id="procrustes-anova-h1" class="section level3">
<h3><span class="header-section-number">7.6.2</span> Procrustes ANOVA (H1)</h3>
<p><em>Are there differences in the symmetric shapes of Gahagan bifaces found in the southern Caddo area (north/south) and central Texas?</em></p>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for the Procrustes ANOVA <span class="citation">(Adams and Collyer <a href="#ref-RN1655" role="doc-biblioref">2015</a>; Collyer and Adams <a href="#ref-RN11775" role="doc-biblioref">2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(Anderson and Ter Braak <a href="#ref-RN1719" role="doc-biblioref">2003</a>)</span>. To assess whether symmetric shape differs by group (region), Procrustes ANOVAs <span class="citation">(Goodall <a href="#ref-RN1749" role="doc-biblioref">1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(Collyer, Sekora, and Adams <a href="#ref-RN1756" role="doc-biblioref">2015</a>)</span>.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="symmetry-and-asymmetry.html#cb21-1"></a><span class="co"># ANOVA: do symmetric gahagan biface point shapes differ by region?</span></span>
<span id="cb21-2"><a href="symmetry-and-asymmetry.html#cb21-2"></a><span class="co">## (are there differences in design intent?)</span></span>
<span id="cb21-3"><a href="symmetry-and-asymmetry.html#cb21-3"></a><span class="kw">anova</span>(reg.symm)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## reg         2 0.16520 0.082600 0.33881 25.365 5.4485  1e-04 ***
## Residuals  99 0.32239 0.003256 0.66119                         
## Total     101 0.48759                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ reg, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="symmetry-and-asymmetry.html#cb23-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb23-2"><a href="symmetry-and-asymmetry.html#cb23-2"></a>symm.reg &lt;-<span class="st"> </span><span class="kw">pairwise</span>(reg.symm,</span>
<span id="cb23-3"><a href="symmetry-and-asymmetry.html#cb23-3"></a>                     <span class="dt">groups =</span> qdata<span class="op">$</span>region2)</span>
<span id="cb23-4"><a href="symmetry-and-asymmetry.html#cb23-4"></a><span class="kw">summary</span>(symm.reg, </span>
<span id="cb23-5"><a href="symmetry-and-asymmetry.html#cb23-5"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb23-6"><a href="symmetry-and-asymmetry.html#cb23-6"></a>        <span class="dt">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                              d  UCL (95%)        Z Pr &gt; d
## CTX:SCA-north       0.12060397 0.04024245 4.307016  1e-04
## CTX:SCA-south       0.05105117 0.02736041 3.102597  1e-04
## SCA-north:SCA-south 0.09251518 0.03740856 3.770890  1e-04</code></pre>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="symmetry-and-asymmetry.html#cb25-1"></a><span class="co"># pairwise distance between variances = standardization?</span></span>
<span id="cb25-2"><a href="symmetry-and-asymmetry.html#cb25-2"></a><span class="kw">summary</span>(symm.reg, </span>
<span id="cb25-3"><a href="symmetry-and-asymmetry.html#cb25-3"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb25-4"><a href="symmetry-and-asymmetry.html#cb25-4"></a>        <span class="dt">test.type =</span> <span class="st">&quot;var&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## 
## Observed variances by group
## 
##          CTX    SCA-north    SCA-south 
## 0.0040131894 0.0008460774 0.0032371786 
## 
## Pairwise distances between variances, plus statistics
##                                d   UCL (95%)         Z Pr &gt; d
## CTX:SCA-north       0.0031671121 0.002780193 1.8208235 0.0281
## CTX:SCA-south       0.0007760108 0.001848097 0.2061869 0.4424
## SCA-north:SCA-south 0.0023911013 0.002500175 1.4669611 0.0600</code></pre>
</div>
<div id="mean-symmetric-shapes-h1" class="section level3">
<h3><span class="header-section-number">7.6.3</span> Mean Symmetric Shapes (H1)</h3>
<p>Mean symmetric shapes were rendered and contrasted for each region, and illustrate those morphological differences that occur between each. These symmetric shapes may represent regional differences in <em>design intent</em> related to the conceptual aesthetic of Gahagan bifaces, which differs significantly between all three regions (see above). Each region may also articulate with a period of discrete shape change in Gahagan biface morphology that began in the SCA-north, continued through the SCA-south, and terminated in central Texas (inference based upon the <a href="https://aksel-blaise.github.io/gahaganmorph3/temporal-dynamics.html#chronological-model-for-sca-north-sca-south-and-central-texas-contexts-yielding-gahagan-bifaces-with-baker-site-dates">chronological model</a>).</p>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="symmetry-and-asymmetry.html#cb27-1"></a><span class="co"># subset landmark coordinates to produce mean shapes for groups</span></span>
<span id="cb27-2"><a href="symmetry-and-asymmetry.html#cb27-2"></a><span class="co">## (design intent)</span></span>
<span id="cb27-3"><a href="symmetry-and-asymmetry.html#cb27-3"></a>new.coords &lt;-<span class="st"> </span><span class="kw">coords.subset</span>(<span class="dt">A =</span> res.bilat<span class="op">$</span>symm.shape, </span>
<span id="cb27-4"><a href="symmetry-and-asymmetry.html#cb27-4"></a>                            <span class="dt">group =</span> qdata<span class="op">$</span>region2)</span>
<span id="cb27-5"><a href="symmetry-and-asymmetry.html#cb27-5"></a><span class="kw">names</span>(new.coords)</span></code></pre></div>
<pre><code>## [1] &quot;CTX&quot;       &quot;SCA-north&quot; &quot;SCA-south&quot;</code></pre>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="symmetry-and-asymmetry.html#cb29-1"></a><span class="co"># group shape means</span></span>
<span id="cb29-2"><a href="symmetry-and-asymmetry.html#cb29-2"></a>mean &lt;-<span class="st"> </span><span class="kw">lapply</span>(new.coords, mshape)</span>
<span id="cb29-3"><a href="symmetry-and-asymmetry.html#cb29-3"></a></span>
<span id="cb29-4"><a href="symmetry-and-asymmetry.html#cb29-4"></a><span class="co"># plot mean shapes</span></span>
<span id="cb29-5"><a href="symmetry-and-asymmetry.html#cb29-5"></a><span class="kw">plot</span>(mean<span class="op">$</span>CTX)</span>
<span id="cb29-6"><a href="symmetry-and-asymmetry.html#cb29-6"></a><span class="kw">plot</span>(mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-north</span><span class="st">`</span>)</span>
<span id="cb29-7"><a href="symmetry-and-asymmetry.html#cb29-7"></a><span class="kw">plot</span>(mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-south</span><span class="st">`</span>)</span>
<span id="cb29-8"><a href="symmetry-and-asymmetry.html#cb29-8"></a></span>
<span id="cb29-9"><a href="symmetry-and-asymmetry.html#cb29-9"></a><span class="co"># comparison plots</span></span>
<span id="cb29-10"><a href="symmetry-and-asymmetry.html#cb29-10"></a><span class="kw">plotRefToTarget</span>(mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-north</span><span class="st">`</span>,</span>
<span id="cb29-11"><a href="symmetry-and-asymmetry.html#cb29-11"></a>                mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-south</span><span class="st">`</span>, </span>
<span id="cb29-12"><a href="symmetry-and-asymmetry.html#cb29-12"></a>                <span class="dt">method =</span> <span class="st">&quot;vector&quot;</span>,</span>
<span id="cb29-13"><a href="symmetry-and-asymmetry.html#cb29-13"></a>                <span class="dt">mag =</span> <span class="dv">2</span>)</span>
<span id="cb29-14"><a href="symmetry-and-asymmetry.html#cb29-14"></a><span class="kw">plotRefToTarget</span>(mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-north</span><span class="st">`</span>,</span>
<span id="cb29-15"><a href="symmetry-and-asymmetry.html#cb29-15"></a>                mean<span class="op">$</span>CTX, </span>
<span id="cb29-16"><a href="symmetry-and-asymmetry.html#cb29-16"></a>                <span class="dt">method =</span> <span class="st">&quot;vector&quot;</span>,</span>
<span id="cb29-17"><a href="symmetry-and-asymmetry.html#cb29-17"></a>                <span class="dt">mag =</span> <span class="dv">2</span>)</span>
<span id="cb29-18"><a href="symmetry-and-asymmetry.html#cb29-18"></a><span class="kw">plotRefToTarget</span>(mean<span class="op">$</span><span class="st">`</span><span class="dt">SCA-south</span><span class="st">`</span>,</span>
<span id="cb29-19"><a href="symmetry-and-asymmetry.html#cb29-19"></a>                mean<span class="op">$</span>CTX, </span>
<span id="cb29-20"><a href="symmetry-and-asymmetry.html#cb29-20"></a>                <span class="dt">method =</span> <span class="st">&quot;vector&quot;</span>,</span>
<span id="cb29-21"><a href="symmetry-and-asymmetry.html#cb29-21"></a>                <span class="dt">mag =</span> <span class="dv">2</span>)</span>
<span id="cb29-22"><a href="symmetry-and-asymmetry.html#cb29-22"></a></span>
<span id="cb29-23"><a href="symmetry-and-asymmetry.html#cb29-23"></a>knitr<span class="op">::</span><span class="kw">include_graphics</span>(<span class="st">&#39;images/mshape.symmetric.jpg&#39;</span>)</span></code></pre></div>
<div class="figure">
<img src="images/mshape.symmetric.jpg" alt="_**Figure 6. Comparison of mean symmetric shapes for Gahagan bifaces from the central Texas (CTX), southern Caddo area - South (SCA-South), and southern Caddo area - North (SCA-North) regions. The mean for each row is represented by gray spheres, which is compared with those shapes in each column using linear vectors.**_" width="100%" />
<p class="caption">
(#fig:m.symm.shapes)<em><strong>Figure 6. Comparison of mean symmetric shapes for Gahagan bifaces from the central Texas (CTX), southern Caddo area - South (SCA-South), and southern Caddo area - North (SCA-North) regions. The mean for each row is represented by gray spheres, which is compared with those shapes in each column using linear vectors.</strong></em>
</p>
</div>
</div>
<div id="morphological-disparity-h1" class="section level3">
<h3><span class="header-section-number">7.6.4</span> Morphological disparity (H1)</h3>
<p><em>Does fluctuating asymmetry (standardization) vary between regions?</em></p>
<p>Morphological disparity is estimated as the Procrustes variance, for groups in this instance, using residuals of a linear model fit <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. Procrustes variance is the same sum of the diagonal elements of the group covariance matrix divided by the number of observations in the group <span class="citation">(Zelditch et al. <a href="#ref-RN5703" role="doc-biblioref">2004</a>; Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. Absolute differences in Procrustes variances are test statistics that can be used to test differences in morphological disparity among groups <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. These differences are statistically evaluated through permutation, where the vectors of residuals are randomized among groups <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>.</p>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="symmetry-and-asymmetry.html#cb30-1"></a><span class="co"># morphological disparity: do any of the groups display greater asymmetric </span></span>
<span id="cb30-2"><a href="symmetry-and-asymmetry.html#cb30-2"></a><span class="co"># shape variation among individuals relative to the other group?</span></span>
<span id="cb30-3"><a href="symmetry-and-asymmetry.html#cb30-3"></a><span class="co">## (standardization?)</span></span>
<span id="cb30-4"><a href="symmetry-and-asymmetry.html#cb30-4"></a><span class="kw">morphol.disparity</span>(reg.asymm, </span>
<span id="cb30-5"><a href="symmetry-and-asymmetry.html#cb30-5"></a>                  <span class="dt">groups =</span> qdata<span class="op">$</span>region2, </span>
<span id="cb30-6"><a href="symmetry-and-asymmetry.html#cb30-6"></a>                  <span class="dt">data =</span> gdf, </span>
<span id="cb30-7"><a href="symmetry-and-asymmetry.html#cb30-7"></a>                  <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb30-8"><a href="symmetry-and-asymmetry.html#cb30-8"></a>                  <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = reg.asymm, groups = qdata$region2, iter = 9999,  
##     data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##          CTX    SCA-north    SCA-south 
## 0.0038990210 0.0008669788 0.0020447937 
## 
## 
## Pairwise absolute differences between variances
##                   CTX   SCA-north   SCA-south
## CTX       0.000000000 0.003032042 0.001854227
## SCA-north 0.003032042 0.000000000 0.001177815
## SCA-south 0.001854227 0.001177815 0.000000000
## 
## 
## P-Values
##              CTX SCA-north SCA-south
## CTX       1.0000    0.0013     0.002
## SCA-north 0.0013    1.0000     0.142
## SCA-south 0.0020    0.1420     1.000</code></pre>
</div>
</div>
<div id="hypothesis-2-1" class="section level2">
<h2><span class="header-section-number">7.7</span> Hypothesis 2</h2>
<p>Assessment of <em>design intent</em> and <em>standardization</em> by mortuary context.</p>
<div id="principal-components-analysis-h2" class="section level3">
<h3><span class="header-section-number">7.7.1</span> Principal Components Analysis (H2)</h3>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="symmetry-and-asymmetry.html#cb32-1"></a><span class="co"># plot pca (symmetric) by region</span></span>
<span id="cb32-2"><a href="symmetry-and-asymmetry.html#cb32-2"></a>pc.plot2 &lt;-<span class="st"> </span><span class="kw">plot</span>(pca, </span>
<span id="cb32-3"><a href="symmetry-and-asymmetry.html#cb32-3"></a>                 <span class="dt">asp =</span> <span class="dv">1</span>,</span>
<span id="cb32-4"><a href="symmetry-and-asymmetry.html#cb32-4"></a>                 <span class="dt">pch =</span> pch.gps.mort,</span>
<span id="cb32-5"><a href="symmetry-and-asymmetry.html#cb32-5"></a>                 <span class="dt">col =</span> col.gps.mort)</span>
<span id="cb32-6"><a href="symmetry-and-asymmetry.html#cb32-6"></a>                    <span class="kw">shapeHulls</span>(pc.plot2, </span>
<span id="cb32-7"><a href="symmetry-and-asymmetry.html#cb32-7"></a>                               <span class="dt">groups =</span> mort,</span>
<span id="cb32-8"><a href="symmetry-and-asymmetry.html#cb32-8"></a>                               <span class="dt">group.cols =</span> col.hull.mort)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/symm.pca.2-1.png" alt="_**Figure 5. PCA (symmetric) for SCA-north (orange triangles), SCA-south (tan diamonds), and central Texas (gray squares).**_" width="100%" />
<p class="caption">
(#fig:symm.pca.2)<em><strong>Figure 5. PCA (symmetric) for SCA-north (orange triangles), SCA-south (tan diamonds), and central Texas (gray squares).</strong></em>
</p>
</div>
</div>
<div id="procrustes-anova-h2" class="section level3">
<h3><span class="header-section-number">7.7.2</span> Procrustes ANOVA (H2)</h3>
<p><em>Are there differences in the symmetric shapes of Gahagan bifaces found in/out of mortuary contexts?</em></p>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for the Procrustes ANOVA <span class="citation">(Adams and Collyer <a href="#ref-RN1655" role="doc-biblioref">2015</a>; Collyer and Adams <a href="#ref-RN11775" role="doc-biblioref">2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(Anderson and Ter Braak <a href="#ref-RN1719" role="doc-biblioref">2003</a>)</span>. To assess whether symmetric shape differs by group (region), Procrustes ANOVAs <span class="citation">(Goodall <a href="#ref-RN1749" role="doc-biblioref">1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(Collyer, Sekora, and Adams <a href="#ref-RN1756" role="doc-biblioref">2015</a>)</span>.</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="symmetry-and-asymmetry.html#cb33-1"></a><span class="co"># ANOVA: do symmetric gahagan biface point shapes differ by mortuary context?</span></span>
<span id="cb33-2"><a href="symmetry-and-asymmetry.html#cb33-2"></a><span class="co">## (are there differences in design intent?)</span></span>
<span id="cb33-3"><a href="symmetry-and-asymmetry.html#cb33-3"></a><span class="kw">anova</span>(mort.symm)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## mort        1 0.04948 0.049481 0.10148 11.294 3.1181  1e-04 ***
## Residuals 100 0.43811 0.004381 0.89852                         
## Total     101 0.48759                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ mort, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
<div id="mean-asymmetric-shapes-h2" class="section level3">
<h3><span class="header-section-number">7.7.3</span> Mean Asymmetric Shapes (H2)</h3>
<p>Mean asymmetric shapes were rendered and contrasted for Gahagan bifaces recovered in/out of mortuary contexts, and illustrate those morphological differences that occur between each. These symmetric shapes may represent differences in <em>design intent</em> related to the conceptual aesthetic of Gahagan bifaces, which differs significantly between mortuary contexts (see above).</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="symmetry-and-asymmetry.html#cb35-1"></a><span class="co"># subset landmark coordinates to produce mean shapes for groups</span></span>
<span id="cb35-2"><a href="symmetry-and-asymmetry.html#cb35-2"></a><span class="co">## (design intent)</span></span>
<span id="cb35-3"><a href="symmetry-and-asymmetry.html#cb35-3"></a>new.coords2 &lt;-<span class="st"> </span><span class="kw">coords.subset</span>(<span class="dt">A =</span> res.bilat<span class="op">$</span>symm.shape,</span>
<span id="cb35-4"><a href="symmetry-and-asymmetry.html#cb35-4"></a>                             <span class="dt">group =</span> qdata<span class="op">$</span>mortuary)</span>
<span id="cb35-5"><a href="symmetry-and-asymmetry.html#cb35-5"></a></span>
<span id="cb35-6"><a href="symmetry-and-asymmetry.html#cb35-6"></a><span class="kw">names</span>(new.coords2)</span></code></pre></div>
<pre><code>## [1] &quot;no&quot;  &quot;yes&quot;</code></pre>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="symmetry-and-asymmetry.html#cb37-1"></a><span class="co"># group shape means</span></span>
<span id="cb37-2"><a href="symmetry-and-asymmetry.html#cb37-2"></a>mean2 &lt;-<span class="st"> </span><span class="kw">lapply</span>(new.coords2, mshape)</span>
<span id="cb37-3"><a href="symmetry-and-asymmetry.html#cb37-3"></a></span>
<span id="cb37-4"><a href="symmetry-and-asymmetry.html#cb37-4"></a><span class="co"># plot mean shapes</span></span>
<span id="cb37-5"><a href="symmetry-and-asymmetry.html#cb37-5"></a><span class="kw">plot</span>(mean2<span class="op">$</span>yes)</span>
<span id="cb37-6"><a href="symmetry-and-asymmetry.html#cb37-6"></a><span class="kw">plot</span>(mean2<span class="op">$</span>no)</span>
<span id="cb37-7"><a href="symmetry-and-asymmetry.html#cb37-7"></a></span>
<span id="cb37-8"><a href="symmetry-and-asymmetry.html#cb37-8"></a></span>
<span id="cb37-9"><a href="symmetry-and-asymmetry.html#cb37-9"></a><span class="co"># comparison plots</span></span>
<span id="cb37-10"><a href="symmetry-and-asymmetry.html#cb37-10"></a><span class="kw">plotRefToTarget</span>(mean2<span class="op">$</span>yes,</span>
<span id="cb37-11"><a href="symmetry-and-asymmetry.html#cb37-11"></a>                mean2<span class="op">$</span>no,</span>
<span id="cb37-12"><a href="symmetry-and-asymmetry.html#cb37-12"></a>                <span class="dt">method =</span> <span class="st">&quot;vector&quot;</span>,</span>
<span id="cb37-13"><a href="symmetry-and-asymmetry.html#cb37-13"></a>                <span class="dt">mag =</span> <span class="dv">2</span>)</span>
<span id="cb37-14"><a href="symmetry-and-asymmetry.html#cb37-14"></a></span>
<span id="cb37-15"><a href="symmetry-and-asymmetry.html#cb37-15"></a><span class="co">#knitr::include_graphics(&#39;images/mshape.symmetric.jpg&#39;)</span></span></code></pre></div>
</div>
<div id="morphological-disparity-h2" class="section level3">
<h3><span class="header-section-number">7.7.4</span> Morphological disparity (H2)</h3>
<p><em>Does fluctuating asymmetry vary more when Gahagan bifaces are found in/out of mortuary contexts?</em></p>
<p>Morphological disparity is estimated as the Procrustes variance, for groups in this instance, using residuals of a linear model fit <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. Procrustes variance is the same sum of the diagonal elements of the group covariance matrix divided by the number of observations in the group <span class="citation">(Zelditch et al. <a href="#ref-RN5703" role="doc-biblioref">2004</a>; Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. Absolute differences in Procrustes variances are test statistics that can be used to test differences in morphological disparity among groups <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>. These differences are statistically evaluated through permutation, where the vectors of residuals are randomized among groups <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>.</p>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="symmetry-and-asymmetry.html#cb38-1"></a><span class="co"># morphological disparity: do any of the groups display greater asymmetric </span></span>
<span id="cb38-2"><a href="symmetry-and-asymmetry.html#cb38-2"></a><span class="co"># shape variation among individuals relative to the other group?</span></span>
<span id="cb38-3"><a href="symmetry-and-asymmetry.html#cb38-3"></a><span class="co">## (standardization?)</span></span>
<span id="cb38-4"><a href="symmetry-and-asymmetry.html#cb38-4"></a><span class="kw">morphol.disparity</span>(mort.asymm, </span>
<span id="cb38-5"><a href="symmetry-and-asymmetry.html#cb38-5"></a>                  <span class="dt">groups =</span> qdata<span class="op">$</span>mortuary, </span>
<span id="cb38-6"><a href="symmetry-and-asymmetry.html#cb38-6"></a>                  <span class="dt">data =</span> gdf, </span>
<span id="cb38-7"><a href="symmetry-and-asymmetry.html#cb38-7"></a>                  <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb38-8"><a href="symmetry-and-asymmetry.html#cb38-8"></a>                  <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = mort.asymm, groups = qdata$mortuary, iter = 9999,  
##     data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##          no         yes 
## 0.003621620 0.001859079 
## 
## 
## Pairwise absolute differences between variances
##              no         yes
## no  0.000000000 0.001762541
## yes 0.001762541 0.000000000
## 
## 
## P-Values
##         no    yes
## no  1.0000 0.0014
## yes 0.0014 1.0000</code></pre>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-RN1655">
<p>Adams, Dean C., and Michael L. Collyer. 2015. “Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters.” <em>Evolution</em> 69 (3): 823–9. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.</p>
</div>
<div id="ref-RN11530">
<p>Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. “Package ’geomorph’: Geometric Morphometric Analyses of 2D/3D Landmark Data. R package version 3.0.5.” <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.</p>
</div>
<div id="ref-RN1774">
<p>Adams, Dean C., and Erik Otárola-Castillo. 2013. “geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data.” <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.</p>
</div>
<div id="ref-RN1719">
<p>Anderson, M. J., and C. J. F. Ter Braak. 2003. “Permutation Tests for Multi-Factoral Analysis of Variance.” <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.</p>
</div>
<div id="ref-RN11775">
<p>Collyer, Michael L., and Dean C. Adams. 2018. “RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization.” <em>Methods in Ecology and Evolution</em> 9 (7): 1772–9. <a href="https://doi.org/https://doi.org/10.1111/2041-210X.13029">https://doi.org/https://doi.org/10.1111/2041-210X.13029</a>.</p>
</div>
<div id="ref-RN1756">
<p>Collyer, M. L., D. J. Sekora, and D. C. Adams. 2015. “A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data.” <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.</p>
</div>
<div id="ref-RN1749">
<p>Goodall, Colin. 1991. “Procrustes Methods in the Statistical Analysis of Shape.” <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.</p>
</div>
<div id="ref-RN11564">
<p>Gower, J. C. 1975. “Generalized Procrustes Analysis.” <em>Psychometrika</em> 40 (1): 33–51. <a href="https://doi.org/https://doi.org/10.1007/BF02291478">https://doi.org/https://doi.org/10.1007/BF02291478</a>.</p>
</div>
<div id="ref-RN1746">
<p>Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.</p>
</div>
<div id="ref-RN11622">
<p>Kendall, David G. 1981. “The Statistics of Shape.” In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.</p>
</div>
<div id="ref-RN11623">
<p>Kendall, David G. 1984. “Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces.” <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.</p>
</div>
<div id="ref-RN1731">
<p>Klingenberg, Christian Peter. 2013. “Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes.” <em>Hystrix</em> 24 (1): 15–24. <a href="https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691">https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.</p>
</div>
<div id="ref-RN8579">
<p>Klingenberg, C. P., M. Barluenga, and A. Meyer. 2002. “Shape Analysis of Symmetric Structures: Quantifying Variation Among Individuals and Asymmetry.” Journal Article. <em>Evolution</em> 56 (10): 1909–20. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.0014-3820.2002.tb00117.x?sid=nlm%3Apubmed">https://onlinelibrary.wiley.com/doi/abs/10.1111/j.0014-3820.2002.tb00117.x?sid=nlm%3Apubmed</a>.</p>
</div>
<div id="ref-R">
<p>R Core Development Team, 2020. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</p>
</div>
<div id="ref-RN1646">
<p>Rohlf, F. James. 1999. “Shape Statistics: Procrustes Superimpositions and Tangent Spaces.” <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.</p>
</div>
<div id="ref-RN478">
<p>Rohlf, F. James, and Dennis Slice. 1990. “Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks.” <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.</p>
</div>
<div id="ref-RN479">
<p>Sherratt, E., D. J. Gower, C. P. Klingenberg, and M. Wilkinson. 2014. “Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona).” <em>Evolutionary Biology</em> 41 (4): 528–45. <a href="https://doi.org/https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/https://doi.org/10.1007/s11692-014-9287-2</a>.</p>
</div>
<div id="ref-RN11563">
<p>Slice, Dennis E. 2001. “Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space.” <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.</p>
</div>
<div id="ref-RN5703">
<p>Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="geometric-morphometrics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["gahaganmorph3.pdf", "gahaganmorph3.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
