<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Geometric morphometrics | Morphological differences in midline and cross-section profiles for Gahagan bifaces from central Texas and the southern Caddo area</title>
  <meta name="description" content="Gahagan biface shape change." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Geometric morphometrics | Morphological differences in midline and cross-section profiles for Gahagan bifaces from central Texas and the southern Caddo area" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Gahagan biface shape change." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Geometric morphometrics | Morphological differences in midline and cross-section profiles for Gahagan bifaces from central Texas and the southern Caddo area" />
  
  <meta name="twitter:description" content="Gahagan biface shape change." />
  

<meta name="author" content="Robert Z. Selden, Jr." />


<meta name="date" content="2021-01-19" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="landmark-sampling-evaluation-curve-lasec.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">gahaganmorph3</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminaries</a><ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.1</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.2</b> Funding</a></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.3</b> Data management</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html"><i class="fa fa-check"></i><b>1</b> Temporal Dynamics</a><ul>
<li class="chapter" data-level="1.1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-5-at-the-mounds-plantation-site"><i class="fa fa-check"></i><b>1.1</b> Calibration and plots for dates from Burial Pit 5 at the Mounds Plantation site</a></li>
<li class="chapter" data-level="1.2" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-2-at-the-gahagan-mound-site"><i class="fa fa-check"></i><b>1.2</b> Calibration and plots for dates from Burial Pit 2 at the Gahagan Mound site</a></li>
<li class="chapter" data-level="1.3" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-f119-and-f134-at-the-george-c.-davis-site"><i class="fa fa-check"></i><b>1.3</b> Calibration and plots for the dates from F119 and F134 at the George C. Davis site</a></li>
<li class="chapter" data-level="1.4" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-au2-at-the-j.-b.-white-site"><i class="fa fa-check"></i><b>1.4</b> Calibration and plots for dates from AU2 at the J. B. White site</a></li>
<li class="chapter" data-level="1.5" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-the-baker-site"><i class="fa fa-check"></i><b>1.5</b> Calibration and plots for the dates from the Baker site</a></li>
<li class="chapter" data-level="1.6" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#summed-probability-distributions-for-contexts-yielding-gahagan-bifaces"><i class="fa fa-check"></i><b>1.6</b> Summed probability distributions for contexts yielding Gahagan bifaces</a></li>
<li class="chapter" data-level="1.7" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#chronological-model-for-sca-north-sca-south-and-central-texas-contexts-yielding-gahagan-bifaces-with-baker-site-dates"><i class="fa fa-check"></i><b>1.7</b> Chronological model for SCA-north, SCA-south, and central Texas contexts yielding Gahagan bifaces with Baker site dates</a></li>
<li class="chapter" data-level="1.8" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#acknowledgments-1"><i class="fa fa-check"></i><b>1.8</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="modeling-geometry.html"><a href="modeling-geometry.html"><i class="fa fa-check"></i><b>2</b> Modeling Geometry</a><ul>
<li class="chapter" data-level="2.1" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section"><i class="fa fa-check"></i><b>2.1</b> 4078-8</a></li>
<li class="chapter" data-level="2.2" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section-1"><i class="fa fa-check"></i><b>2.2</b> 4078-72</a></li>
<li class="chapter" data-level="2.3" data-path="modeling-geometry.html"><a href="modeling-geometry.html#references-cited"><i class="fa fa-check"></i><b>2.3</b> References cited</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html"><i class="fa fa-check"></i><b>3</b> Landmarking Protocol 3d2</a><ul>
<li class="chapter" data-level="3.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#foundations"><i class="fa fa-check"></i><b>3.1</b> Foundations</a></li>
<li class="chapter" data-level="3.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#generating-the-peripheral-plan-view-spline"><i class="fa fa-check"></i><b>3.2</b> Generating the peripheral (plan view) spline</a><ul>
<li class="chapter" data-level="3.2.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-spline-1"><i class="fa fa-check"></i><b>3.2.1</b> Splitting Spline 1</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-geometry-build-1"><i class="fa fa-check"></i><b>3.3</b> Reference geometry, Build 1</a><ul>
<li class="chapter" data-level="3.3.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-vector-1-point-2-and-plane-2"><i class="fa fa-check"></i><b>3.3.1</b> Reference Vector 1, Point 2, and Plane 2</a></li>
<li class="chapter" data-level="3.3.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-planes-3-and-4"><i class="fa fa-check"></i><b>3.3.2</b> Reference Planes 3 and 4</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#cross-sections"><i class="fa fa-check"></i><b>3.4</b> Cross-sections</a><ul>
<li class="chapter" data-level="3.4.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-1"><i class="fa fa-check"></i><b>3.4.1</b> Splitting the curves (Step 1)</a></li>
<li class="chapter" data-level="3.4.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-2"><i class="fa fa-check"></i><b>3.4.2</b> Splitting the curves (Step 2)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#landmark-and-semilandmark-placement"><i class="fa fa-check"></i><b>3.5</b> Landmark and semilandmark placement</a></li>
<li class="chapter" data-level="3.6" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#acknowledgments-2"><i class="fa fa-check"></i><b>3.6</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html"><i class="fa fa-check"></i><b>4</b> Landmark Sampling Evaluation Curve (LaSEC)</a><ul>
<li class="chapter" data-level="4.1" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#current-3d-iteration-lm3d2"><i class="fa fa-check"></i><b>4.1</b> Current 3D iteration (LM3d2)</a></li>
<li class="chapter" data-level="4.2" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#acknowledgments-3"><i class="fa fa-check"></i><b>4.2</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html"><i class="fa fa-check"></i><b>5</b> Geometric morphometrics</a><ul>
<li class="chapter" data-level="5.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#load-geomorph-and-data"><i class="fa fa-check"></i><b>5.1</b> Load geomorph and data</a></li>
<li class="chapter" data-level="5.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#generalized-procrustes-analysis"><i class="fa fa-check"></i><b>5.2</b> Generalized Procrustes Analysis</a></li>
<li class="chapter" data-level="5.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#test-hypothesis"><i class="fa fa-check"></i><b>5.3</b> Test Hypothesis</a><ul>
<li class="chapter" data-level="5.3.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#boxplot"><i class="fa fa-check"></i><b>5.3.1</b> Boxplot</a></li>
<li class="chapter" data-level="5.3.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#principal-components-analysis"><i class="fa fa-check"></i><b>5.3.2</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="5.3.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#define-models"><i class="fa fa-check"></i><b>5.3.3</b> Define models</a></li>
<li class="chapter" data-level="5.3.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#allometry"><i class="fa fa-check"></i><b>5.3.4</b> Allometry</a></li>
<li class="chapter" data-level="5.3.5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#procrustes-anova"><i class="fa fa-check"></i><b>5.3.5</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="5.3.6" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity"><i class="fa fa-check"></i><b>5.3.6</b> Morphological disparity</a></li>
<li class="chapter" data-level="5.3.7" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#mean-shapes"><i class="fa fa-check"></i><b>5.3.7</b> Mean shapes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Morphological differences in midline and cross-section profiles for Gahagan bifaces from central Texas and the southern Caddo area</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="geometric-morphometrics" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Geometric morphometrics</h1>
<p>This effort follows recent analyses of Gahagan biface shapes across the southern Caddo area and central Texas <span class="citation">(Selden Jr., Dockall, and Shafer <a href="#ref-RN11783" role="doc-biblioref">2018</a>; Selden Jr., Dockall, and Dubied <a href="#ref-RN21001" role="doc-biblioref">2020</a>)</span>, where Gahgagan bifaces were found to differ in shape across the same geography as the Smithport Plain and Hickory Engraved bottles <span class="citation">(Selden Jr. <a href="#ref-RN11801" role="doc-biblioref">2018</a><a href="#ref-RN11801" role="doc-biblioref">a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018</a><a href="#ref-RN11782" role="doc-biblioref">b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN20697" role="doc-biblioref">2021</a>)</span>. The analysis builds upon previous discussions and analyses conducted by Shafer <span class="citation">(<a href="#ref-RN3684" role="doc-biblioref">1973</a>, <a href="#ref-RN20701" role="doc-biblioref">1974</a>, <a href="#ref-RN4924" role="doc-biblioref">2006</a>)</span>. A succinct overview of the analytical procedures used for this analysis is provided in the manuscript, and the analytical code provided in this document can be used to reproduce the results exactly.</p>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="geometric-morphometrics.html#cb56-1"></a>knitr<span class="op">::</span><span class="kw">include_graphics</span>(<span class="st">&#39;images/figure02.jpg&#39;</span>)</span></code></pre></div>
<div class="figure">
<img src="images/figure02.jpg" alt="Gahagan bifaces from the southern Caddo area (SCA) north (left), south (center), and central Texas regions (right)." width="100%" />
<p class="caption">
(#fig:gahagan bifaces 2D)Gahagan bifaces from the southern Caddo area (SCA) north (left), south (center), and central Texas regions (right).
</p>
</div>
<div id="load-geomorph-and-data" class="section level2">
<h2><span class="header-section-number">5.1</span> Load geomorph and data</h2>
<div class="sourceCode" id="cb57"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb57-1"><a href="geometric-morphometrics.html#cb57-1"></a><span class="co"># load geomorph</span></span>
<span id="cb57-2"><a href="geometric-morphometrics.html#cb57-2"></a><span class="co">#devtools::install_github(&quot;geomorphR/geomorph&quot;, ref = &quot;Stable&quot;, build_vignettes = TRUE)</span></span>
<span id="cb57-3"><a href="geometric-morphometrics.html#cb57-3"></a><span class="kw">library</span>(here)</span>
<span id="cb57-4"><a href="geometric-morphometrics.html#cb57-4"></a><span class="kw">library</span>(geomorph)</span>
<span id="cb57-5"><a href="geometric-morphometrics.html#cb57-5"></a><span class="kw">library</span>(tidyverse)</span>
<span id="cb57-6"><a href="geometric-morphometrics.html#cb57-6"></a><span class="kw">library</span>(wesanderson)</span>
<span id="cb57-7"><a href="geometric-morphometrics.html#cb57-7"></a></span>
<span id="cb57-8"><a href="geometric-morphometrics.html#cb57-8"></a><span class="kw">source</span>(<span class="st">&#39;readmulti.csv.R&#39;</span>)</span>
<span id="cb57-9"><a href="geometric-morphometrics.html#cb57-9"></a></span>
<span id="cb57-10"><a href="geometric-morphometrics.html#cb57-10"></a><span class="co"># read .csv files</span></span>
<span id="cb57-11"><a href="geometric-morphometrics.html#cb57-11"></a><span class="kw">setwd</span>(<span class="st">&quot;./data.3d1&quot;</span>)</span>
<span id="cb57-12"><a href="geometric-morphometrics.html#cb57-12"></a>filelist &lt;-<span class="st"> </span><span class="kw">list.files</span>(<span class="dt">pattern =</span> <span class="st">&quot;.csv&quot;</span>)</span>
<span id="cb57-13"><a href="geometric-morphometrics.html#cb57-13"></a>coords &lt;-<span class="st"> </span><span class="kw">readmulti.csv</span>(filelist)</span>
<span id="cb57-14"><a href="geometric-morphometrics.html#cb57-14"></a><span class="kw">setwd</span>(<span class="st">&quot;../&quot;</span>)</span>
<span id="cb57-15"><a href="geometric-morphometrics.html#cb57-15"></a></span>
<span id="cb57-16"><a href="geometric-morphometrics.html#cb57-16"></a><span class="co"># read qualitative data</span></span>
<span id="cb57-17"><a href="geometric-morphometrics.html#cb57-17"></a>qdata&lt;-<span class="kw">read.csv</span>(<span class="st">&quot;qdata.csv&quot;</span>,<span class="dt">header=</span><span class="ot">TRUE</span>, <span class="dt">row.names=</span><span class="dv">1</span>)</span>
<span id="cb57-18"><a href="geometric-morphometrics.html#cb57-18"></a>qdata&lt;-qdata[<span class="kw">match</span>(<span class="kw">dimnames</span>(coords)[[<span class="dv">3</span>]],<span class="kw">rownames</span>(qdata)),]</span></code></pre></div>
</div>
<div id="generalized-procrustes-analysis" class="section level2">
<h2><span class="header-section-number">5.2</span> Generalized Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(Kendall <a href="#ref-RN11622" role="doc-biblioref">1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span> performed in R 4.0.2 <span class="citation">(R Core Development Team, <a href="#ref-R" role="doc-biblioref">2020</a>)</span> using the <code>geomorph</code> library v. 3.3.1 <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>; Adams and Otárola-Castillo <a href="#ref-RN1774" role="doc-biblioref">2013</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(Gower <a href="#ref-RN11564" role="doc-biblioref">1975</a>; Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(Rohlf <a href="#ref-RN1646" role="doc-biblioref">1999</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>.</p>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="geometric-morphometrics.html#cb58-1"></a>Y.gpa&lt;-<span class="kw">gpagen</span>(coords, </span>
<span id="cb58-2"><a href="geometric-morphometrics.html#cb58-2"></a>              <span class="dt">PrinAxes =</span> <span class="ot">TRUE</span>, </span>
<span id="cb58-3"><a href="geometric-morphometrics.html#cb58-3"></a>              <span class="dt">print.progress =</span> <span class="ot">FALSE</span>)</span>
<span id="cb58-4"><a href="geometric-morphometrics.html#cb58-4"></a><span class="co"># plot(Y.gpa)</span></span>
<span id="cb58-5"><a href="geometric-morphometrics.html#cb58-5"></a></span>
<span id="cb58-6"><a href="geometric-morphometrics.html#cb58-6"></a><span class="co"># 3D gpa plot</span></span>
<span id="cb58-7"><a href="geometric-morphometrics.html#cb58-7"></a><span class="co">#knitr::include_graphics(&#39;images/gpa3d.png&#39;)</span></span></code></pre></div>
<div class="sourceCode" id="cb59"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb59-1"><a href="geometric-morphometrics.html#cb59-1"></a><span class="co"># geomorph data frame</span></span>
<span id="cb59-2"><a href="geometric-morphometrics.html#cb59-2"></a>gdf&lt;-<span class="kw">geomorph.data.frame</span>(<span class="dt">shape =</span> Y.gpa<span class="op">$</span>coords, </span>
<span id="cb59-3"><a href="geometric-morphometrics.html#cb59-3"></a>                         <span class="dt">size =</span> Y.gpa<span class="op">$</span>Csize, </span>
<span id="cb59-4"><a href="geometric-morphometrics.html#cb59-4"></a>                         <span class="dt">region =</span> qdata<span class="op">$</span>region, </span>
<span id="cb59-5"><a href="geometric-morphometrics.html#cb59-5"></a>                         <span class="dt">region2 =</span> qdata<span class="op">$</span>region2, </span>
<span id="cb59-6"><a href="geometric-morphometrics.html#cb59-6"></a>                         <span class="dt">mortuary =</span> qdata<span class="op">$</span>mortuary,</span>
<span id="cb59-7"><a href="geometric-morphometrics.html#cb59-7"></a>                         <span class="dt">cntxt =</span> qdata<span class="op">$</span>context) </span>
<span id="cb59-8"><a href="geometric-morphometrics.html#cb59-8"></a></span>
<span id="cb59-9"><a href="geometric-morphometrics.html#cb59-9"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb59-10"><a href="geometric-morphometrics.html#cb59-10"></a>qdata<span class="op">$</span>csz &lt;-<span class="st"> </span>Y.gpa<span class="op">$</span>Csize</span>
<span id="cb59-11"><a href="geometric-morphometrics.html#cb59-11"></a></span>
<span id="cb59-12"><a href="geometric-morphometrics.html#cb59-12"></a><span class="co"># boxplot - region 2 by centroid size</span></span>
<span id="cb59-13"><a href="geometric-morphometrics.html#cb59-13"></a>csz &lt;-<span class="st"> </span>qdata<span class="op">$</span>csz</span>
<span id="cb59-14"><a href="geometric-morphometrics.html#cb59-14"></a>cntxt &lt;-<span class="st"> </span>qdata<span class="op">$</span>context</span>
<span id="cb59-15"><a href="geometric-morphometrics.html#cb59-15"></a></span>
<span id="cb59-16"><a href="geometric-morphometrics.html#cb59-16"></a><span class="co"># print qdata</span></span>
<span id="cb59-17"><a href="geometric-morphometrics.html#cb59-17"></a>knitr<span class="op">::</span><span class="kw">kable</span>(qdata, </span>
<span id="cb59-18"><a href="geometric-morphometrics.html#cb59-18"></a>             <span class="dt">align =</span> <span class="st">&quot;ccccc&quot;</span>, </span>
<span id="cb59-19"><a href="geometric-morphometrics.html#cb59-19"></a>             <span class="dt">caption =</span> <span class="st">&quot;Attributes included in qdata.&quot;</span>)</span></code></pre></div>
<table>
<caption>(#tab:centsize.box)Attributes included in qdata.</caption>
<thead>
<tr class="header">
<th align="left"></th>
<th align="center">trinomial</th>
<th align="center">sitename</th>
<th align="center">context</th>
<th align="center">region</th>
<th align="center">region2</th>
<th align="center">mortuary</th>
<th align="center">length</th>
<th align="center">csz</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">3Bb1</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">131.5037</td>
<td align="center">333.3894</td>
</tr>
<tr class="even">
<td align="left">3Bb3</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">94.3363</td>
<td align="center">242.0058</td>
</tr>
<tr class="odd">
<td align="left">3Bb4</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">92.0793</td>
<td align="center">233.0918</td>
</tr>
<tr class="even">
<td align="left">3Bb5</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">72.0949</td>
<td align="center">180.6294</td>
</tr>
<tr class="odd">
<td align="left">3Bb6</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">146.7357</td>
<td align="center">364.1225</td>
</tr>
<tr class="even">
<td align="left">3Bb7</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">101.0797</td>
<td align="center">252.6581</td>
</tr>
<tr class="odd">
<td align="left">3Bb8</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">83.7998</td>
<td align="center">208.9919</td>
</tr>
<tr class="even">
<td align="left">4078-11</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">117.2529</td>
<td align="center">305.8570</td>
</tr>
<tr class="odd">
<td align="left">4078-12</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">115.4852</td>
<td align="center">297.2717</td>
</tr>
<tr class="even">
<td align="left">4078-13</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">116.9862</td>
<td align="center">301.3703</td>
</tr>
<tr class="odd">
<td align="left">4078-14</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">95.4462</td>
<td align="center">251.5336</td>
</tr>
<tr class="even">
<td align="left">4078-14B</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">108.8920</td>
<td align="center">291.8615</td>
</tr>
<tr class="odd">
<td align="left">4078-22</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">100.1365</td>
<td align="center">261.8585</td>
</tr>
<tr class="even">
<td align="left">4078-30</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">113.6609</td>
<td align="center">298.6175</td>
</tr>
<tr class="odd">
<td align="left">4078-32</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">104.2836</td>
<td align="center">277.5665</td>
</tr>
<tr class="even">
<td align="left">4078-45</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">116.8559</td>
<td align="center">292.6731</td>
</tr>
<tr class="odd">
<td align="left">4078-72</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">113.0991</td>
<td align="center">298.0688</td>
</tr>
<tr class="even">
<td align="left">4078-8</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">170.5638</td>
<td align="center">430.0098</td>
</tr>
<tr class="odd">
<td align="left">4078-9</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f134</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">151.0481</td>
<td align="center">400.5757</td>
</tr>
<tr class="even">
<td align="left">424-169</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">99.8860</td>
<td align="center">260.7870</td>
</tr>
<tr class="odd">
<td align="left">424-218</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">169.5278</td>
<td align="center">469.4424</td>
</tr>
<tr class="even">
<td align="left">424-221</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">176.6937</td>
<td align="center">473.3687</td>
</tr>
<tr class="odd">
<td align="left">424-230</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">105.1862</td>
<td align="center">276.4771</td>
</tr>
<tr class="even">
<td align="left">424-33</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">98.1234</td>
<td align="center">257.0546</td>
</tr>
<tr class="odd">
<td align="left">424-39</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">217.1066</td>
<td align="center">552.4138</td>
</tr>
<tr class="even">
<td align="left">424-40</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">137.8055</td>
<td align="center">393.3146</td>
</tr>
<tr class="odd">
<td align="left">424-41</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">185.9029</td>
<td align="center">488.9323</td>
</tr>
<tr class="even">
<td align="left">424-50</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">195.5831</td>
<td align="center">497.9108</td>
</tr>
<tr class="odd">
<td align="left">424-53</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">205.5500</td>
<td align="center">532.5167</td>
</tr>
<tr class="even">
<td align="left">463-1</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">265.5225</td>
<td align="center">677.5336</td>
</tr>
<tr class="odd">
<td align="left">463-16</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">128.0659</td>
<td align="center">330.4671</td>
</tr>
<tr class="even">
<td align="left">463-23</td>
<td align="center">41CE19</td>
<td align="center">George C Davis</td>
<td align="center">41ce19-f119</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">101.2441</td>
<td align="center">266.2417</td>
</tr>
<tr class="odd">
<td align="left">489</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">127.7488</td>
<td align="center">326.2118</td>
</tr>
<tr class="even">
<td align="left">490</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">105.2406</td>
<td align="center">271.2320</td>
</tr>
<tr class="odd">
<td align="left">532</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">117.7216</td>
<td align="center">311.6831</td>
</tr>
<tr class="even">
<td align="left">533</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">114.1056</td>
<td align="center">284.6977</td>
</tr>
<tr class="odd">
<td align="left">541</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">157.0695</td>
<td align="center">400.2734</td>
</tr>
<tr class="even">
<td align="left">542</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">95.8444</td>
<td align="center">246.6156</td>
</tr>
<tr class="odd">
<td align="left">543</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">171.9795</td>
<td align="center">438.0853</td>
</tr>
<tr class="even">
<td align="left">544</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">148.7254</td>
<td align="center">376.3687</td>
</tr>
<tr class="odd">
<td align="left">545</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">146.8326</td>
<td align="center">411.2969</td>
</tr>
<tr class="even">
<td align="left">546</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">156.8799</td>
<td align="center">406.2742</td>
</tr>
<tr class="odd">
<td align="left">547</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">106.5601</td>
<td align="center">271.2284</td>
</tr>
<tr class="even">
<td align="left">548</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">119.8263</td>
<td align="center">310.6772</td>
</tr>
<tr class="odd">
<td align="left">549</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">108.1385</td>
<td align="center">284.7020</td>
</tr>
<tr class="even">
<td align="left">550</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">121.3647</td>
<td align="center">304.1039</td>
</tr>
<tr class="odd">
<td align="left">551</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">163.9165</td>
<td align="center">424.7098</td>
</tr>
<tr class="even">
<td align="left">569</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">190.7678</td>
<td align="center">500.4181</td>
</tr>
<tr class="odd">
<td align="left">593</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">246.2476</td>
<td align="center">625.5836</td>
</tr>
<tr class="even">
<td align="left">605</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">181.5256</td>
<td align="center">465.5751</td>
</tr>
<tr class="odd">
<td align="left">606</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">150.1445</td>
<td align="center">388.6381</td>
</tr>
<tr class="even">
<td align="left">607</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">125.2065</td>
<td align="center">330.2037</td>
</tr>
<tr class="odd">
<td align="left">608</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">129.3802</td>
<td align="center">335.0552</td>
</tr>
<tr class="even">
<td align="left">609</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">130.6773</td>
<td align="center">337.8171</td>
</tr>
<tr class="odd">
<td align="left">610</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">107.4900</td>
<td align="center">278.8302</td>
</tr>
<tr class="even">
<td align="left">611</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">114.8306</td>
<td align="center">292.5582</td>
</tr>
<tr class="odd">
<td align="left">612</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">98.4037</td>
<td align="center">251.0676</td>
</tr>
<tr class="even">
<td align="left">613</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">92.3582</td>
<td align="center">240.2875</td>
</tr>
<tr class="odd">
<td align="left">614</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">64.8445</td>
<td align="center">168.1358</td>
</tr>
<tr class="even">
<td align="left">622</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">161.2902</td>
<td align="center">436.7907</td>
</tr>
<tr class="odd">
<td align="left">666</td>
<td align="center">16RR1</td>
<td align="center">Gahagan Mound</td>
<td align="center">16rr1-bp3</td>
<td align="center">SCA</td>
<td align="center">SCA-south</td>
<td align="center">yes</td>
<td align="center">199.4707</td>
<td align="center">524.4048</td>
</tr>
<tr class="even">
<td align="left">BlkThn</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp2</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">107.3962</td>
<td align="center">264.6985</td>
</tr>
<tr class="odd">
<td align="left">Case2LG</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp5</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">149.3291</td>
<td align="center">372.3756</td>
</tr>
<tr class="even">
<td align="left">Case2SM</td>
<td align="center">16CD12</td>
<td align="center">Mounds Plantation</td>
<td align="center">16cd12-bp5</td>
<td align="center">SCA</td>
<td align="center">SCA-north</td>
<td align="center">yes</td>
<td align="center">120.9068</td>
<td align="center">298.4132</td>
</tr>
</tbody>
</table>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb60-1"><a href="geometric-morphometrics.html#cb60-1"></a><span class="co"># principal components analysis</span></span>
<span id="cb60-2"><a href="geometric-morphometrics.html#cb60-2"></a>pca&lt;-<span class="kw">gm.prcomp</span>(Y.gpa<span class="op">$</span>coords)</span>
<span id="cb60-3"><a href="geometric-morphometrics.html#cb60-3"></a><span class="kw">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 64 
## Number of vectors 64 
## 
## Importance of Components:
##                              Comp1        Comp2        Comp3        Comp4        Comp5
## Eigenvalues            0.003457092 0.0004976554 0.0002224016 0.0001317437 0.0001162059
## Proportion of Variance 0.717706160 0.1033152769 0.0461714611 0.0273505304 0.0241248104
## Cumulative Proportion  0.717706160 0.8210214371 0.8671928982 0.8945434286 0.9186682390
##                               Comp6        Comp7        Comp8        Comp9       Comp10
## Eigenvalues            8.374617e-05 6.729966e-05 5.183255e-05 4.457031e-05 0.0000195789
## Proportion of Variance 1.738604e-02 1.397168e-02 1.076065e-02 9.252976e-03 0.0040646577
## Cumulative Proportion  9.360543e-01 9.500260e-01 9.607866e-01 9.700396e-01 0.9741042421
##                              Comp11       Comp12       Comp13       Comp14       Comp15
## Eigenvalues            1.769174e-05 1.271407e-05 1.102342e-05 9.127530e-06 7.975748e-06
## Proportion of Variance 3.672877e-03 2.639492e-03 2.288507e-03 1.894912e-03 1.655797e-03
## Cumulative Proportion  9.777771e-01 9.804166e-01 9.827051e-01 9.846000e-01 9.862558e-01
##                              Comp16       Comp17       Comp18       Comp19       Comp20
## Eigenvalues            7.307882e-06 5.655133e-06 5.080836e-06 4.524058e-06 3.916429e-06
## Proportion of Variance 1.517146e-03 1.174028e-03 1.054802e-03 9.392127e-04 8.130665e-04
## Cumulative Proportion  9.877730e-01 9.889470e-01 9.900018e-01 9.909410e-01 9.917541e-01
##                              Comp21       Comp22       Comp23       Comp24       Comp25
## Eigenvalues            3.285265e-06 3.089308e-06 2.785556e-06 2.664293e-06 2.291321e-06
## Proportion of Variance 6.820343e-04 6.413528e-04 5.782927e-04 5.531180e-04 4.756875e-04
## Cumulative Proportion  9.924361e-01 9.930775e-01 9.936558e-01 9.942089e-01 9.946846e-01
##                              Comp26       Comp27       Comp28       Comp29       Comp30
## Eigenvalues            2.231610e-06 1.965617e-06 1.802707e-06 1.622826e-06 1.497360e-06
## Proportion of Variance 4.632913e-04 4.080700e-04 3.742492e-04 3.369053e-04 3.108579e-04
## Cumulative Proportion  9.951479e-01 9.955559e-01 9.959302e-01 9.962671e-01 9.965779e-01
##                              Comp31       Comp32       Comp33       Comp34       Comp35
## Eigenvalues            1.469836e-06 1.228177e-06 1.118590e-06 1.074815e-06 1.022641e-06
## Proportion of Variance 3.051438e-04 2.549745e-04 2.322237e-04 2.231360e-04 2.123045e-04
## Cumulative Proportion  9.968831e-01 9.971381e-01 9.973703e-01 9.975934e-01 9.978057e-01
##                              Comp36       Comp37       Comp38       Comp39       Comp40
## Eigenvalues            9.345124e-07 8.381300e-07 7.742587e-07 6.399529e-07 6.300426e-07
## Proportion of Variance 1.940086e-04 1.739992e-04 1.607392e-04 1.328568e-04 1.307994e-04
## Cumulative Proportion  9.979997e-01 9.981737e-01 9.983345e-01 9.984673e-01 9.985981e-01
##                              Comp41       Comp42       Comp43       Comp44       Comp45
## Eigenvalues            5.777701e-07 5.269950e-07 5.209454e-07 4.717766e-07 4.398575e-07
## Proportion of Variance 1.199474e-04 1.094063e-04 1.081504e-04 9.794273e-05 9.131620e-05
## Cumulative Proportion  9.987181e-01 9.988275e-01 9.989356e-01 9.990336e-01 9.991249e-01
##                              Comp46       Comp47       Comp48       Comp49       Comp50
## Eigenvalues            4.054882e-07 4.001211e-07 3.796989e-07 3.426457e-07 3.132800e-07
## Proportion of Variance 8.418098e-05 8.306676e-05 7.882703e-05 7.113462e-05 6.503820e-05
## Cumulative Proportion  9.992091e-01 9.992921e-01 9.993710e-01 9.994421e-01 9.995071e-01
##                              Comp51      Comp52       Comp53       Comp54       Comp55
## Eigenvalues            2.991601e-07 2.58718e-07 2.458877e-07 2.363831e-07 2.262233e-07
## Proportion of Variance 6.210684e-05 5.37109e-05 5.104728e-05 4.907409e-05 4.696487e-05
## Cumulative Proportion  9.995692e-01 9.99623e-01 9.996740e-01 9.997231e-01 9.997700e-01
##                              Comp56       Comp57       Comp58       Comp59       Comp60
## Eigenvalues            2.077933e-07 1.850015e-07 1.670361e-07 1.459504e-07 1.343719e-07
## Proportion of Variance 4.313874e-05 3.840706e-05 3.467737e-05 3.029989e-05 2.789615e-05
## Cumulative Proportion  9.998132e-01 9.998516e-01 9.998863e-01 9.999166e-01 9.999445e-01
##                              Comp61       Comp62       Comp63       Comp64
## Eigenvalues            1.014766e-07 9.166551e-08 7.437560e-08 1.423063e-33
## Proportion of Variance 2.106695e-05 1.903013e-05 1.544068e-05 2.954337e-31
## Cumulative Proportion  9.999655e-01 9.999846e-01 1.000000e+00 1.000000e+00</code></pre>
</div>
<div id="test-hypothesis" class="section level2">
<h2><span class="header-section-number">5.3</span> Test Hypothesis</h2>
<div id="boxplot" class="section level3">
<h3><span class="header-section-number">5.3.1</span> Boxplot</h3>
<div class="sourceCode" id="cb62"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb62-1"><a href="geometric-morphometrics.html#cb62-1"></a><span class="co"># boxplot of Gahagan biface centroid size by burial context</span></span>
<span id="cb62-2"><a href="geometric-morphometrics.html#cb62-2"></a>csz.cntxt &lt;-<span class="st"> </span><span class="kw">ggplot</span>(qdata, <span class="kw">aes</span>(<span class="dt">x =</span> cntxt, <span class="dt">y =</span> csz, <span class="dt">color =</span> cntxt)) <span class="op">+</span><span class="st"> </span></span>
<span id="cb62-3"><a href="geometric-morphometrics.html#cb62-3"></a><span class="st">  </span><span class="kw">geom_boxplot</span>() <span class="op">+</span></span>
<span id="cb62-4"><a href="geometric-morphometrics.html#cb62-4"></a><span class="st">  </span><span class="kw">geom_dotplot</span>(<span class="dt">binaxis =</span> <span class="st">&#39;y&#39;</span>, <span class="dt">stackdir =</span> <span class="st">&#39;center&#39;</span>, <span class="dt">dotsize =</span> <span class="fl">0.3</span>) <span class="op">+</span></span>
<span id="cb62-5"><a href="geometric-morphometrics.html#cb62-5"></a><span class="st">  </span><span class="kw">scale_colour_manual</span>(<span class="dt">values =</span> <span class="kw">wes_palette</span>(<span class="st">&quot;IsleofDogs1&quot;</span>)) <span class="op">+</span></span>
<span id="cb62-6"><a href="geometric-morphometrics.html#cb62-6"></a><span class="st">  </span><span class="kw">theme</span>(<span class="dt">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="op">+</span></span>
<span id="cb62-7"><a href="geometric-morphometrics.html#cb62-7"></a><span class="st">  </span><span class="kw">labs</span>(<span class="dt">x =</span> <span class="st">&#39;Burial Context&#39;</span>, <span class="dt">y =</span> <span class="st">&#39;Centroid Size&#39;</span>)</span>
<span id="cb62-8"><a href="geometric-morphometrics.html#cb62-8"></a><span class="co"># render plot</span></span>
<span id="cb62-9"><a href="geometric-morphometrics.html#cb62-9"></a>csz.cntxt</span></code></pre></div>
<pre><code>## `stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/box.h1-1.png" alt="Boxplot of centroid size by Heartland (in/out)." width="100%" />
<p class="caption">
(#fig:box.h1)Boxplot of centroid size by Heartland (in/out).
</p>
</div>
</div>
<div id="principal-components-analysis" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Principal Components Analysis</h3>
<p>Principal components analysis <span class="citation">(Jolliffe <a href="#ref-RN1746" role="doc-biblioref">2002</a>)</span> was used to visualise shape variation among the bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(Klingenberg <a href="#ref-RN1731" role="doc-biblioref">2013</a>; Sherratt et al. <a href="#ref-RN479" role="doc-biblioref">2014</a>)</span>.</p>
<div class="sourceCode" id="cb64"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb64-1"><a href="geometric-morphometrics.html#cb64-1"></a><span class="co"># set plot parameters to plot by burial context</span></span>
<span id="cb64-2"><a href="geometric-morphometrics.html#cb64-2"></a>pch.gps.cntxt &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">3</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">17</span><span class="op">:</span><span class="dv">19</span>)[<span class="kw">as.factor</span>(cntxt)]</span>
<span id="cb64-3"><a href="geometric-morphometrics.html#cb64-3"></a>col.gps.cntxt &lt;-<span class="st"> </span><span class="kw">wes_palette</span>(<span class="st">&quot;IsleofDogs1&quot;</span>)[<span class="kw">as.factor</span>(cntxt)]</span>
<span id="cb64-4"><a href="geometric-morphometrics.html#cb64-4"></a>col.hull.cntxt &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;#9986A5&quot;</span>,<span class="st">&quot;#8D8680&quot;</span>,<span class="st">&quot;#D9D0D3&quot;</span>,<span class="st">&quot;#CCBA72&quot;</span>,<span class="st">&quot;#0F0D0E&quot;</span>,<span class="st">&quot;#79402E&quot;</span>)</span>
<span id="cb64-5"><a href="geometric-morphometrics.html#cb64-5"></a></span>
<span id="cb64-6"><a href="geometric-morphometrics.html#cb64-6"></a><span class="co"># plot pca by region 2</span></span>
<span id="cb64-7"><a href="geometric-morphometrics.html#cb64-7"></a>pc.plot1 &lt;-<span class="st"> </span><span class="kw">plot</span>(pca, <span class="dt">asp =</span> <span class="dv">1</span>,</span>
<span id="cb64-8"><a href="geometric-morphometrics.html#cb64-8"></a>                  <span class="dt">pch =</span> pch.gps.cntxt,</span>
<span id="cb64-9"><a href="geometric-morphometrics.html#cb64-9"></a>                  <span class="dt">col =</span> col.gps.cntxt)</span>
<span id="cb64-10"><a href="geometric-morphometrics.html#cb64-10"></a>                  <span class="kw">shapeHulls</span>(pc.plot1, </span>
<span id="cb64-11"><a href="geometric-morphometrics.html#cb64-11"></a>                             <span class="dt">groups =</span> cntxt,</span>
<span id="cb64-12"><a href="geometric-morphometrics.html#cb64-12"></a>                             <span class="dt">group.cols =</span> col.hull.cntxt)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/pca.h1-1.png" alt="Results of PCA with central Texas sample in gray squares, southern Caddo area (north) in orange triangles, and southern Caddo area (south) in tan circles." width="100%" />
<p class="caption">
(#fig:pca.h1)Results of PCA with central Texas sample in gray squares, southern Caddo area (north) in orange triangles, and southern Caddo area (south) in tan circles.
</p>
</div>
<div class="sourceCode" id="cb65"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb65-1"><a href="geometric-morphometrics.html#cb65-1"></a><span class="co"># plot PCA with warp grids</span></span>
<span id="cb65-2"><a href="geometric-morphometrics.html#cb65-2"></a><span class="co">#knitr::include_graphics(&#39;images/pca-warp-ref.png&#39;)</span></span></code></pre></div>
</div>
<div id="define-models" class="section level3">
<h3><span class="header-section-number">5.3.3</span> Define models</h3>
<div class="sourceCode" id="cb66"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb66-1"><a href="geometric-morphometrics.html#cb66-1"></a><span class="co"># allometry</span></span>
<span id="cb66-2"><a href="geometric-morphometrics.html#cb66-2"></a>fit.size &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size, </span>
<span id="cb66-3"><a href="geometric-morphometrics.html#cb66-3"></a>                     <span class="dt">data =</span> gdf, </span>
<span id="cb66-4"><a href="geometric-morphometrics.html#cb66-4"></a>                     <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb66-5"><a href="geometric-morphometrics.html#cb66-5"></a>                     <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb66-6"><a href="geometric-morphometrics.html#cb66-6"></a></span>
<span id="cb66-7"><a href="geometric-morphometrics.html#cb66-7"></a><span class="co"># allometry - common allometry, different means -&gt; cntxt</span></span>
<span id="cb66-8"><a href="geometric-morphometrics.html#cb66-8"></a>fit.sz.ccntxt &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size <span class="op">+</span><span class="st"> </span>cntxt, </span>
<span id="cb66-9"><a href="geometric-morphometrics.html#cb66-9"></a>                       <span class="dt">data =</span> gdf, </span>
<span id="cb66-10"><a href="geometric-morphometrics.html#cb66-10"></a>                       <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb66-11"><a href="geometric-morphometrics.html#cb66-11"></a>                       <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb66-12"><a href="geometric-morphometrics.html#cb66-12"></a></span>
<span id="cb66-13"><a href="geometric-morphometrics.html#cb66-13"></a><span class="co"># allometry - unique allometries -&gt; cntxt</span></span>
<span id="cb66-14"><a href="geometric-morphometrics.html#cb66-14"></a>fit.sz.ucntxt &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size <span class="op">*</span><span class="st"> </span>cntxt, </span>
<span id="cb66-15"><a href="geometric-morphometrics.html#cb66-15"></a>                        <span class="dt">data =</span> gdf, </span>
<span id="cb66-16"><a href="geometric-morphometrics.html#cb66-16"></a>                        <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb66-17"><a href="geometric-morphometrics.html#cb66-17"></a>                        <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb66-18"><a href="geometric-morphometrics.html#cb66-18"></a></span>
<span id="cb66-19"><a href="geometric-morphometrics.html#cb66-19"></a><span class="co"># size as a function of group</span></span>
<span id="cb66-20"><a href="geometric-morphometrics.html#cb66-20"></a>fit.sizecntxt &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(size <span class="op">~</span><span class="st"> </span>cntxt, </span>
<span id="cb66-21"><a href="geometric-morphometrics.html#cb66-21"></a>                         <span class="dt">data =</span> gdf, </span>
<span id="cb66-22"><a href="geometric-morphometrics.html#cb66-22"></a>                         <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb66-23"><a href="geometric-morphometrics.html#cb66-23"></a>                         <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb66-24"><a href="geometric-morphometrics.html#cb66-24"></a></span>
<span id="cb66-25"><a href="geometric-morphometrics.html#cb66-25"></a><span class="co"># shape as a function of group</span></span>
<span id="cb66-26"><a href="geometric-morphometrics.html#cb66-26"></a>fit.shapecntxt &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>cntxt, </span>
<span id="cb66-27"><a href="geometric-morphometrics.html#cb66-27"></a>                          <span class="dt">data =</span> gdf, </span>
<span id="cb66-28"><a href="geometric-morphometrics.html#cb66-28"></a>                          <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb66-29"><a href="geometric-morphometrics.html#cb66-29"></a>                          <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
</div>
<div id="allometry" class="section level3">
<h3><span class="header-section-number">5.3.4</span> Allometry</h3>
<div class="sourceCode" id="cb67"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb67-1"><a href="geometric-morphometrics.html#cb67-1"></a><span class="co"># allometry - does shape change with size?</span></span>
<span id="cb67-2"><a href="geometric-morphometrics.html#cb67-2"></a><span class="kw">anova</span>(fit.size)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df       SS        MS     Rsq     F       Z Pr(&gt;F)
## size       1 0.004323 0.0043230 0.01425 0.896 0.35497 0.3667
## Residuals 62 0.299139 0.0048248 0.98575                     
## Total     63 0.303462                                       
## 
## Call: procD.lm(f1 = shape ~ size, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb69"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb69-1"><a href="geometric-morphometrics.html#cb69-1"></a><span class="co"># cntxt</span></span>
<span id="cb69-2"><a href="geometric-morphometrics.html#cb69-2"></a><span class="kw">anova</span>(fit.sz.ccntxt) <span class="co"># common allometry (cntxt)</span></span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df       SS       MS     Rsq      F      Z Pr(&gt;F)    
## size       1 0.004323 0.004323 0.01425 1.2288 0.6996 0.2516    
## cntxt      5 0.098613 0.019723 0.32496 5.6061 3.9259  1e-04 ***
## Residuals 57 0.200527 0.003518 0.66080                         
## Total     63 0.303462                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ size + cntxt, iter = 9999, data = gdf,      print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb71"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb71-1"><a href="geometric-morphometrics.html#cb71-1"></a><span class="kw">anova</span>(fit.sz.ucntxt) <span class="co"># unique allometry (cntxt)</span></span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df       SS        MS     Rsq      F      Z Pr(&gt;F)    
## size        1 0.004323 0.0043230 0.01425 1.2390 0.7063 0.2504    
## cntxt       5 0.098613 0.0197225 0.32496 5.6528 3.9082  1e-04 ***
## size:cntxt  5 0.019101 0.0038201 0.06294 1.0949 0.3978 0.3445    
## Residuals  52 0.181426 0.0034890 0.59785                         
## Total      63 0.303462                                           
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ size * cntxt, iter = 9999, data = gdf,      print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb73"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb73-1"><a href="geometric-morphometrics.html#cb73-1"></a><span class="co"># allometry plots</span></span>
<span id="cb73-2"><a href="geometric-morphometrics.html#cb73-2"></a><span class="co"># regscore (Drake and Klingenberg 2008)</span></span>
<span id="cb73-3"><a href="geometric-morphometrics.html#cb73-3"></a><span class="kw">plot</span>(fit.size, <span class="dt">type =</span> <span class="st">&quot;regression&quot;</span>, <span class="dt">reg.type =</span> <span class="st">&quot;RegScore&quot;</span>, </span>
<span id="cb73-4"><a href="geometric-morphometrics.html#cb73-4"></a>     <span class="dt">predictor =</span> <span class="kw">log</span>(gdf<span class="op">$</span>size), <span class="dt">pch =</span> pch.gps.cntxt, </span>
<span id="cb73-5"><a href="geometric-morphometrics.html#cb73-5"></a>     <span class="dt">col =</span> col.gps.cntxt)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-1.png" width="100%" /></p>
<div class="sourceCode" id="cb74"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb74-1"><a href="geometric-morphometrics.html#cb74-1"></a><span class="co"># common allometric component (Mitteroecker 2004)</span></span>
<span id="cb74-2"><a href="geometric-morphometrics.html#cb74-2"></a><span class="kw">plotAllometry</span>(fit.sz.ccntxt, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, </span>
<span id="cb74-3"><a href="geometric-morphometrics.html#cb74-3"></a>              <span class="dt">method =</span> <span class="st">&quot;CAC&quot;</span>, <span class="dt">pch =</span> pch.gps.cntxt, </span>
<span id="cb74-4"><a href="geometric-morphometrics.html#cb74-4"></a>              <span class="dt">col =</span> col.gps.cntxt)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-2.png" width="100%" /></p>
<div class="sourceCode" id="cb75"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb75-1"><a href="geometric-morphometrics.html#cb75-1"></a><span class="co"># size-shape pca (Mitteroecker 2004)</span></span>
<span id="cb75-2"><a href="geometric-morphometrics.html#cb75-2"></a><span class="kw">plotAllometry</span>(fit.size, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, </span>
<span id="cb75-3"><a href="geometric-morphometrics.html#cb75-3"></a>              <span class="dt">method =</span> <span class="st">&quot;size.shape&quot;</span>, <span class="dt">pch =</span> pch.gps.cntxt, </span>
<span id="cb75-4"><a href="geometric-morphometrics.html#cb75-4"></a>              <span class="dt">col =</span> col.gps.cntxt)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-3.png" width="100%" /></p>
<div class="sourceCode" id="cb76"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb76-1"><a href="geometric-morphometrics.html#cb76-1"></a><span class="co"># predline (Adams and Nistri 2010)</span></span>
<span id="cb76-2"><a href="geometric-morphometrics.html#cb76-2"></a><span class="kw">plotAllometry</span>(fit.sz.ucntxt, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, </span>
<span id="cb76-3"><a href="geometric-morphometrics.html#cb76-3"></a>              <span class="dt">method =</span> <span class="st">&quot;PredLine&quot;</span>, <span class="dt">pch =</span> pch.gps.cntxt, </span>
<span id="cb76-4"><a href="geometric-morphometrics.html#cb76-4"></a>              <span class="dt">col =</span> col.gps.cntxt)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-4.png" width="100%" /></p>
<div class="sourceCode" id="cb77"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb77-1"><a href="geometric-morphometrics.html#cb77-1"></a><span class="co"># plot form</span></span>
<span id="cb77-2"><a href="geometric-morphometrics.html#cb77-2"></a><span class="co">#knitr::include_graphics(&#39;images/gbiface-allom-form.png&#39;)</span></span></code></pre></div>
</div>
<div id="procrustes-anova" class="section level3">
<h3><span class="header-section-number">5.3.5</span> Procrustes ANOVA</h3>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(Adams and Collyer <a href="#ref-RN1655" role="doc-biblioref">2015</a>; Collyer and Adams <a href="#ref-RN11775" role="doc-biblioref">2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(Anderson and Ter Braak <a href="#ref-RN1719" role="doc-biblioref">2003</a>)</span>. To assess whether shape changes with size (allometry), and differs by group (region), Procrustes ANOVAs <span class="citation">(Goodall <a href="#ref-RN1749" role="doc-biblioref">1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(Collyer, Sekora, and Adams <a href="#ref-RN1756" role="doc-biblioref">2015</a>)</span>.</p>
<div class="sourceCode" id="cb78"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb78-1"><a href="geometric-morphometrics.html#cb78-1"></a><span class="co"># ANOVA: do gahagan biface shapes differ by cntxt?</span></span>
<span id="cb78-2"><a href="geometric-morphometrics.html#cb78-2"></a><span class="kw">anova</span>(fit.shapecntxt)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df       SS        MS     Rsq      F      Z Pr(&gt;F)    
## cntxt      5 0.095517 0.0191035 0.31476 5.3283 3.8054  1e-04 ***
## Residuals 58 0.207945 0.0035853 0.68524                         
## Total     63 0.303462                                           
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ cntxt, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb80"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb80-1"><a href="geometric-morphometrics.html#cb80-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb80-2"><a href="geometric-morphometrics.html#cb80-2"></a>sh.cntxt &lt;-<span class="st"> </span><span class="kw">pairwise</span>(fit.shapecntxt, </span>
<span id="cb80-3"><a href="geometric-morphometrics.html#cb80-3"></a>                    <span class="dt">groups =</span> qdata<span class="op">$</span>context)</span>
<span id="cb80-4"><a href="geometric-morphometrics.html#cb80-4"></a><span class="kw">summary</span>(sh.cntxt, </span>
<span id="cb80-5"><a href="geometric-morphometrics.html#cb80-5"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb80-6"><a href="geometric-morphometrics.html#cb80-6"></a>        <span class="dt">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: 16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                                  d  UCL (95%)          Z Pr &gt; d
## 16cd12-bp2:16cd12-bp5   0.02125514 0.09839057 -1.6768553 0.9550
## 16cd12-bp2:16rr1-bp2    0.09142420 0.05306725  2.7052099 0.0002
## 16cd12-bp2:16rr1-bp3    0.09610764 0.05513031  2.7333541 0.0004
## 16cd12-bp2:41ce19-f119  0.11466654 0.05513982  3.0099286 0.0001
## 16cd12-bp2:41ce19-f134  0.11022001 0.05561002  2.9734350 0.0002
## 16cd12-bp5:16rr1-bp2    0.08227757 0.09478945  1.4272384 0.0889
## 16cd12-bp5:16rr1-bp3    0.08625964 0.09583233  1.5002139 0.0764
## 16cd12-bp5:41ce19-f119  0.10672388 0.09441947  1.9081618 0.0272
## 16cd12-bp5:41ce19-f134  0.10050463 0.09576898  1.7967481 0.0392
## 16rr1-bp2:16rr1-bp3     0.01887681 0.04490732 -0.2572097 0.5892
## 16rr1-bp2:41ce19-f119   0.02776345 0.04526319  0.6022139 0.2878
## 16rr1-bp2:41ce19-f134   0.02598337 0.04637444  0.4282074 0.3418
## 16rr1-bp3:41ce19-f119   0.02696049 0.04709234  0.4608496 0.3301
## 16rr1-bp3:41ce19-f134   0.01894269 0.04857723 -0.4164250 0.6376
## 41ce19-f119:41ce19-f134 0.01793345 0.04823926 -0.5681164 0.6971</code></pre>
<div class="sourceCode" id="cb82"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb82-1"><a href="geometric-morphometrics.html#cb82-1"></a><span class="co"># pairwise distance between variances = standardization?</span></span>
<span id="cb82-2"><a href="geometric-morphometrics.html#cb82-2"></a><span class="kw">summary</span>(sh.cntxt, </span>
<span id="cb82-3"><a href="geometric-morphometrics.html#cb82-3"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb82-4"><a href="geometric-morphometrics.html#cb82-4"></a>        <span class="dt">test.type =</span> <span class="st">&quot;var&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: 16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134 
## 
## RRPP: 10000 permutations
## 
## 
## Observed variances by group
## 
##   16cd12-bp2   16cd12-bp5    16rr1-bp2    16rr1-bp3  41ce19-f119  41ce19-f134 
## 0.0010385632 0.0004395759 0.0046066606 0.0019679229 0.0047478771 0.0031454353 
## 
## Pairwise distances between variances, plus statistics
##                                    d   UCL (95%)           Z Pr &gt; d
## 16cd12-bp2:16cd12-bp5   0.0005989873 0.009856478 -0.81843418 0.8077
## 16cd12-bp2:16rr1-bp2    0.0035680974 0.003931076  1.38463106 0.0771
## 16cd12-bp2:16rr1-bp3    0.0009293597 0.004097712 -0.37080827 0.6376
## 16cd12-bp2:41ce19-f119  0.0037093139 0.004167899  1.36425467 0.0820
## 16cd12-bp2:41ce19-f134  0.0021068721 0.004175347  0.51103900 0.3340
## 16cd12-bp5:16rr1-bp2    0.0041670847 0.010043677  1.01034400 0.1040
## 16cd12-bp5:16rr1-bp3    0.0015283470 0.009880238 -0.04521613 0.5357
## 16cd12-bp5:41ce19-f119  0.0043083012 0.009984647  1.03088774 0.1085
## 16cd12-bp5:41ce19-f134  0.0027058595 0.009840269  0.51000311 0.2832
## 16rr1-bp2:16rr1-bp3     0.0026387377 0.003409376  1.14262391 0.1328
## 16rr1-bp2:41ce19-f119   0.0001412165 0.003410169 -1.64796100 0.9448
## 16rr1-bp2:41ce19-f134   0.0014612252 0.003476177  0.21928255 0.4377
## 16rr1-bp3:41ce19-f119   0.0027799542 0.003589771  1.13172610 0.1344
## 16rr1-bp3:41ce19-f134   0.0011775125 0.003657438 -0.08442833 0.5483
## 41ce19-f119:41ce19-f134 0.0016024418 0.003646380  0.29697562 0.4057</code></pre>
<div class="sourceCode" id="cb84"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb84-1"><a href="geometric-morphometrics.html#cb84-1"></a><span class="co"># ANOVA: do gahagan biface sizes differ by cntxt?</span></span>
<span id="cb84-2"><a href="geometric-morphometrics.html#cb84-2"></a><span class="kw">anova</span>(fit.sizecntxt)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df     SS      MS     Rsq      F      Z Pr(&gt;F)  
## cntxt      5 152049 30409.7 0.21598 3.1956 2.1851 0.0165 *
## Residuals 58 551940  9516.2 0.78402                       
## Total     63 703988                                       
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = size ~ cntxt, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb86"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb86-1"><a href="geometric-morphometrics.html#cb86-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb86-2"><a href="geometric-morphometrics.html#cb86-2"></a>sz.cntxt &lt;-<span class="st"> </span><span class="kw">pairwise</span>(fit.sizecntxt, </span>
<span id="cb86-3"><a href="geometric-morphometrics.html#cb86-3"></a>                    <span class="dt">groups =</span> qdata<span class="op">$</span>context)</span>
<span id="cb86-4"><a href="geometric-morphometrics.html#cb86-4"></a></span>
<span id="cb86-5"><a href="geometric-morphometrics.html#cb86-5"></a><span class="kw">summary</span>(sz.cntxt, </span>
<span id="cb86-6"><a href="geometric-morphometrics.html#cb86-6"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb86-7"><a href="geometric-morphometrics.html#cb86-7"></a>        <span class="dt">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: 16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                                 d UCL (95%)          Z Pr &gt; d
## 16cd12-bp2:16cd12-bp5    75.44602 162.01597  0.3956018 0.3696
## 16cd12-bp2:16rr1-bp2     88.08773  89.87976  1.5404487 0.0560
## 16cd12-bp2:16rr1-bp3     99.66295  92.22125  1.7124509 0.0352
## 16cd12-bp2:41ce19-f119  161.31777  92.84969  2.7315780 0.0008
## 16cd12-bp2:41ce19-f134   48.99025  95.48466  0.5300390 0.3165
## 16cd12-bp5:16rr1-bp2     12.64172 150.61992 -1.2753198 0.8861
## 16cd12-bp5:16rr1-bp3     24.21693 153.92746 -0.7862466 0.7747
## 16cd12-bp5:41ce19-f119   85.87175 153.54956  0.6270347 0.2776
## 16cd12-bp5:41ce19-f134   26.45577 154.41576 -0.7124003 0.7545
## 16rr1-bp2:16rr1-bp3      11.57521  79.03659 -0.7802840 0.7685
## 16rr1-bp2:41ce19-f119    73.23003  77.84386  1.4782308 0.0647
## 16rr1-bp2:41ce19-f134    39.09748  78.88175  0.4777654 0.3403
## 16rr1-bp3:41ce19-f119    61.65482  80.65322  1.1309261 0.1340
## 16rr1-bp3:41ce19-f134    50.67269  83.21894  0.7641621 0.2398
## 41ce19-f119:41ce19-f134 112.32752  83.35145  2.1609596 0.0081</code></pre>
<div class="sourceCode" id="cb88"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb88-1"><a href="geometric-morphometrics.html#cb88-1"></a><span class="co"># pairwise distance between variances = standardization?</span></span>
<span id="cb88-2"><a href="geometric-morphometrics.html#cb88-2"></a><span class="kw">summary</span>(sz.cntxt, </span>
<span id="cb88-3"><a href="geometric-morphometrics.html#cb88-3"></a>        <span class="dt">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb88-4"><a href="geometric-morphometrics.html#cb88-4"></a>        <span class="dt">test.type =</span> <span class="st">&quot;var&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: 16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134 
## 
## RRPP: 10000 permutations
## 
## 
## Observed variances by group
## 
##  16cd12-bp2  16cd12-bp5   16rr1-bp2   16rr1-bp3 41ce19-f119 41ce19-f134 
##    3281.602    1367.608    5210.250   14736.384   16729.179    2544.620 
## 
## Pairwise distances between variances, plus statistics
##                                  d UCL (95%)          Z Pr &gt; d
## 16cd12-bp2:16cd12-bp5    1913.9937 24949.587 -0.9101666 0.8225
## 16cd12-bp2:16rr1-bp2     1928.6484 11304.901 -0.6997874 0.7460
## 16cd12-bp2:16rr1-bp3    11454.7824 12010.968  1.5084402 0.0603
## 16cd12-bp2:41ce19-f119  13447.5768 11866.986  1.7987217 0.0266
## 16cd12-bp2:41ce19-f134    736.9823 12158.701 -1.3912325 0.9064
## 16cd12-bp5:16rr1-bp2     3842.6421 25853.524 -0.3221968 0.6637
## 16cd12-bp5:16rr1-bp3    13368.7761 25419.854  1.0596901 0.1068
## 16cd12-bp5:41ce19-f119  15361.5705 25619.623  1.2371751 0.0833
## 16cd12-bp5:41ce19-f134   1177.0114 25122.545 -1.2724378 0.8973
## 16rr1-bp2:16rr1-bp3      9526.1340  9827.746  1.5192591 0.0600
## 16rr1-bp2:41ce19-f119   11518.9284  9847.145  1.8755375 0.0211
## 16rr1-bp2:41ce19-f134    2665.6307 10085.152 -0.2970893 0.6212
## 16rr1-bp3:41ce19-f119    1992.7944 10454.213 -0.5864931 0.7135
## 16rr1-bp3:41ce19-f134   12191.7647 10496.121  1.8781392 0.0205
## 41ce19-f119:41ce19-f134 14184.5591 10578.504  2.1856751 0.0066</code></pre>
</div>
<div id="morphological-disparity" class="section level3">
<h3><span class="header-section-number">5.3.6</span> Morphological disparity</h3>
<p>Procrustes variance was used to discriminate between regions and compare the amount of shape variation (morphological disparity) <span class="citation">(Zelditch et al. <a href="#ref-RN5703" role="doc-biblioref">2004</a>)</span>, estimated as the Procrustes variance using residuals of linear model fit <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>.</p>
<div class="sourceCode" id="cb90"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb90-1"><a href="geometric-morphometrics.html#cb90-1"></a><span class="co"># morphological disparity: do either of the groups display greater </span></span>
<span id="cb90-2"><a href="geometric-morphometrics.html#cb90-2"></a><span class="co"># shape variation among individuals relative to the other group?</span></span>
<span id="cb90-3"><a href="geometric-morphometrics.html#cb90-3"></a><span class="kw">morphol.disparity</span>(fit.shapecntxt, </span>
<span id="cb90-4"><a href="geometric-morphometrics.html#cb90-4"></a>                  <span class="dt">groups =</span> qdata<span class="op">$</span>context,</span>
<span id="cb90-5"><a href="geometric-morphometrics.html#cb90-5"></a>                  <span class="dt">data =</span> gdf, </span>
<span id="cb90-6"><a href="geometric-morphometrics.html#cb90-6"></a>                  <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb90-7"><a href="geometric-morphometrics.html#cb90-7"></a>                  <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.shapecntxt, groups = qdata$context,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##   16cd12-bp2   16cd12-bp5    16rr1-bp2    16rr1-bp3  41ce19-f119  41ce19-f134 
## 0.0010385632 0.0004395759 0.0046066606 0.0019679229 0.0047478771 0.0031454353 
## 
## 
## Pairwise absolute differences between variances
##               16cd12-bp2   16cd12-bp5    16rr1-bp2    16rr1-bp3  41ce19-f119 41ce19-f134
## 16cd12-bp2  0.0000000000 0.0005989873 0.0035680974 0.0009293597 0.0037093139 0.002106872
## 16cd12-bp5  0.0005989873 0.0000000000 0.0041670847 0.0015283470 0.0043083012 0.002705859
## 16rr1-bp2   0.0035680974 0.0041670847 0.0000000000 0.0026387377 0.0001412165 0.001461225
## 16rr1-bp3   0.0009293597 0.0015283470 0.0026387377 0.0000000000 0.0027799542 0.001177512
## 41ce19-f119 0.0037093139 0.0043083012 0.0001412165 0.0027799542 0.0000000000 0.001602442
## 41ce19-f134 0.0021068721 0.0027058595 0.0014612252 0.0011775125 0.0016024418 0.000000000
## 
## 
## P-Values
##             16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134
## 16cd12-bp2      1.0000     0.8077    0.0771    0.6376      0.0820      0.3340
## 16cd12-bp5      0.8077     1.0000    0.1040    0.5357      0.1085      0.2832
## 16rr1-bp2       0.0771     0.1040    1.0000    0.1328      0.9448      0.4377
## 16rr1-bp3       0.6376     0.5357    0.1328    1.0000      0.1344      0.5483
## 41ce19-f119     0.0820     0.1085    0.9448    0.1344      1.0000      0.4057
## 41ce19-f134     0.3340     0.2832    0.4377    0.5483      0.4057      1.0000</code></pre>
<div class="sourceCode" id="cb92"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb92-1"><a href="geometric-morphometrics.html#cb92-1"></a><span class="co"># morphological disparity: do either of the groups display greater </span></span>
<span id="cb92-2"><a href="geometric-morphometrics.html#cb92-2"></a><span class="co"># size variation among individuals relative to the other group?</span></span>
<span id="cb92-3"><a href="geometric-morphometrics.html#cb92-3"></a><span class="kw">morphol.disparity</span>(fit.sizecntxt, </span>
<span id="cb92-4"><a href="geometric-morphometrics.html#cb92-4"></a>                  <span class="dt">groups =</span> qdata<span class="op">$</span>context, </span>
<span id="cb92-5"><a href="geometric-morphometrics.html#cb92-5"></a>                  <span class="dt">data =</span> gdf, </span>
<span id="cb92-6"><a href="geometric-morphometrics.html#cb92-6"></a>                  <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, </span>
<span id="cb92-7"><a href="geometric-morphometrics.html#cb92-7"></a>                  <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.sizecntxt, groups = qdata$context,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##  16cd12-bp2  16cd12-bp5   16rr1-bp2   16rr1-bp3 41ce19-f119 41ce19-f134 
##    3281.602    1367.608    5210.250   14736.384   16729.179    2544.620 
## 
## 
## Pairwise absolute differences between variances
##             16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134
## 16cd12-bp2      0.0000   1913.994  1928.648 11454.782   13447.577    736.9823
## 16cd12-bp5   1913.9937      0.000  3842.642 13368.776   15361.570   1177.0114
## 16rr1-bp2    1928.6484   3842.642     0.000  9526.134   11518.928   2665.6307
## 16rr1-bp3   11454.7824  13368.776  9526.134     0.000    1992.794  12191.7647
## 41ce19-f119 13447.5768  15361.570 11518.928  1992.794       0.000  14184.5591
## 41ce19-f134   736.9823   1177.011  2665.631 12191.765   14184.559      0.0000
## 
## 
## P-Values
##             16cd12-bp2 16cd12-bp5 16rr1-bp2 16rr1-bp3 41ce19-f119 41ce19-f134
## 16cd12-bp2      1.0000     0.8225    0.7460    0.0603      0.0266      0.9064
## 16cd12-bp5      0.8225     1.0000    0.6637    0.1068      0.0833      0.8973
## 16rr1-bp2       0.7460     0.6637    1.0000    0.0600      0.0211      0.6212
## 16rr1-bp3       0.0603     0.1068    0.0600    1.0000      0.7135      0.0205
## 41ce19-f119     0.0266     0.0833    0.0211    0.7135      1.0000      0.0066
## 41ce19-f134     0.9064     0.8973    0.6212    0.0205      0.0066      1.0000</code></pre>
</div>
<div id="mean-shapes" class="section level3">
<h3><span class="header-section-number">5.3.7</span> Mean shapes</h3>
<div class="sourceCode" id="cb94"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb94-1"><a href="geometric-morphometrics.html#cb94-1"></a><span class="co">#subset landmark coordinates to produce mean shapes for groups</span></span>
<span id="cb94-2"><a href="geometric-morphometrics.html#cb94-2"></a>new.coords&lt;-<span class="kw">coords.subset</span>(<span class="dt">A =</span> Y.gpa<span class="op">$</span>coords, </span>
<span id="cb94-3"><a href="geometric-morphometrics.html#cb94-3"></a>                          <span class="dt">group =</span> qdata<span class="op">$</span>context)</span>
<span id="cb94-4"><a href="geometric-morphometrics.html#cb94-4"></a></span>
<span id="cb94-5"><a href="geometric-morphometrics.html#cb94-5"></a><span class="kw">names</span>(new.coords)</span></code></pre></div>
<pre><code>## [1] &quot;16cd12-bp2&quot;  &quot;16cd12-bp5&quot;  &quot;16rr1-bp2&quot;   &quot;16rr1-bp3&quot;   &quot;41ce19-f119&quot; &quot;41ce19-f134&quot;</code></pre>
<div class="sourceCode" id="cb96"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb96-1"><a href="geometric-morphometrics.html#cb96-1"></a><span class="co">#group shape means</span></span>
<span id="cb96-2"><a href="geometric-morphometrics.html#cb96-2"></a>mean&lt;-<span class="kw">lapply</span>(new.coords, mshape)</span>
<span id="cb96-3"><a href="geometric-morphometrics.html#cb96-3"></a><span class="co"># plot(mean$CTX)</span></span>
<span id="cb96-4"><a href="geometric-morphometrics.html#cb96-4"></a><span class="co"># plot(mean$SCA-north)</span></span>
<span id="cb96-5"><a href="geometric-morphometrics.html#cb96-5"></a><span class="co"># plotRefToTarget(mean$SCA-north,mean$CTX, method=&quot;vector&quot;,mag=2)</span></span></code></pre></div>
<div class="sourceCode" id="cb97"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb97-1"><a href="geometric-morphometrics.html#cb97-1"></a><span class="co"># knitr::include_graphics(&#39;images/mshape-region.png&#39;)</span></span></code></pre></div>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-RN1655">
<p>Adams, Dean C., and Michael L. Collyer. 2015. “Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters.” <em>Evolution</em> 69 (3): 823–9. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.</p>
</div>
<div id="ref-RN11530">
<p>Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. “Package ’geomorph’: Geometric Morphometric Analyses of 2D/3D Landmark Data. R package version 3.0.5.” <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.</p>
</div>
<div id="ref-RN1774">
<p>Adams, Dean C., and Erik Otárola-Castillo. 2013. “geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data.” <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.</p>
</div>
<div id="ref-RN1719">
<p>Anderson, M. J., and C. J. F. Ter Braak. 2003. “Permutation Tests for Multi-Factoral Analysis of Variance.” <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.</p>
</div>
<div id="ref-RN11775">
<p>Collyer, Michael L., and Dean C. Adams. 2018. “RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization.” <em>Methods in Ecology and Evolution</em> 9 (7): 1772–9. <a href="https://doi.org/https://doi.org/10.1111/2041-210X.13029">https://doi.org/https://doi.org/10.1111/2041-210X.13029</a>.</p>
</div>
<div id="ref-RN1756">
<p>Collyer, M. L., D. J. Sekora, and D. C. Adams. 2015. “A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data.” <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.</p>
</div>
<div id="ref-RN1749">
<p>Goodall, Colin. 1991. “Procrustes Methods in the Statistical Analysis of Shape.” <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.</p>
</div>
<div id="ref-RN11564">
<p>Gower, J. C. 1975. “Generalized Procrustes Analysis.” <em>Psychometrika</em> 40 (1): 33–51. <a href="https://doi.org/https://doi.org/10.1007/BF02291478">https://doi.org/https://doi.org/10.1007/BF02291478</a>.</p>
</div>
<div id="ref-RN1746">
<p>Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.</p>
</div>
<div id="ref-RN11622">
<p>Kendall, David G. 1981. “The Statistics of Shape.” In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.</p>
</div>
<div id="ref-RN11623">
<p>Kendall, David G. 1984. “Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces.” <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.</p>
</div>
<div id="ref-RN1731">
<p>Klingenberg, Christian Peter. 2013. “Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes.” <em>Hystrix</em> 24 (1): 15–24. <a href="https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691">https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.</p>
</div>
<div id="ref-R">
<p>R Core Development Team, 2020. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</p>
</div>
<div id="ref-RN1646">
<p>Rohlf, F. James. 1999. “Shape Statistics: Procrustes Superimpositions and Tangent Spaces.” <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.</p>
</div>
<div id="ref-RN478">
<p>Rohlf, F. James, and Dennis Slice. 1990. “Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks.” <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.</p>
</div>
<div id="ref-RN11801">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p>
</div>
<div id="ref-RN11782">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p>
</div>
<div id="ref-RN11716">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p> 2019. “Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections.” <em>Journal of Cultural Heritage</em> 35: 41–55. <a href="https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002</a>.</p>
</div>
<div id="ref-RN20697">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p> 2019. “Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections.” <em>Journal of Cultural Heritage</em> 35: 41–55. <a href="https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002</a>.</p> 2021. “Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.” In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, (in press). Baton Rouge: LSU Press.</p>
</div>
<div id="ref-RN21001">
<p>Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. “A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.” <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.</p>
</div>
<div id="ref-RN11783">
<p>Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. “Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.</p>
</div>
<div id="ref-RN3684">
<p>Shafer, Harry J. 1973. “Lithic Technology at the George C. Davis Site, Cherokee County, Texas.” Unpublished Ph.D. Dissertation, The University of Texas at Austin.</p>
</div>
<div id="ref-RN20701">
<p>Shafer, Harry J. 1974. “Lithic Reduction Strategies at the George C. Davis Site.” <em>Louisiana Archaeology</em> 1: 66–74. <a href="https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf">https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf</a>.</p>
</div>
<div id="ref-RN4924">
<p>Shafer, Harry J. 2006. “People of the Prairie: A Possible Connection to the Davis Site Caddo.” Texas Department of Transportation; Prewitt &amp; Associates, Inc.</p>
</div>
<div id="ref-RN479">
<p>Sherratt, E., D. J. Gower, C. P. Klingenberg, and M. Wilkinson. 2014. “Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona).” <em>Evolutionary Biology</em> 41 (4): 528–45. <a href="https://doi.org/https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/https://doi.org/10.1007/s11692-014-9287-2</a>.</p>
</div>
<div id="ref-RN11563">
<p>Slice, Dennis E. 2001. “Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space.” <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.</p>
</div>
<div id="ref-RN5703">
<p>Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="landmark-sampling-evaluation-curve-lasec.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["gahaganmorph3.pdf", "gahaganmorph3.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
