<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Geometric morphometrics | Supplementary materials for paper: Temporal and spatial dynamics of Gahagan biface morphology</title>
  <meta name="description" content="Gahagan biface shape change." />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Geometric morphometrics | Supplementary materials for paper: Temporal and spatial dynamics of Gahagan biface morphology" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Gahagan biface shape change." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Geometric morphometrics | Supplementary materials for paper: Temporal and spatial dynamics of Gahagan biface morphology" />
  
  <meta name="twitter:description" content="Gahagan biface shape change." />
  

<meta name="author" content="Robert Z. Selden, Jr." />


<meta name="date" content="2020-09-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="landmark-sampling-evaluation-curve-lasec.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">gahaganmorph3</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminaries</a><ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.1</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.2</b> Funding</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html"><i class="fa fa-check"></i><b>1</b> Temporal Dynamics</a><ul>
<li class="chapter" data-level="1.1" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-5-at-the-mounds-plantation-site"><i class="fa fa-check"></i><b>1.1</b> Calibration and plots for dates from Burial Pit 5 at the Mounds Plantation site</a></li>
<li class="chapter" data-level="1.2" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-burial-pit-2-at-the-gahagan-mound-site"><i class="fa fa-check"></i><b>1.2</b> Calibration and plots for dates from Burial Pit 2 at the Gahagan Mound site</a></li>
<li class="chapter" data-level="1.3" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-f119-and-f134-at-the-george-c.-davis-site"><i class="fa fa-check"></i><b>1.3</b> Calibration and plots for the dates from F119 and F134 at the George C. Davis site</a></li>
<li class="chapter" data-level="1.4" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-dates-from-au2-at-the-j.-b.-white-site"><i class="fa fa-check"></i><b>1.4</b> Calibration and plots for dates from AU2 at the J. B. White site</a></li>
<li class="chapter" data-level="1.5" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#calibration-and-plots-for-the-dates-from-the-baker-site"><i class="fa fa-check"></i><b>1.5</b> Calibration and plots for the dates from the Baker site</a></li>
<li class="chapter" data-level="1.6" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#summed-probability-distributions-for-contexts-yielding-gahagan-bifaces"><i class="fa fa-check"></i><b>1.6</b> Summed probability distributions for contexts yielding Gahagan bifaces</a></li>
<li class="chapter" data-level="1.7" data-path="temporal-dynamics.html"><a href="temporal-dynamics.html#chronological-model-for-sca-north-sca-south-and-central-texas-contexts-yielding-gahagan-bifaces-with-baker-site-dates"><i class="fa fa-check"></i><b>1.7</b> Chronological model for SCA-north, SCA-south, and central Texas contexts yielding Gahagan bifaces with Baker site dates</a></li>
<li class="chapter" data-level="1.8" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>1.8</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="modeling-geometry.html"><a href="modeling-geometry.html"><i class="fa fa-check"></i><b>2</b> Modeling Geometry</a><ul>
<li class="chapter" data-level="2.1" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section"><i class="fa fa-check"></i><b>2.1</b> 4078-8</a></li>
<li class="chapter" data-level="2.2" data-path="modeling-geometry.html"><a href="modeling-geometry.html#section-1"><i class="fa fa-check"></i><b>2.2</b> 4078-72</a></li>
<li class="chapter" data-level="2.3" data-path="modeling-geometry.html"><a href="modeling-geometry.html#references-cited"><i class="fa fa-check"></i><b>2.3</b> References cited</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html"><i class="fa fa-check"></i><b>3</b> Landmarking Protocol 3d2</a><ul>
<li class="chapter" data-level="3.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#foundations"><i class="fa fa-check"></i><b>3.1</b> Foundations</a></li>
<li class="chapter" data-level="3.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#generating-the-peripheral-plan-view-spline"><i class="fa fa-check"></i><b>3.2</b> Generating the peripheral (plan view) spline</a><ul>
<li class="chapter" data-level="3.2.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-spline-1"><i class="fa fa-check"></i><b>3.2.1</b> Splitting Spline 1</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-geometry-build-1"><i class="fa fa-check"></i><b>3.3</b> Reference geometry, Build 1</a><ul>
<li class="chapter" data-level="3.3.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-vector-1-point-2-and-plane-2"><i class="fa fa-check"></i><b>3.3.1</b> Reference Vector 1, Point 2, and Plane 2</a></li>
<li class="chapter" data-level="3.3.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#reference-planes-3-and-4"><i class="fa fa-check"></i><b>3.3.2</b> Reference Planes 3 and 4</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#cross-sections"><i class="fa fa-check"></i><b>3.4</b> Cross-sections</a><ul>
<li class="chapter" data-level="3.4.1" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-1"><i class="fa fa-check"></i><b>3.4.1</b> Splitting the curves (Step 1)</a></li>
<li class="chapter" data-level="3.4.2" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#splitting-the-curves-step-2"><i class="fa fa-check"></i><b>3.4.2</b> Splitting the curves (Step 2)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="landmarking-protocol-3d2.html"><a href="landmarking-protocol-3d2.html#landmark-and-semilandmark-placement"><i class="fa fa-check"></i><b>3.5</b> Landmark and semilandmark placement</a></li>
<li class="chapter" data-level="3.6" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>3.6</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html"><i class="fa fa-check"></i><b>4</b> Landmark Sampling Evaluation Curve (LaSEC)</a><ul>
<li class="chapter" data-level="4.1" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#first-2d-iteration"><i class="fa fa-check"></i><b>4.1</b> First 2D iteration</a></li>
<li class="chapter" data-level="4.2" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#first-3d-iteration"><i class="fa fa-check"></i><b>4.2</b> First 3D iteration</a></li>
<li class="chapter" data-level="4.3" data-path="landmark-sampling-evaluation-curve-lasec.html"><a href="landmark-sampling-evaluation-curve-lasec.html#current-3d-iteration"><i class="fa fa-check"></i><b>4.3</b> Current 3D iteration</a></li>
<li class="chapter" data-level="4.4" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>4.4</b> Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html"><i class="fa fa-check"></i><b>5</b> Geometric morphometrics</a><ul>
<li class="chapter" data-level="5.1" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#load-geomorph-and-data"><i class="fa fa-check"></i><b>5.1</b> Load geomorph and data</a></li>
<li class="chapter" data-level="5.2" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#generalised-procrustes-analysis"><i class="fa fa-check"></i><b>5.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="5.3" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#principal-components-analysis"><i class="fa fa-check"></i><b>5.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="5.4" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#define-models"><i class="fa fa-check"></i><b>5.4</b> Define models</a></li>
<li class="chapter" data-level="5.5" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#allometry"><i class="fa fa-check"></i><b>5.5</b> Allometry</a></li>
<li class="chapter" data-level="5.6" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#sizeshape-region"><i class="fa fa-check"></i><b>5.6</b> Size/Shape ~ Region?</a></li>
<li class="chapter" data-level="5.7" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#morphological-disparity"><i class="fa fa-check"></i><b>5.7</b> Morphological disparity</a></li>
<li class="chapter" data-level="5.8" data-path="geometric-morphometrics.html"><a href="geometric-morphometrics.html#mean-shapes"><i class="fa fa-check"></i><b>5.8</b> Mean shapes</a></li>
<li class="chapter" data-level="5.9" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>5.9</b> Acknowledgments</a></li>
<li class="chapter" data-level="5.10" data-path="modeling-geometry.html"><a href="modeling-geometry.html#references-cited"><i class="fa fa-check"></i><b>5.10</b> References cited</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary materials for paper: Temporal and spatial dynamics of Gahagan biface morphology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="geometric-morphometrics" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Geometric morphometrics</h1>
<p>This effort follows recent analyses of Gahagan biface shapes across the southern Caddo area and central Texas <span class="citation">(Selden Jr., Dockall, and Shafer <a href="#ref-RN11783" role="doc-biblioref">2018</a>; Selden Jr., Dockall, and Dubied <a href="#ref-RN21001" role="doc-biblioref">2020</a>)</span>, where Gahgagan bifaces were found to differ in shape across the same geography as the Smithport Plain and Hickory Engraved bottles <span class="citation">(Selden Jr. <a href="#ref-RN11801" role="doc-biblioref">2018</a><a href="#ref-RN11801" role="doc-biblioref">a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018</a><a href="#ref-RN11782" role="doc-biblioref">b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN20697" role="doc-biblioref">2021</a>)</span>. The analysis builds upon previous discussions and analyses conducted by Shafer <span class="citation">(<a href="#ref-RN3684" role="doc-biblioref">1973</a>, <a href="#ref-RN20701" role="doc-biblioref">1974</a>, <a href="#ref-RN4924" role="doc-biblioref">2006</a>)</span>. A succinct overview of the analytical procedures used for this analysis is provided in the manuscript, and the analytical code provided in this document can be used to reproduce the results exactly.</p>
<div class="sourceCode" id="cb83"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb83-1"><a href="geometric-morphometrics.html#cb83-1"></a>knitr<span class="op">::</span><span class="kw">include_graphics</span>(<span class="st">&#39;images/figure02.jpg&#39;</span>)</span></code></pre></div>
<div class="figure">
<img src="images/figure02.jpg" alt="Gahagan bifaces from the southern Caddo area (SCA) north (left), south (center), and central Texas regions (right)." width="100%" />
<p class="caption">
(#fig:gahagan bifaces 2D)Gahagan bifaces from the southern Caddo area (SCA) north (left), south (center), and central Texas regions (right).
</p>
</div>
<div id="load-geomorph-and-data" class="section level2">
<h2><span class="header-section-number">5.1</span> Load geomorph and data</h2>
<div class="sourceCode" id="cb84"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb84-1"><a href="geometric-morphometrics.html#cb84-1"></a><span class="co"># load geomorph</span></span>
<span id="cb84-2"><a href="geometric-morphometrics.html#cb84-2"></a><span class="co">#devtools::install_github(&quot;geomorphR/geomorph&quot;, ref = &quot;Stable&quot;, build_vignettes = TRUE)</span></span>
<span id="cb84-3"><a href="geometric-morphometrics.html#cb84-3"></a><span class="kw">library</span>(geomorph)</span>
<span id="cb84-4"><a href="geometric-morphometrics.html#cb84-4"></a><span class="kw">library</span>(wesanderson)</span>
<span id="cb84-5"><a href="geometric-morphometrics.html#cb84-5"></a></span>
<span id="cb84-6"><a href="geometric-morphometrics.html#cb84-6"></a><span class="co"># set working directory</span></span>
<span id="cb84-7"><a href="geometric-morphometrics.html#cb84-7"></a><span class="kw">setwd</span>(<span class="kw">getwd</span>())</span>
<span id="cb84-8"><a href="geometric-morphometrics.html#cb84-8"></a><span class="kw">source</span>(<span class="st">&#39;readmulti.csv.R&#39;</span>)</span>
<span id="cb84-9"><a href="geometric-morphometrics.html#cb84-9"></a></span>
<span id="cb84-10"><a href="geometric-morphometrics.html#cb84-10"></a><span class="co"># read .csv files</span></span>
<span id="cb84-11"><a href="geometric-morphometrics.html#cb84-11"></a><span class="kw">setwd</span>(<span class="st">&quot;./data&quot;</span>)</span>
<span id="cb84-12"><a href="geometric-morphometrics.html#cb84-12"></a>filelist &lt;-<span class="st"> </span><span class="kw">list.files</span>(<span class="dt">pattern =</span> <span class="st">&quot;.csv&quot;</span>)</span>
<span id="cb84-13"><a href="geometric-morphometrics.html#cb84-13"></a>coords &lt;-<span class="st"> </span><span class="kw">readmulti.csv</span>(filelist)</span>
<span id="cb84-14"><a href="geometric-morphometrics.html#cb84-14"></a><span class="kw">setwd</span>(<span class="st">&quot;../&quot;</span>)</span>
<span id="cb84-15"><a href="geometric-morphometrics.html#cb84-15"></a></span>
<span id="cb84-16"><a href="geometric-morphometrics.html#cb84-16"></a><span class="co"># read qualitative data</span></span>
<span id="cb84-17"><a href="geometric-morphometrics.html#cb84-17"></a>qdata&lt;-<span class="kw">read.csv</span>(<span class="st">&quot;qdata.csv&quot;</span>,<span class="dt">header=</span><span class="ot">TRUE</span>, <span class="dt">row.names=</span><span class="dv">1</span>)</span>
<span id="cb84-18"><a href="geometric-morphometrics.html#cb84-18"></a>qdata&lt;-qdata[<span class="kw">match</span>(<span class="kw">dimnames</span>(coords)[[<span class="dv">3</span>]],<span class="kw">rownames</span>(qdata)),]</span>
<span id="cb84-19"><a href="geometric-morphometrics.html#cb84-19"></a>qdata</span></code></pre></div>
<pre><code>##                Type trinomial          sitename region
## 2300        Gahagan     bvmnh Doerge Collection    CTX
## 2302        Gahagan     bvmnh Doerge Collection    CTX
## 2303        Gahagan     bvmnh Doerge Collection    CTX
## 2304        Gahagan     bvmnh Doerge Collection    CTX
## 2305        Gahagan     bvmnh Doerge Collection    CTX
## 2317        Gahagan     bvmnh Doerge Collection    CTX
## 2318        Gahagan     bvmnh Doerge Collection    CTX
## 2319        Gahagan     bvmnh Doerge Collection    CTX
## 2320        Gahagan     bvmnh Doerge Collection    CTX
## 2322        Gahagan     bvmnh Doerge Collection    CTX
## 2323        Gahagan     bvmnh Doerge Collection    CTX
## 2324        Gahagan     bvmnh Doerge Collection    CTX
## 2325        Gahagan     bvmnh Doerge Collection    CTX
## 2361        Gahagan     bvmnh Doerge Collection    CTX
## 2362        Gahagan     bvmnh Doerge Collection    CTX
## 2363        Gahagan     bvmnh Doerge Collection    CTX
## 2364        Gahagan     bvmnh Doerge Collection    CTX
## 2365        Gahagan     bvmnh Doerge Collection    CTX
## 2366        Gahagan     bvmnh Doerge Collection    CTX
## 2419        Gahagan     bvmnh Doerge Collection    CTX
## 2426        Gahagan     bvmnh Doerge Collection    CTX
## 3Ba90       Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb1        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb3        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb4        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb5        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb6        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb7        Gahagan    16CD12 Mounds Plantation    SCA
## 3Bb8        Gahagan    16CD12 Mounds Plantation    SCA
## 4078-11     Gahagan    41CE19    George C Davis    SCA
## 4078-12     Gahagan    41CE19    George C Davis    SCA
## 4078-13     Gahagan    41CE19    George C Davis    SCA
## 4078-14     Gahagan    41CE19    George C Davis    SCA
## 4078-14B    Gahagan    41CE19    George C Davis    SCA
## 4078-22     Gahagan    41CE19    George C Davis    SCA
## 4078-30     Gahagan    41CE19    George C Davis    SCA
## 4078-32     Gahagan    41CE19    George C Davis    SCA
## 4078-45     Gahagan    41CE19    George C Davis    SCA
## 4078-72     Gahagan    41CE19    George C Davis    SCA
## 4078-8      Gahagan    41CE19    George C Davis    SCA
## 4078-9      Gahagan    41CE19    George C Davis    SCA
## 424-169     Gahagan    41CE19    George C Davis    SCA
## 424-218     Gahagan    41CE19    George C Davis    SCA
## 424-221     Gahagan    41CE19    George C Davis    SCA
## 424-230     Gahagan    41CE19    George C Davis    SCA
## 424-33      Gahagan    41CE19    George C Davis    SCA
## 424-39      Gahagan    41CE19    George C Davis    SCA
## 424-40      Gahagan    41CE19    George C Davis    SCA
## 424-41      Gahagan    41CE19    George C Davis    SCA
## 424-50      Gahagan    41CE19    George C Davis    SCA
## 424-53      Gahagan    41CE19    George C Davis    SCA
## 463-1       Gahagan    41CE19    George C Davis    SCA
## 463-16      Gahagan    41CE19    George C Davis    SCA
## 463-23      Gahagan    41CE19    George C Davis    SCA
## 489         Gahagan     16RR1     Gahagan Mound    SCA
## 490         Gahagan     16RR1     Gahagan Mound    SCA
## 532         Gahagan     16RR1     Gahagan Mound    SCA
## 533         Gahagan     16RR1     Gahagan Mound    SCA
## 541         Gahagan     16RR1     Gahagan Mound    SCA
## 542         Gahagan     16RR1     Gahagan Mound    SCA
## 543         Gahagan     16RR1     Gahagan Mound    SCA
## 544         Gahagan     16RR1     Gahagan Mound    SCA
## 545         Gahagan     16RR1     Gahagan Mound    SCA
## 546         Gahagan     16RR1     Gahagan Mound    SCA
## 547         Gahagan     16RR1     Gahagan Mound    SCA
## 548         Gahagan     16RR1     Gahagan Mound    SCA
## 549         Gahagan     16RR1     Gahagan Mound    SCA
## 550         Gahagan     16RR1     Gahagan Mound    SCA
## 551         Gahagan     16RR1     Gahagan Mound    SCA
## 569         Gahagan     16RR1     Gahagan Mound    SCA
## 593         Gahagan     16RR1     Gahagan Mound    SCA
## 605         Gahagan     16RR1     Gahagan Mound    SCA
## 606         Gahagan     16RR1     Gahagan Mound    SCA
## 607         Gahagan     16RR1     Gahagan Mound    SCA
## 608         Gahagan     16RR1     Gahagan Mound    SCA
## 609         Gahagan     16RR1     Gahagan Mound    SCA
## 610         Gahagan     16RR1     Gahagan Mound    SCA
## 611         Gahagan     16RR1     Gahagan Mound    SCA
## 612         Gahagan     16RR1     Gahagan Mound    SCA
## 613         Gahagan     16RR1     Gahagan Mound    SCA
## 614         Gahagan     16RR1     Gahagan Mound    SCA
## 622         Gahagan     16RR1     Gahagan Mound    SCA
## 666         Gahagan     16RR1     Gahagan Mound    SCA
## BlkThn      Gahagan    16CD12 Mounds Plantation    SCA
## bsp         Gahagan   41BP882        Bastrop SP    CTX
## Case2LG     Gahagan    16CD12 Mounds Plantation    SCA
## Case2SM     Gahagan    16CD12 Mounds Plantation    SCA
## D11         Gahagan        LA           Pelican    SCA
## DM1         Gahagan        LA         Doc Marks    SCA
## ET221-1016  Gahagan    41CE19    George C Davis    SCA
## ET221-1260A Gahagan    41CE19    George C Davis    SCA
## ET221-993   Gahagan    41CE19    George C Davis    SCA
## Lf50-3      Gahagan      LF50             Spiro    SCA
## LGGray      Gahagan    16CD12 Mounds Plantation    SCA
## unk1        Gahagan     bvmnh             bvmnh    CTX
## unk2        Gahagan     bvmnh             bvmnh    CTX
## unk3        Gahagan     bvmnh             bvmnh    CTX
## unk4        Gahagan     bvmnh             bvmnh    CTX
## unk5        Gahagan     bvmnh             bvmnh    CTX
## unk6        Gahagan     bvmnh             bvmnh    CTX
## unk7        Gahagan     bvmnh             bvmnh    CTX
## unk8        Gahagan     bvmnh             bvmnh    CTX
##               region2
## 2300              CTX
## 2302              CTX
## 2303              CTX
## 2304              CTX
## 2305              CTX
## 2317              CTX
## 2318              CTX
## 2319              CTX
## 2320              CTX
## 2322              CTX
## 2323              CTX
## 2324              CTX
## 2325              CTX
## 2361              CTX
## 2362              CTX
## 2363              CTX
## 2364              CTX
## 2365              CTX
## 2366              CTX
## 2419              CTX
## 2426              CTX
## 3Ba90       SCA-north
## 3Bb1        SCA-north
## 3Bb3        SCA-north
## 3Bb4        SCA-north
## 3Bb5        SCA-north
## 3Bb6        SCA-north
## 3Bb7        SCA-north
## 3Bb8        SCA-north
## 4078-11     SCA-south
## 4078-12     SCA-south
## 4078-13     SCA-south
## 4078-14     SCA-south
## 4078-14B    SCA-south
## 4078-22     SCA-south
## 4078-30     SCA-south
## 4078-32     SCA-south
## 4078-45     SCA-south
## 4078-72     SCA-south
## 4078-8      SCA-south
## 4078-9      SCA-south
## 424-169     SCA-south
## 424-218     SCA-south
## 424-221     SCA-south
## 424-230     SCA-south
## 424-33      SCA-south
## 424-39      SCA-south
## 424-40      SCA-south
## 424-41      SCA-south
## 424-50      SCA-south
## 424-53      SCA-south
## 463-1       SCA-south
## 463-16      SCA-south
## 463-23      SCA-south
## 489         SCA-south
## 490         SCA-south
## 532         SCA-south
## 533         SCA-south
## 541         SCA-south
## 542         SCA-south
## 543         SCA-south
## 544         SCA-south
## 545         SCA-south
## 546         SCA-south
## 547         SCA-south
## 548         SCA-south
## 549         SCA-south
## 550         SCA-south
## 551         SCA-south
## 569         SCA-south
## 593         SCA-south
## 605         SCA-south
## 606         SCA-south
## 607         SCA-south
## 608         SCA-south
## 609         SCA-south
## 610         SCA-south
## 611         SCA-south
## 612         SCA-south
## 613         SCA-south
## 614         SCA-south
## 622         SCA-south
## 666         SCA-south
## BlkThn      SCA-north
## bsp               CTX
## Case2LG     SCA-north
## Case2SM     SCA-north
## D11         SCA-south
## DM1         SCA-south
## ET221-1016  SCA-south
## ET221-1260A SCA-south
## ET221-993   SCA-south
## Lf50-3      SCA-north
## LGGray      SCA-north
## unk1              CTX
## unk2              CTX
## unk3              CTX
## unk4              CTX
## unk5              CTX
## unk6              CTX
## unk7              CTX
## unk8              CTX</code></pre>
</div>
<div id="generalised-procrustes-analysis" class="section level2">
<h2><span class="header-section-number">5.2</span> Generalised Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(Kendall <a href="#ref-RN11622" role="doc-biblioref">1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span> performed in R 4.0.2 <span class="citation">(R Core Development Team, <a href="#ref-R" role="doc-biblioref">2020</a>)</span> using the <code>geomorph</code> library v. 3.3.1 <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>; Adams and Otárola-Castillo <a href="#ref-RN1774" role="doc-biblioref">2013</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(Gower <a href="#ref-RN11564" role="doc-biblioref">1975</a>; Rohlf and Slice <a href="#ref-RN478" role="doc-biblioref">1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(Rohlf <a href="#ref-RN1646" role="doc-biblioref">1999</a>; Slice <a href="#ref-RN11563" role="doc-biblioref">2001</a>)</span>.</p>
<div class="sourceCode" id="cb86"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb86-1"><a href="geometric-morphometrics.html#cb86-1"></a>Y.gpa&lt;-<span class="kw">gpagen</span>(coords, <span class="dt">PrinAxes =</span> <span class="ot">TRUE</span>, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>)</span>
<span id="cb86-2"><a href="geometric-morphometrics.html#cb86-2"></a><span class="co"># plot(Y.gpa)</span></span>
<span id="cb86-3"><a href="geometric-morphometrics.html#cb86-3"></a></span>
<span id="cb86-4"><a href="geometric-morphometrics.html#cb86-4"></a><span class="co"># 3D GPA plot</span></span>
<span id="cb86-5"><a href="geometric-morphometrics.html#cb86-5"></a>knitr<span class="op">::</span><span class="kw">include_graphics</span>(<span class="st">&#39;images/gpa3d.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span id="fig:gpa"></span>
<img src="images/gpa3d.png" alt="Results of generalized Procrustes analysis." width="100%" />
<p class="caption">
Figure 5.1: Results of generalized Procrustes analysis.
</p>
</div>
<div class="sourceCode" id="cb87"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb87-1"><a href="geometric-morphometrics.html#cb87-1"></a><span class="co"># geomorph data frame</span></span>
<span id="cb87-2"><a href="geometric-morphometrics.html#cb87-2"></a>gdf&lt;-<span class="kw">geomorph.data.frame</span>(<span class="dt">shape=</span>Y.gpa<span class="op">$</span>coords, <span class="dt">size=</span>Y.gpa<span class="op">$</span>Csize, <span class="dt">region=</span>qdata<span class="op">$</span>region, <span class="dt">region2=</span>qdata<span class="op">$</span>region2) </span>
<span id="cb87-3"><a href="geometric-morphometrics.html#cb87-3"></a></span>
<span id="cb87-4"><a href="geometric-morphometrics.html#cb87-4"></a><span class="co"># boxplot - region 2 by centroid size</span></span>
<span id="cb87-5"><a href="geometric-morphometrics.html#cb87-5"></a>csz&lt;-Y.gpa<span class="op">$</span>Csize <span class="co"># attribute for boxplot</span></span>
<span id="cb87-6"><a href="geometric-morphometrics.html#cb87-6"></a>rgn2&lt;-qdata<span class="op">$</span>region2 <span class="co"># attribute for boxplot</span></span>
<span id="cb87-7"><a href="geometric-morphometrics.html#cb87-7"></a><span class="co"># boxplot of SCA-north, SCA-south, and central Texas Gahagan bifaces by centroid size</span></span>
<span id="cb87-8"><a href="geometric-morphometrics.html#cb87-8"></a><span class="kw">boxplot</span>(csz<span class="op">~</span>rgn2, </span>
<span id="cb87-9"><a href="geometric-morphometrics.html#cb87-9"></a>        <span class="dt">names =</span> <span class="kw">c</span>(<span class="st">&quot;CTX&quot;</span>, <span class="st">&quot;SCA-north&quot;</span>, <span class="st">&quot;SCA-south&quot;</span>),</span>
<span id="cb87-10"><a href="geometric-morphometrics.html#cb87-10"></a>        <span class="dt">xlab =</span> <span class="st">&quot;Region&quot;</span>,</span>
<span id="cb87-11"><a href="geometric-morphometrics.html#cb87-11"></a>        <span class="dt">ylab =</span> <span class="st">&quot;Centroid Size&quot;</span>,</span>
<span id="cb87-12"><a href="geometric-morphometrics.html#cb87-12"></a>        <span class="dt">col =</span> <span class="kw">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>),</span>
<span id="cb87-13"><a href="geometric-morphometrics.html#cb87-13"></a>        )</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/centsize.box-1.png" alt="Centroid size of Gahagan bifaces by Region 2." width="100%" />
<p class="caption">
(#fig:centsize.box)Centroid size of Gahagan bifaces by Region 2.
</p>
</div>
<div class="sourceCode" id="cb88"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb88-1"><a href="geometric-morphometrics.html#cb88-1"></a><span class="co"># plot consensus configuration in 2D to illustrate axial twisting</span></span>
<span id="cb88-2"><a href="geometric-morphometrics.html#cb88-2"></a><span class="kw">par</span>(<span class="dt">mfrow=</span><span class="kw">c</span>(<span class="dv">1</span>,<span class="dv">3</span>))</span>
<span id="cb88-3"><a href="geometric-morphometrics.html#cb88-3"></a><span class="kw">plot</span>(Y.gpa<span class="op">$</span>consensus[,<span class="kw">c</span>(<span class="st">&quot;Y&quot;</span>, <span class="st">&quot;X&quot;</span>)], <span class="dt">pch=</span><span class="dv">20</span>)</span>
<span id="cb88-4"><a href="geometric-morphometrics.html#cb88-4"></a><span class="kw">plot</span>(Y.gpa<span class="op">$</span>consensus[,<span class="kw">c</span>(<span class="st">&quot;Z&quot;</span>, <span class="st">&quot;X&quot;</span>)], <span class="dt">pch=</span><span class="dv">20</span>)</span>
<span id="cb88-5"><a href="geometric-morphometrics.html#cb88-5"></a><span class="kw">plot</span>(Y.gpa<span class="op">$</span>consensus[,<span class="kw">c</span>(<span class="st">&quot;Z&quot;</span>, <span class="st">&quot;Y&quot;</span>)], <span class="dt">pch=</span><span class="dv">20</span>)</span></code></pre></div>
<div class="figure">
<img src="gahaganmorph3_files/figure-html/consensus.config-1.png" alt="Two-dimensional plot of consensus configuration illustrating morphological characteristics associated with axial twisting---middle and right---which were absent from the 2D analysis." width="100%" />
<p class="caption">
(#fig:consensus.config)Two-dimensional plot of consensus configuration illustrating morphological characteristics associated with axial twisting—middle and right—which were absent from the 2D analysis.
</p>
</div>
</div>
<div id="principal-components-analysis" class="section level2">
<h2><span class="header-section-number">5.3</span> Principal Components Analysis</h2>
<p>Principal components analysis <span class="citation">(Jolliffe <a href="#ref-RN1746" role="doc-biblioref">2002</a>)</span> was used to visualise shape variation among the bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(Klingenberg <a href="#ref-RN1731" role="doc-biblioref">2013</a>; Sherratt et al. <a href="#ref-RN479" role="doc-biblioref">2014</a>)</span>.</p>
<div class="sourceCode" id="cb89"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb89-1"><a href="geometric-morphometrics.html#cb89-1"></a><span class="co"># principal components analysis</span></span>
<span id="cb89-2"><a href="geometric-morphometrics.html#cb89-2"></a>pca&lt;-<span class="kw">gm.prcomp</span>(Y.gpa<span class="op">$</span>coords)</span>
<span id="cb89-3"><a href="geometric-morphometrics.html#cb89-3"></a><span class="kw">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering and projection: OLS 
## Number of observations 102 
## Number of vectors 102 
## 
## Importance of Components:
##                              Comp1        Comp2        Comp3
## Eigenvalues            0.003456919 0.0008015117 0.0003561083
## Proportion of Variance 0.624607852 0.1448198637 0.0643428519
## Cumulative Proportion  0.624607852 0.7694277161 0.8337705680
##                               Comp4       Comp5        Comp6
## Eigenvalues            0.0002230681 0.000201588 9.695002e-05
## Proportion of Variance 0.0403047130 0.036423602 1.751726e-02
## Cumulative Proportion  0.8740752810 0.910498883 9.280161e-01
##                               Comp7        Comp8        Comp9
## Eigenvalues            8.436491e-05 5.604465e-05 4.496637e-05
## Proportion of Variance 1.524334e-02 1.012634e-02 8.124677e-03
## Cumulative Proportion  9.432595e-01 9.533858e-01 9.615105e-01
##                              Comp10       Comp11       Comp12
## Eigenvalues            2.705187e-05 2.237266e-05 1.879107e-05
## Proportion of Variance 4.887824e-03 4.042368e-03 3.395234e-03
## Cumulative Proportion  9.663983e-01 9.704407e-01 9.738359e-01
##                              Comp13       Comp14       Comp15
## Eigenvalues            1.540029e-05 0.0000110222 1.040191e-05
## Proportion of Variance 2.782577e-03 0.0019915288 1.879453e-03
## Cumulative Proportion  9.766185e-01 0.9786100299 9.804895e-01
##                              Comp16       Comp17       Comp18
## Eigenvalues            8.987219e-06 8.549225e-06 7.318254e-06
## Proportion of Variance 1.623841e-03 1.544703e-03 1.322287e-03
## Cumulative Proportion  9.821133e-01 9.836580e-01 9.849803e-01
##                              Comp19       Comp20       Comp21
## Eigenvalues            6.361362e-06 5.314828e-06 4.813501e-06
## Proportion of Variance 1.149393e-03 9.603013e-04 8.697197e-04
## Cumulative Proportion  9.861297e-01 9.870900e-01 9.879597e-01
##                              Comp22       Comp23       Comp24
## Eigenvalues            4.055465e-06 3.815912e-06 3.582330e-06
## Proportion of Variance 7.327553e-04 6.894720e-04 6.472675e-04
## Cumulative Proportion  9.886925e-01 9.893820e-01 9.900292e-01
##                              Comp25       Comp26       Comp27
## Eigenvalues            3.502677e-06 3.217001e-06 2.912249e-06
## Proportion of Variance 6.328756e-04 5.812587e-04 5.261951e-04
## Cumulative Proportion  9.906621e-01 9.912434e-01 9.917696e-01
##                              Comp28       Comp29       Comp30
## Eigenvalues            2.824777e-06 2.501451e-06 2.338948e-06
## Proportion of Variance 5.103904e-04 4.519707e-04 4.226091e-04
## Cumulative Proportion  9.922799e-01 9.927319e-01 9.931545e-01
##                              Comp31       Comp32       Comp33
## Eigenvalues            2.179895e-06 2.054475e-06 1.939460e-06
## Proportion of Variance 3.938709e-04 3.712096e-04 3.504283e-04
## Cumulative Proportion  9.935484e-01 9.939196e-01 9.942700e-01
##                              Comp34       Comp35       Comp36
## Eigenvalues            1.886723e-06 1.825971e-06 1.778283e-06
## Proportion of Variance 3.408995e-04 3.299226e-04 3.213062e-04
## Cumulative Proportion  9.946109e-01 9.949409e-01 9.952622e-01
##                              Comp37       Comp38       Comp39
## Eigenvalues            1.590082e-06 1.514771e-06 1.397117e-06
## Proportion of Variance 2.873014e-04 2.736940e-04 2.524359e-04
## Cumulative Proportion  9.955495e-01 9.958232e-01 9.960756e-01
##                              Comp40       Comp41       Comp42
## Eigenvalues            1.310930e-06 1.194399e-06 1.143650e-06
## Proportion of Variance 2.368633e-04 2.158081e-04 2.066385e-04
## Cumulative Proportion  9.963125e-01 9.965283e-01 9.967349e-01
##                              Comp43       Comp44       Comp45
## Eigenvalues            1.084225e-06 9.552895e-07 9.315219e-07
## Proportion of Variance 1.959016e-04 1.726050e-04 1.683106e-04
## Cumulative Proportion  9.969308e-01 9.971034e-01 9.972717e-01
##                              Comp46       Comp47       Comp48
## Eigenvalues            9.195996e-07 8.443343e-07 8.158838e-07
## Proportion of Variance 1.661564e-04 1.525572e-04 1.474167e-04
## Cumulative Proportion  9.974379e-01 9.975904e-01 9.977378e-01
##                              Comp49       Comp50       Comp51
## Eigenvalues            7.740276e-07 7.409567e-07 7.274802e-07
## Proportion of Variance 1.398540e-04 1.338786e-04 1.314436e-04
## Cumulative Proportion  9.978777e-01 9.980116e-01 9.981430e-01
##                              Comp52       Comp53       Comp54
## Eigenvalues            6.601756e-07 6.362967e-07 5.823388e-07
## Proportion of Variance 1.192828e-04 1.149682e-04 1.052190e-04
## Cumulative Proportion  9.982623e-01 9.983773e-01 9.984825e-01
##                              Comp55       Comp56       Comp57
## Eigenvalues            5.451560e-07 5.084282e-07 4.731003e-07
## Proportion of Variance 9.850065e-05 9.186453e-05 8.548137e-05
## Cumulative Proportion  9.985810e-01 9.986729e-01 9.987583e-01
##                              Comp58       Comp59       Comp60
## Eigenvalues            4.693755e-07 4.137550e-07 4.029531e-07
## Proportion of Variance 8.480837e-05 7.475866e-05 7.280694e-05
## Cumulative Proportion  9.988431e-01 9.989179e-01 9.989907e-01
##                              Comp61       Comp62       Comp63
## Eigenvalues            3.896934e-07 3.640046e-07 3.513582e-07
## Proportion of Variance 7.041114e-05 6.576960e-05 6.348460e-05
## Cumulative Proportion  9.990611e-01 9.991269e-01 9.991904e-01
##                              Comp64       Comp65       Comp66
## Eigenvalues            3.214590e-07 3.109272e-07 2.829565e-07
## Proportion of Variance 5.808231e-05 5.617938e-05 5.112554e-05
## Cumulative Proportion  9.992485e-01 9.993046e-01 9.993558e-01
##                              Comp67       Comp68       Comp69
## Eigenvalues            2.726509e-07 2.535681e-07 2.451163e-07
## Proportion of Variance 4.926349e-05 4.581555e-05 4.428844e-05
## Cumulative Proportion  9.994050e-01 9.994508e-01 9.994951e-01
##                              Comp70       Comp71       Comp72
## Eigenvalues            2.208437e-07 2.048011e-07 1.939748e-07
## Proportion of Variance 3.990279e-05 3.700417e-05 3.504804e-05
## Cumulative Proportion  9.995350e-01 9.995720e-01 9.996071e-01
##                              Comp73       Comp74       Comp75
## Eigenvalues            1.799089e-07 1.686136e-07 1.565267e-07
## Proportion of Variance 3.250655e-05 3.046567e-05 2.828177e-05
## Cumulative Proportion  9.996396e-01 9.996701e-01 9.996983e-01
##                              Comp76       Comp77       Comp78
## Eigenvalues            1.511513e-07 1.390247e-07 1.261567e-07
## Proportion of Variance 2.731053e-05 2.511946e-05 2.279443e-05
## Cumulative Proportion  9.997257e-01 9.997508e-01 9.997736e-01
##                              Comp79       Comp80       Comp81
## Eigenvalues            1.193092e-07 1.061959e-07 1.003169e-07
## Proportion of Variance 2.155720e-05 1.918783e-05 1.812559e-05
## Cumulative Proportion  9.997951e-01 9.998143e-01 9.998324e-01
##                              Comp82       Comp83       Comp84
## Eigenvalues            9.211416e-08 8.807853e-08 8.202946e-08
## Proportion of Variance 1.664350e-05 1.591433e-05 1.482136e-05
## Cumulative Proportion  9.998491e-01 9.998650e-01 9.998798e-01
##                              Comp85       Comp86       Comp87
## Eigenvalues            7.747433e-08 7.308207e-08 6.540497e-08
## Proportion of Variance 1.399833e-05 1.320472e-05 1.181759e-05
## Cumulative Proportion  9.998938e-01 9.999070e-01 9.999188e-01
##                              Comp88       Comp89       Comp90
## Eigenvalues            6.143039e-08 5.377294e-08 4.885165e-08
## Proportion of Variance 1.109945e-05 9.715878e-06 8.826683e-06
## Cumulative Proportion  9.999299e-01 9.999397e-01 9.999485e-01
##                              Comp91       Comp92       Comp93
## Eigenvalues            4.717929e-08 4.032717e-08 3.837788e-08
## Proportion of Variance 8.524515e-06 7.286451e-06 6.934247e-06
## Cumulative Proportion  9.999570e-01 9.999643e-01 9.999712e-01
##                              Comp94       Comp95       Comp96
## Eigenvalues            3.371704e-08 2.560835e-08 2.370381e-08
## Proportion of Variance 6.092109e-06 4.627005e-06 4.282885e-06
## Cumulative Proportion  9.999773e-01 9.999819e-01 9.999862e-01
##                              Comp97       Comp98       Comp99
## Eigenvalues            2.025419e-08 1.983573e-08 1.544328e-08
## Proportion of Variance 3.659595e-06 3.583987e-06 2.790345e-06
## Cumulative Proportion  9.999899e-01 9.999935e-01 9.999963e-01
##                             Comp100      Comp101      Comp102
## Eigenvalues            1.290993e-08 7.781488e-09 1.037741e-33
## Proportion of Variance 2.332611e-06 1.405986e-06 1.875026e-31
## Cumulative Proportion  9.999986e-01 1.000000e+00 1.000000e+00</code></pre>
<div class="sourceCode" id="cb91"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb91-1"><a href="geometric-morphometrics.html#cb91-1"></a><span class="co"># set plot parameters to plot by region2</span></span>
<span id="cb91-2"><a href="geometric-morphometrics.html#cb91-2"></a>reg2 &lt;-<span class="st"> </span>qdata<span class="op">$</span>region2</span>
<span id="cb91-3"><a href="geometric-morphometrics.html#cb91-3"></a>pch.gps.reg2 &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="dv">15</span>,<span class="dv">17</span>,<span class="dv">19</span>)[<span class="kw">as.factor</span>(reg2)]</span>
<span id="cb91-4"><a href="geometric-morphometrics.html#cb91-4"></a>col.gps.reg2 &lt;-<span class="st"> </span><span class="kw">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)[<span class="kw">as.factor</span>(reg2)]</span>
<span id="cb91-5"><a href="geometric-morphometrics.html#cb91-5"></a>col.hull &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="st">&quot;#798E87&quot;</span>,<span class="st">&quot;#C27D38&quot;</span>,<span class="st">&quot;#CCC591&quot;</span>)</span>
<span id="cb91-6"><a href="geometric-morphometrics.html#cb91-6"></a></span>
<span id="cb91-7"><a href="geometric-morphometrics.html#cb91-7"></a><span class="co"># plot pca by region 2</span></span>
<span id="cb91-8"><a href="geometric-morphometrics.html#cb91-8"></a>pc.plot1 &lt;-<span class="st"> </span><span class="kw">plot</span>(pca, <span class="dt">asp =</span> <span class="dv">1</span>,</span>
<span id="cb91-9"><a href="geometric-morphometrics.html#cb91-9"></a>                  <span class="dt">pch =</span> pch.gps.reg2,</span>
<span id="cb91-10"><a href="geometric-morphometrics.html#cb91-10"></a>                  <span class="dt">col =</span> col.gps.reg2)</span>
<span id="cb91-11"><a href="geometric-morphometrics.html#cb91-11"></a>                  <span class="kw">shapeHulls</span>(pc.plot1, </span>
<span id="cb91-12"><a href="geometric-morphometrics.html#cb91-12"></a>                             <span class="dt">groups =</span> reg2,</span>
<span id="cb91-13"><a href="geometric-morphometrics.html#cb91-13"></a>                             <span class="dt">group.cols =</span> col.hull)</span></code></pre></div>
<div class="figure"><span id="fig:pca"></span>
<img src="gahaganmorph3_files/figure-html/pca-1.png" alt="Results of PCA with central Texas sample in gray squares, southern Caddo area (north) in orange triangles, and southern Caddo area (south) in tan circles." width="100%" />
<p class="caption">
Figure 5.2: Results of PCA with central Texas sample in gray squares, southern Caddo area (north) in orange triangles, and southern Caddo area (south) in tan circles.
</p>
</div>
<div class="sourceCode" id="cb92"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb92-1"><a href="geometric-morphometrics.html#cb92-1"></a><span class="co"># plot PCA with warp grids</span></span>
<span id="cb92-2"><a href="geometric-morphometrics.html#cb92-2"></a><span class="co">#knitr::include_graphics(&#39;images/pca-warp-ref.png&#39;)</span></span></code></pre></div>
</div>
<div id="define-models" class="section level2">
<h2><span class="header-section-number">5.4</span> Define models</h2>
<div class="sourceCode" id="cb93"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb93-1"><a href="geometric-morphometrics.html#cb93-1"></a><span class="co"># allometry</span></span>
<span id="cb93-2"><a href="geometric-morphometrics.html#cb93-2"></a>fit.size &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb93-3"><a href="geometric-morphometrics.html#cb93-3"></a><span class="co"># allometry - common allometry, different means -&gt; reg2</span></span>
<span id="cb93-4"><a href="geometric-morphometrics.html#cb93-4"></a>fit.sz.reg &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size <span class="op">+</span><span class="st"> </span>reg2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb93-5"><a href="geometric-morphometrics.html#cb93-5"></a><span class="co"># allometry - unique allometries -&gt; reg2</span></span>
<span id="cb93-6"><a href="geometric-morphometrics.html#cb93-6"></a>fit.sz.reg2 &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>size <span class="op">*</span><span class="st"> </span>reg2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb93-7"><a href="geometric-morphometrics.html#cb93-7"></a></span>
<span id="cb93-8"><a href="geometric-morphometrics.html#cb93-8"></a><span class="co"># size as a function of group</span></span>
<span id="cb93-9"><a href="geometric-morphometrics.html#cb93-9"></a>fit.sizereg2 &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(size <span class="op">~</span><span class="st"> </span>reg2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb93-10"><a href="geometric-morphometrics.html#cb93-10"></a></span>
<span id="cb93-11"><a href="geometric-morphometrics.html#cb93-11"></a><span class="co"># shape as a function of group</span></span>
<span id="cb93-12"><a href="geometric-morphometrics.html#cb93-12"></a>fit.shapereg2 &lt;-<span class="st"> </span><span class="kw">procD.lm</span>(shape <span class="op">~</span><span class="st"> </span>reg2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
</div>
<div id="allometry" class="section level2">
<h2><span class="header-section-number">5.5</span> Allometry</h2>
<div class="sourceCode" id="cb94"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb94-1"><a href="geometric-morphometrics.html#cb94-1"></a><span class="co"># allometry - does shape change with size?</span></span>
<span id="cb94-2"><a href="geometric-morphometrics.html#cb94-2"></a><span class="kw">anova</span>(fit.size)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## size        1 0.04166 0.041664 0.07453 8.0538 3.0348  6e-04 ***
## Residuals 100 0.51732 0.005173 0.92547                         
## Total     101 0.55899                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ size, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb96"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb96-1"><a href="geometric-morphometrics.html#cb96-1"></a><span class="co"># reg2</span></span>
<span id="cb96-2"><a href="geometric-morphometrics.html#cb96-2"></a><span class="kw">anova</span>(fit.sz.reg) <span class="co"># common allometry (reg2)</span></span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## size        1 0.04166 0.041664 0.07453 10.639 3.3865  1e-04 ***
## reg2        2 0.13352 0.066760 0.23886 17.046 4.9948  1e-04 ***
## Residuals  98 0.38380 0.003916 0.68660                         
## Total     101 0.55899                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ size + reg2, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb98"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb98-1"><a href="geometric-morphometrics.html#cb98-1"></a><span class="kw">anova</span>(fit.sz.reg2) <span class="co"># unique allometry (reg2)</span></span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq       F      Z Pr(&gt;F)
## size        1 0.04166 0.041664 0.07453 10.7071 3.3932  1e-04
## reg2        2 0.13352 0.066760 0.23886 17.1564 5.0036  1e-04
## size:reg2   2 0.01024 0.005122 0.01832  1.3162 0.7733 0.2238
## Residuals  96 0.37356 0.003891 0.66828                      
## Total     101 0.55899                                       
##              
## size      ***
## reg2      ***
## size:reg2    
## Residuals    
## Total        
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ size * reg2, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb100"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb100-1"><a href="geometric-morphometrics.html#cb100-1"></a><span class="kw">anova</span>(fit.sz.reg2, fit.sz.reg2, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Effect sizes (Z) based on F distributions
## 
##          ResDf Df     RSS    SS MS   Rsq F Z  P Pr(&gt;F)
## 1 (Null)    96  1 0.37356          1e-32              
## 2           96  0 0.37356 1e-32    1e-32      1       
## Total      101    0.55899</code></pre>
<div class="sourceCode" id="cb102"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb102-1"><a href="geometric-morphometrics.html#cb102-1"></a><span class="co"># allometry plots</span></span>
<span id="cb102-2"><a href="geometric-morphometrics.html#cb102-2"></a><span class="co"># regscore (Drake and Klingenberg 2008)</span></span>
<span id="cb102-3"><a href="geometric-morphometrics.html#cb102-3"></a><span class="kw">plot</span>(fit.size, <span class="dt">type =</span> <span class="st">&quot;regression&quot;</span>, <span class="dt">reg.type =</span> <span class="st">&quot;RegScore&quot;</span>, <span class="dt">predictor =</span> <span class="kw">log</span>(gdf<span class="op">$</span>size), <span class="dt">pch =</span> pch.gps.reg2, <span class="dt">col =</span> col.gps.reg2)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-1.png" width="100%" /></p>
<div class="sourceCode" id="cb103"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb103-1"><a href="geometric-morphometrics.html#cb103-1"></a><span class="co"># common allometric component (Mitteroecker 2004)</span></span>
<span id="cb103-2"><a href="geometric-morphometrics.html#cb103-2"></a><span class="kw">plotAllometry</span>(fit.sz.reg, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, <span class="dt">method =</span> <span class="st">&quot;CAC&quot;</span>, <span class="dt">pch =</span> pch.gps.reg2, <span class="dt">col =</span> col.gps.reg2)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-2.png" width="100%" /></p>
<div class="sourceCode" id="cb104"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb104-1"><a href="geometric-morphometrics.html#cb104-1"></a><span class="co"># size-shape pca (Mitteroecker 2004)</span></span>
<span id="cb104-2"><a href="geometric-morphometrics.html#cb104-2"></a><span class="kw">plotAllometry</span>(fit.size, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, <span class="dt">method =</span> <span class="st">&quot;size.shape&quot;</span>, <span class="dt">pch =</span> pch.gps.reg2, <span class="dt">col =</span> col.gps.reg2)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-3.png" width="100%" /></p>
<div class="sourceCode" id="cb105"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb105-1"><a href="geometric-morphometrics.html#cb105-1"></a><span class="co"># predline (Adams and Nistri 2010)</span></span>
<span id="cb105-2"><a href="geometric-morphometrics.html#cb105-2"></a><span class="kw">plotAllometry</span>(fit.sz.reg2, <span class="dt">size =</span> gdf<span class="op">$</span>size, <span class="dt">logsz =</span> <span class="ot">TRUE</span>, <span class="dt">method =</span> <span class="st">&quot;PredLine&quot;</span>, <span class="dt">pch =</span> pch.gps.reg2, <span class="dt">col =</span> col.gps.reg2)</span></code></pre></div>
<p><img src="gahaganmorph3_files/figure-html/allometry-4.png" width="100%" /></p>
<div class="sourceCode" id="cb106"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb106-1"><a href="geometric-morphometrics.html#cb106-1"></a><span class="co"># plot form</span></span>
<span id="cb106-2"><a href="geometric-morphometrics.html#cb106-2"></a><span class="co">#knitr::include_graphics(&#39;images/gbiface-allom-form.png&#39;)</span></span></code></pre></div>
</div>
<div id="sizeshape-region" class="section level2">
<h2><span class="header-section-number">5.6</span> Size/Shape ~ Region?</h2>
<p>A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(Adams and Collyer <a href="#ref-RN1655" role="doc-biblioref">2015</a>; Collyer and Adams <a href="#ref-RN11775" role="doc-biblioref">2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(Anderson and Ter Braak <a href="#ref-RN1719" role="doc-biblioref">2003</a>)</span>. To assess whether shape changes with size (allometry), and differs by group (region), Procrustes ANOVAs <span class="citation">(Goodall <a href="#ref-RN1749" role="doc-biblioref">1991</a>)</span> were also run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(Collyer, Sekora, and Adams <a href="#ref-RN1756" role="doc-biblioref">2015</a>)</span>.</p>
<div class="sourceCode" id="cb107"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb107-1"><a href="geometric-morphometrics.html#cb107-1"></a><span class="co"># ANOVA: do gahagan biface shapes differ by region2?</span></span>
<span id="cb107-2"><a href="geometric-morphometrics.html#cb107-2"></a><span class="kw">anova</span>(fit.shapereg2)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS       MS     Rsq      F      Z Pr(&gt;F)    
## reg2        2 0.16999 0.084997 0.30411 21.632 5.3584  1e-04 ***
## Residuals  99 0.38899 0.003929 0.69589                         
## Total     101 0.55899                                          
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = shape ~ reg2, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb109"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb109-1"><a href="geometric-morphometrics.html#cb109-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb109-2"><a href="geometric-morphometrics.html#cb109-2"></a>sh.reg2 &lt;-<span class="st"> </span><span class="kw">pairwise</span>(fit.shapereg2, <span class="dt">groups =</span> qdata<span class="op">$</span>region2)</span>
<span id="cb109-3"><a href="geometric-morphometrics.html#cb109-3"></a><span class="kw">summary</span>(sh.reg2, <span class="dt">confidence =</span> <span class="fl">0.95</span>, <span class="dt">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                              d  UCL (95%)         Z Pr &gt; d
## CTX:SCA-north       0.12180388 0.04135610 10.475838  1e-04
## CTX:SCA-south       0.05267038 0.02814872  5.800386  1e-04
## SCA-north:SCA-south 0.09306458 0.03842682  8.237202  1e-04</code></pre>
<div class="sourceCode" id="cb111"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb111-1"><a href="geometric-morphometrics.html#cb111-1"></a><span class="co"># pairwise distance between variances = standardization?</span></span>
<span id="cb111-2"><a href="geometric-morphometrics.html#cb111-2"></a><span class="kw">summary</span>(sh.reg2, <span class="dt">confidence =</span> <span class="fl">0.95</span>, <span class="dt">test.type =</span> <span class="st">&quot;var&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## 
## Observed variances by group
## 
##         CTX   SCA-north   SCA-south 
## 0.005043921 0.001078684 0.003790750 
## 
## Pairwise distances between variances, plus statistics
##                               d   UCL (95%)         Z Pr &gt; d
## CTX:SCA-north       0.003965237 0.002953361 3.0023002 0.0117
## CTX:SCA-south       0.001253171 0.001955585 0.7101339 0.2262
## SCA-north:SCA-south 0.002712066 0.002653129 1.8896042 0.0457</code></pre>
<div class="sourceCode" id="cb113"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb113-1"><a href="geometric-morphometrics.html#cb113-1"></a><span class="co"># ANOVA: do gahagan biface sizes differ by region2?</span></span>
<span id="cb113-2"><a href="geometric-morphometrics.html#cb113-2"></a><span class="kw">anova</span>(fit.sizereg2)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##            Df      SS     MS     Rsq      F      Z Pr(&gt;F)    
## reg2        2  544677 272339 0.36042 27.895 2.9772  1e-04 ***
## Residuals  99  966535   9763 0.63958                         
## Total     101 1511213                                        
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = size ~ reg2, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb115"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb115-1"><a href="geometric-morphometrics.html#cb115-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb115-2"><a href="geometric-morphometrics.html#cb115-2"></a>sz.reg2 &lt;-<span class="st"> </span><span class="kw">pairwise</span>(fit.sizereg2, <span class="dt">groups =</span> qdata<span class="op">$</span>region2)</span>
<span id="cb115-3"><a href="geometric-morphometrics.html#cb115-3"></a><span class="kw">summary</span>(sz.reg2, <span class="dt">confidence =</span> <span class="fl">0.95</span>, <span class="dt">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                             d UCL (95%)          Z Pr &gt; d
## CTX:SCA-north       145.74964  79.15668  4.6087559 0.0007
## CTX:SCA-south       163.02809  53.30119  8.5253197 0.0001
## SCA-north:SCA-south  17.27845  73.58797 -0.5632569 0.6517</code></pre>
<div class="sourceCode" id="cb117"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb117-1"><a href="geometric-morphometrics.html#cb117-1"></a><span class="co"># pairwise distance between variances = standardization?</span></span>
<span id="cb117-2"><a href="geometric-morphometrics.html#cb117-2"></a><span class="kw">summary</span>(sz.reg2, <span class="dt">confidence =</span> <span class="fl">0.95</span>, <span class="dt">test.type =</span> <span class="st">&quot;var&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: CTX SCA-north SCA-south 
## 
## RRPP: 10000 permutations
## 
## 
## Observed variances by group
## 
##        CTX  SCA-north  SCA-south 
##   688.7314 23563.1976 10839.8620 
## 
## Pairwise distances between variances, plus statistics
##                            d UCL (95%)        Z Pr &gt; d
## CTX:SCA-north       22874.47 14841.912 3.624542 0.0058
## CTX:SCA-south       10151.13  9283.022 2.093560 0.0302
## SCA-north:SCA-south 12723.34 14152.784 1.692350 0.0649</code></pre>
</div>
<div id="morphological-disparity" class="section level2">
<h2><span class="header-section-number">5.7</span> Morphological disparity</h2>
<p>Procrustes variance was used to discriminate between regions and compare the amount of shape variation (morphological disparity) <span class="citation">(Zelditch et al. <a href="#ref-RN5703" role="doc-biblioref">2004</a>)</span>, estimated as the Procrustes variance using residuals of linear model fit <span class="citation">(Adams et al. <a href="#ref-RN11530" role="doc-biblioref">2017</a>)</span>.</p>
<div class="sourceCode" id="cb119"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb119-1"><a href="geometric-morphometrics.html#cb119-1"></a><span class="co"># morphological disparity: do either of the groups display greater shape variation among individuals relative to the other group?</span></span>
<span id="cb119-2"><a href="geometric-morphometrics.html#cb119-2"></a><span class="kw">morphol.disparity</span>(fit.shapereg2, <span class="dt">groups =</span> qdata<span class="op">$</span>region2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.shapereg2, groups = qdata$region2,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##         CTX   SCA-north   SCA-south 
## 0.005043921 0.001078684 0.003790750 
## 
## 
## Pairwise absolute differences between variances
##                   CTX   SCA-north   SCA-south
## CTX       0.000000000 0.003965237 0.001253171
## SCA-north 0.003965237 0.000000000 0.002712066
## SCA-south 0.001253171 0.002712066 0.000000000
## 
## 
## P-Values
##              CTX SCA-north SCA-south
## CTX       1.0000    0.0117    0.2262
## SCA-north 0.0117    1.0000    0.0457
## SCA-south 0.2262    0.0457    1.0000</code></pre>
<div class="sourceCode" id="cb121"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb121-1"><a href="geometric-morphometrics.html#cb121-1"></a><span class="co"># morphological disparity: do either of the groups display greater size variation among individuals relative to the other group?</span></span>
<span id="cb121-2"><a href="geometric-morphometrics.html#cb121-2"></a><span class="kw">morphol.disparity</span>(fit.sizereg2, <span class="dt">groups =</span> qdata<span class="op">$</span>region2, <span class="dt">data =</span> gdf, <span class="dt">print.progress =</span> <span class="ot">FALSE</span>, <span class="dt">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = fit.sizereg2, groups = qdata$region2,  
##     iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##        CTX  SCA-north  SCA-south 
##   688.7314 23563.1976 10839.8620 
## 
## 
## Pairwise absolute differences between variances
##                CTX SCA-north SCA-south
## CTX           0.00  22874.47  10151.13
## SCA-north 22874.47      0.00  12723.34
## SCA-south 10151.13  12723.34      0.00
## 
## 
## P-Values
##              CTX SCA-north SCA-south
## CTX       1.0000    0.0058    0.0302
## SCA-north 0.0058    1.0000    0.0649
## SCA-south 0.0302    0.0649    1.0000</code></pre>
</div>
<div id="mean-shapes" class="section level2">
<h2><span class="header-section-number">5.8</span> Mean shapes</h2>
<div class="sourceCode" id="cb123"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb123-1"><a href="geometric-morphometrics.html#cb123-1"></a><span class="co">#subset landmark coordinates to produce mean shapes for groups</span></span>
<span id="cb123-2"><a href="geometric-morphometrics.html#cb123-2"></a>new.coords&lt;-<span class="kw">coords.subset</span>(<span class="dt">A =</span> Y.gpa<span class="op">$</span>coords, <span class="dt">group =</span> qdata<span class="op">$</span>region2)</span>
<span id="cb123-3"><a href="geometric-morphometrics.html#cb123-3"></a><span class="kw">names</span>(new.coords)</span></code></pre></div>
<pre><code>## [1] &quot;CTX&quot;       &quot;SCA-north&quot; &quot;SCA-south&quot;</code></pre>
<div class="sourceCode" id="cb125"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb125-1"><a href="geometric-morphometrics.html#cb125-1"></a><span class="co">#group shape means</span></span>
<span id="cb125-2"><a href="geometric-morphometrics.html#cb125-2"></a>mean&lt;-<span class="kw">lapply</span>(new.coords, mshape)</span>
<span id="cb125-3"><a href="geometric-morphometrics.html#cb125-3"></a><span class="co"># plot(mean$CTX)</span></span>
<span id="cb125-4"><a href="geometric-morphometrics.html#cb125-4"></a><span class="co"># plot(mean$SCA-north)</span></span>
<span id="cb125-5"><a href="geometric-morphometrics.html#cb125-5"></a><span class="co"># plotRefToTarget(mean$SCA-north,mean$CTX, method=&quot;vector&quot;,mag=2)</span></span></code></pre></div>
<div class="sourceCode" id="cb126"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb126-1"><a href="geometric-morphometrics.html#cb126-1"></a><span class="co"># knitr::include_graphics(&#39;images/mshape-region.png&#39;)</span></span></code></pre></div>
</div>
<div id="acknowledgments" class="section level2">
<h2><span class="header-section-number">5.9</span> Acknowledgments</h2>
<p>I extend my gratitude to the Caddo Tribe of Oklahoma, the Williamson Museum at Northwestern State University, the Louisiana State Exhibit Museum, the Texas Archeological Research Laboratory at The University of Texas at Austin, the Brazos Valley Museum of Natural History, the Texas Parks and Wildlife Department, and the Sam Noble Oklahoma Museum of Natural Science for the requisite permissions and access needed to generate the scans of Gahagan bifaces. Thanks to Harry J. Shafer, Jeffrey S. Girard, Hiram F. (Pete) Gregory, Julian A. Sitters, Timothy K. Perttula, and David K. Thulman for their comments on early drafts of this manuscript. Thanks also to Dean C. Adams, Michael L. Collyer, Emma Sherratt, Lauren Butaric, and Kersten Bergstrom for their constructive criticisms, comments, and suggestions throughout the development of this research programme, and to the editors and anonymous reviewers for their comments and constructive criticisms, which further improved the manuscript.</p>
<p>Components of this analytical work flow were developed and funded by a Preservation Technology and Training grant (P14AP00138) to RZS from the National Center for Preservation Technology and Training, and funding to scan the Gahagan bifaces at the Williamson Museum at Northwestern State University, Louisiana State Exhibit Museum, Texas Archeological Research Laboratory at The University of Texas at Austin, and Sam Noble Oklahoma Museum of Natural Science was provided by the Heritage Research Center at Stephen F. Austin State University.</p>
</div>
<div id="references-cited" class="section level2">
<h2><span class="header-section-number">5.10</span> References cited</h2>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-RN1655">
<p>Adams, Dean C., and Michael L. Collyer. 2015. “Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters.” <em>Evolution</em> 69 (3): 823–9. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.</p>
</div>
<div id="ref-RN11530">
<p>Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. “Package ’geomorph’: Geometric Morphometric Analyses of 2D/3D Landmark Data. R package version 3.0.5.” <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.</p>
</div>
<div id="ref-RN1774">
<p>Adams, Dean C., and Erik Otárola-Castillo. 2013. “geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data.” <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.</p>
</div>
<div id="ref-RN1719">
<p>Anderson, M. J., and C. J. F. Ter Braak. 2003. “Permutation Tests for Multi-Factoral Analysis of Variance.” <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.</p>
</div>
<div id="ref-RN11775">
<p>Collyer, Michael L., and Dean C. Adams. 2018. “RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization.” <em>Methods in Ecology and Evolution</em> 9 (7): 1772–9. <a href="https://doi.org/https://doi.org/10.1111/2041-210X.13029">https://doi.org/https://doi.org/10.1111/2041-210X.13029</a>.</p>
</div>
<div id="ref-RN1756">
<p>Collyer, M. L., D. J. Sekora, and D. C. Adams. 2015. “A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data.” <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.</p>
</div>
<div id="ref-RN1749">
<p>Goodall, Colin. 1991. “Procrustes Methods in the Statistical Analysis of Shape.” <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.</p>
</div>
<div id="ref-RN11564">
<p>Gower, J. C. 1975. “Generalized Procrustes Analysis.” <em>Psychometrika</em> 40 (1): 33–51. <a href="https://doi.org/https://doi.org/10.1007/BF02291478">https://doi.org/https://doi.org/10.1007/BF02291478</a>.</p>
</div>
<div id="ref-RN1746">
<p>Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.</p>
</div>
<div id="ref-RN11622">
<p>Kendall, David G. 1981. “The Statistics of Shape.” In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.</p>
</div>
<div id="ref-RN11623">
<p>Kendall, David G. 1984. “Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces.” <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.</p>
</div>
<div id="ref-RN1731">
<p>Klingenberg, Christian Peter. 2013. “Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes.” <em>Hystrix</em> 24 (1): 15–24. <a href="https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691">https://doi.org/http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.</p>
</div>
<div id="ref-R">
<p>R Core Development Team, 2020. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</p>
</div>
<div id="ref-RN1646">
<p>Rohlf, F. James. 1999. “Shape Statistics: Procrustes Superimpositions and Tangent Spaces.” <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.</p>
</div>
<div id="ref-RN478">
<p>Rohlf, F. James, and Dennis Slice. 1990. “Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks.” <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.</p>
</div>
<div id="ref-RN11801">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p>
</div>
<div id="ref-RN11782">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p>
</div>
<div id="ref-RN11716">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p> 2019. “Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections.” <em>Journal of Cultural Heritage</em> 35: 41–55. <a href="https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002</a>.</p>
</div>
<div id="ref-RN20697">
<p>Selden Jr., Robert Z. 2017. “Asymmetry of Caddo Ceramics from the Washington Square Mound Site: An Exploratory Analysis.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 5: 21–28. <a href="https://doi.org/10.1016/j.daach.2017.04.003">https://doi.org/10.1016/j.daach.2017.04.003</a>.</p> 2018a. “A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.” <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.</p> 2018b. “Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles.” <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.</p> 2019. “Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections.” <em>Journal of Cultural Heritage</em> 35: 41–55. <a href="https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/https://doi.org/10.1016/j.culher.2018.07.002</a>.</p> 2021. “Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.” In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, (in press). Baton Rouge: LSU Press.</p>
</div>
<div id="ref-RN21001">
<p>Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. “A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.” <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.</p>
</div>
<div id="ref-RN11783">
<p>Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. “Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area.” <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.</p>
</div>
<div id="ref-RN3684">
<p>Shafer, Harry J. 1973. “Lithic Technology at the George C. Davis Site, Cherokee County, Texas.” Unpublished Ph.D. Dissertation, The University of Texas at Austin.</p>
</div>
<div id="ref-RN20701">
<p>Shafer, Harry J. 1974. “Lithic Reduction Strategies at the George C. Davis Site.” <em>Louisiana Archaeology</em> 1: 66–74. <a href="https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf">https://docs.wixstatic.com/ugd/fefb33_71a3f0c39e5d47a2b55af09847e6d821.pdf</a>.</p>
</div>
<div id="ref-RN4924">
<p>Shafer, Harry J. 2006. “People of the Prairie: A Possible Connection to the Davis Site Caddo.” Texas Department of Transportation; Prewitt &amp; Associates, Inc.</p>
</div>
<div id="ref-RN479">
<p>Sherratt, E., D. J. Gower, C. P. Klingenberg, and M. Wilkinson. 2014. “Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona).” <em>Evolutionary Biology</em> 41 (4): 528–45. <a href="https://doi.org/https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/https://doi.org/10.1007/s11692-014-9287-2</a>.</p>
</div>
<div id="ref-RN11563">
<p>Slice, Dennis E. 2001. “Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space.” <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.</p>
</div>
<div id="ref-RN5703">
<p>Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="landmark-sampling-evaluation-curve-lasec.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["gahaganmorph3.pdf", "gahaganmorph3.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
